
<!doctype html>
<html class="no-js" lang="en" dir="ltr">

<head>

<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>CSIRO PUBLISHING | Australian Systematic Botany</title>

<link rel="shortcut icon" href="/media/client/JI_SB.gif">
<link rel="apple-touch-icon" href="/media/client/JI_SB.png"/>
<link rel="apple-touch-icon-precomposed" href="/media/client/JI_SB.png"/>
<link rel="icon" href="/media/client/JI_SB.png"/>

<link rel="stylesheet" href="/themes/new/css/foundation.min.css">
<link rel="stylesheet" href="/themes/new/css/foundation-icons.css">
<link rel="stylesheet" href="/themes/new/css/styles.css">
<link rel="stylesheet" href="/themes/new/css/custom.css">
<link rel="stylesheet" href="/themes/new/css/okayNav.min.css" media="screen">
<link rel="schema.dc" href="https://purl.org/dc/elements/1.1/">

<meta name="description" content="Australian Systematic Botany is an international journal devoted to the taxonomy, biogeography and evolution of all plant groups including fossils publishing papers and critical reviews that aim to advance systematic botany"/>

<meta name="keywords" content="plant , plants , algae, fungi , lichen , vascular , vascular plants , taxonomy , genetics , morphology , phylogeny , ecology , systematics , systematic botany , plant taxonomy , plant genetics , plant morphology , plant phylogeny , plant ecology , plant systematics , Australian botany , Gondwanan , CSIRO , CSIRO PUBLISHING , publications , science , educational , scientific , journal , journals , Australia , Australian , international"/>


<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,700,800' rel="stylesheet">
<link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>



<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-TFHK7DZ5');</script>
<!-- End Google Tag Manager -->

<script type="text/javascript">

			var _gaq = _gaq || [];
			_gaq.push(['_setAccount', 'UA-3590465-1']);
			_gaq.push(['_trackPageview']);

			(function() {
			var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
			ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
			var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
			})();

</script>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Q860Q9VBFL"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Q860Q9VBFL');
</script>

<!-- Google tag (gtag.js) --> <script async src="https://www.googletagmanager.com/gtag/js?id=G-4GHBLY4XHS"></script> 
<script> 
	window.dataLayer = window.dataLayer || []; 
	function gtag(){dataLayer.push(arguments);} 
	gtag('js', new Date()); 
	gtag('config', 'G-4GHBLY4XHS'); 
</script>


<script src="https://scholar.google.com/scholar_js/casa.js" async></script>


			<meta name="citation_journal_title" content="Australian Systematic Botany">
			<meta name="citation_journal_abbrev" content="Aust. Systematic Bot.">
			<meta name="citation_publisher" content="CSIRO PUBLISHING">
			<meta name="citation_author" content="E. M. Joyce">
				<meta name="citation_author_orcid" content="0000-0001-8291-8058">
				<meta name="citation_author_institution" content=", , , .">
				<meta name="citation_author" content="A. N. Schmidt-Lebuhn">
				<meta name="citation_author_orcid" content="0000-0002-7402-8941">
				<meta name="citation_author_institution" content=", , , , .">
				<meta name="citation_author" content="H. K. Orel">
				<meta name="citation_author_orcid" content="0000-0001-7971-8709">
				<meta name="citation_author_institution" content=", , , .">
				<meta name="citation_author" content="F. J. Nge">
				<meta name="citation_author_orcid" content="0000-0002-0361-8709">
				<meta name="citation_author_institution" content=", , , .">
				<meta name="citation_author" content="B. M. Anderson">
				<meta name="citation_author_institution" content=", , , .">
				<meta name="citation_author" content="T. A. Hammer">
				<meta name="citation_author_orcid" content="0000-0003-3816-7933">
				<meta name="citation_author_institution" content=", , , .">
				<meta name="citation_author_institution" content=", , , , .">
				<meta name="citation_author" content="T. G. B. McLay">
				<meta name="citation_author_institution" content=", , , .">
				<meta name="citation_author_institution" content=", , , .">
				<meta name="citation_author_institution" content=", , , .">
				
			<meta name="citation_title" content="Navigating phylogenetic conflict and evolutionary inference in plants with target-capture data">
			
			<meta name="citation_online_date" content="2025/05/07">
			<meta name="citation_publication_date" content="2025/05/07">
						
			<meta name="citation_volume" content="38">
			<meta name="citation_issue" content="3">
			<meta name="citation_firstpage" content="NULL">
			<meta name="citation_lastpage" content="NULL">
			
			<meta name="citation_doi" content="10.1071/SB24011">
			
			<meta name="citation_fulltext_world_readable" content="">
			
			<meta name="citation_keywords" content="Angiosperms353, deep coalescence, discordance, GAP, Genomics for Australian Plants, hybridisation, HybSeq, incomplete lineage sorting, incongruence, PAFTOL, paralogy, polyploidy, polytomy, target enrichment">
			
			<meta name="citation_pdf_url" content="https://www.publish.csiro.au/sb/pdf/SB24011">
			<meta name="citation_issn" content="1446-5701">
			<meta name="citation_language" content="en">
			<meta name="citation_article_type" content="RESEARCH ARTICLE">
		  	  <meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=ExaBayes: massively parallel bayesian tree inference for the whole-genome era.;  citation_author=AJ Aberer; citation_author=K Kobert; citation_author=A Stamatakis;   citation_volume=31;  citation_firstpage=2553; citation_lastpage=2556; citation_doi=25135941; ">
			<meta name="citation_reference" content="Aïnouche M, Wendel J (2014) Polyploid speciation and genome evolution: lessons from recent allopolyploids. In ‘Evolutionary Biology: Genome Evolution, Speciation, Coevolution and Origin of Life’. (Ed. P Pontarotti) pp. 87–113. (Springer: Cham, Switzerland)">
			<meta name="citation_reference" content="citation_journal_title=Annals of Botany; citation_title=Polyploidy and interspecific hybridization: partners for adaptation, speciation and evolution in plants.;  citation_author=K Alix; citation_author=PR G&#xe9;rard; citation_author=T Schwarzacher; citation_author=J Heslop-Harrison;   citation_volume=120;  citation_firstpage=183; citation_lastpage=194; citation_doi=28854567; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Whole-genome duplications and the long-term evolution of gene regulatory networks in angiosperms.;  citation_author=F Almeida-Silva; citation_author=Y Van de Peer;   citation_volume=40;    citation_doi=37405949; ">
			<meta name="citation_reference" content="citation_journal_title=PeerJ; citation_title=SECAPR—a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments.;  citation_author=T Andermann; citation_author= Cano; citation_author=A Zizka; citation_author=C Bacon; citation_author=A Antonelli;   citation_volume=6;  citation_firstpage=e5175;  citation_doi=10.7717/peerj.5175; ">
			<meta name="citation_reference" content="citation_journal_title=Frontiers in Genetics; citation_title=A guide to carrying out a phylogenomic target sequence capture project.;  citation_author=T Andermann; citation_author=MF Torres Jim&#xe9;nez; citation_author=P Matos-Marav&#xed;; citation_author=R Batista; citation_author=JL Blanco-Pastor; citation_author=ALS Gustafsson; citation_author=L Kistler; citation_author=IM Liberal; citation_author=B Oxelman; citation_author=CD Bacon; citation_author=A Antonelli;   citation_volume=10;    citation_doi=32153629; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Bayesian estimation of concordance among gene trees.;  citation_author=C An&#xe9;; citation_author=B Larget; citation_author=DA Baum; citation_author=SD Smith; citation_author=A Rokas;   citation_volume=24;  citation_firstpage=412; citation_lastpage=426; citation_doi=17095535; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative.;  citation_author=M Anisimova; citation_author=O Gascuel;   citation_volume=55;  citation_firstpage=539; citation_lastpage=552; citation_doi=16785212; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Exploring Angiosperms353: an open, community toolkit for collaborative phylogenomic research on flowering plants.;  citation_author=WJ Baker; citation_author=S Dodsworth; citation_author=F Forest; citation_author=SW Graham; citation_author=MG Johnson; citation_author=A McDonnell; citation_author=L Pokorny; citation_author=JA Tate; citation_author=S Wicke; citation_author=NJ Wickett;   citation_volume=108;  citation_firstpage=1059; citation_lastpage=1065; citation_doi=34293179; ">
			<meta name="citation_reference" content="citation_journal_title=Journal of Computational Biology: A Journal of Computational Molecular Cell Biology; citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.;  citation_author=A Bankevich; citation_author=S Nurk; citation_author=D Antipov; citation_author=AA Gurevich; citation_author=M Dvorkin; citation_author=AS Kulikov; citation_author=VM Lesin; citation_author=SI Nikolenko; citation_author=S Pham; citation_author=AD Prjibelski; citation_author=AV Pyshkin; citation_author=AV Sirotkin; citation_author=N Vyahhi; citation_author=G Tesler; citation_author=MA Alekseyev; citation_author=PA Pevzner;   citation_volume=19;  citation_firstpage=455; citation_lastpage=477; citation_doi=22506599; ">
			<meta name="citation_reference" content="citation_journal_title=Genome Biology and Evolution; citation_title=Assessing rapid relaxed-clock methods for phylogenomic dating.;  citation_author=J Barba-Montoya; citation_author=Q Tao; citation_author=S Kumar;   citation_volume=13;    citation_doi=34751377; ">
			<meta name="citation_reference" content="citation_journal_title=TAXON; citation_title=Concordance trees, concordance factors, and the exploration of reticulate genealogy.;  citation_author=DA Baum;   citation_volume=56;  citation_firstpage=417; citation_lastpage=426; citation_doi=10.1002/tax.562013; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Phylogenetic trees and networks can serve as powerful and complementary approaches for analysis of genomic data.;  citation_author=C Blair; citation_author=C An&#xe9;;   citation_volume=69;  citation_firstpage=593; citation_lastpage=601; citation_doi=31432090; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=HyDe: a Python package for genome-scale hybridization detection.;  citation_author=PD Blischak; citation_author=J Chifman; citation_author=AD Wolfe; citation_author=LS Kubatko;   citation_volume=67;  citation_firstpage=821; citation_lastpage=829; citation_doi=29562307; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=HybPhaser identifies hybrid evolution in Australian Thelypteridaceae.;  citation_author=Z Bloesch; citation_author=L Nauheimer; citation_author=T Elias Almeida; citation_author=D Crayn; citation_author=AR Field;   citation_volume=173;    citation_doi=35577295; ">
			<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology; citation_title=Multi-species outcomes in a common model of sympatric speciation.;  citation_author=DI Bolnick;   citation_volume=241;  citation_firstpage=734; citation_lastpage=744; citation_doi=16483610; ">
			<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society of London – B. Biological Sciences; citation_title=When everything changes at once: finding a new normal after genome duplication.;  citation_author=K Bomblies;   citation_volume=287;    citation_doi=33203329; ">
			<meta name="citation_reference" content="citation_journal_title=PeerJ; citation_title=AMAS: a fast tool for alignment manipulation and computing of summary statistics.;  citation_author=ML Borowiec;   citation_volume=4;    citation_doi=26835189; ">
			<meta name="citation_reference" content="citation_journal_title=PLoS Computational Biology; citation_title=BEAST 2: a software platform for Bayesian evolutionary analysis.;  citation_author=R Bouckaert; citation_author=J Heled; citation_author=D K&#xfc;hnert; citation_author=T Vaughan; citation_author=C-H Wu; citation_author=D Xie; citation_author=MA Suchard; citation_author=A Rambaut; citation_author=AJ Drummond;   citation_volume=10;    citation_doi=24722319; ">
			<meta name="citation_reference" content="Boussau B, Scornavacca C (2020) Reconciling gene trees with species trees. In ‘Phylogenetics in the Genomic Era’. (Eds C Scornavacca, F Delsuc, N Galtier) pp. 3.2:1–3.2:23. (Published by the authors, CC BY-NC-ND) Available at">
			<meta name="citation_reference" content="citation_journal_title=Plants; citation_title=Phylogenomics of southern European taxa in the Ranunculus auricomus species complex: the apple doesn’t fall far from the tree.;  citation_author=JP Bradican; citation_author=S Tomasello; citation_author=F Boscutti; citation_author=K Karbstein; citation_author=E H&#xf6;randl;   citation_volume=12;    citation_doi=37960021; ">
			<meta name="citation_reference" content="citation_journal_title=Applications in Plant Sciences; citation_title=A target enrichment probe set for resolving the flagellate land plant tree of life.;  citation_author=JW Breinholt; citation_author=SB Carey; citation_author=GP Tiley; citation_author=EC Davis; citation_author=L Endara; citation_author=SF McDaniel; citation_author=LG Neves; citation_author=EB Sessa; citation_author=M Von Konrat; citation_author=S Chantanaorrapint; citation_author=S Fawcett; citation_author=SM Ickert-Bond; citation_author=PH Labiak; citation_author=J Larra&#xed;n; citation_author=M Lehnert; citation_author=LR Lewis; citation_author=NS Nagalingum; citation_author=N Patel; citation_author=SA Rensing; citation_author=W Testo; citation_author=A Vasco; citation_author=JC Villarreal; citation_author=EW Williams; citation_author=JG Burleigh;   citation_volume=9;    citation_doi=33552748; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Neighbor-Net: an agglomerative method for the construction of phylogenetic networks.;  citation_author=D Bryant; citation_author=V Moulton;   citation_volume=21;  citation_firstpage=255; citation_lastpage=265; citation_doi=14660700; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=The perfect storm: gene tree estimation error, incomplete lineage sorting, and ancient gene flow explain the most recalcitrant ancient angiosperm clade, Malpighiales.;  citation_author=L Cai; citation_author=Z Xi; citation_author=EM Lemmon; citation_author=AR Lemmon; citation_author=A Mast; citation_author=CE Buddenhagen; citation_author=L Liu; citation_author=CC Davis;   citation_volume=70;  citation_firstpage=491; citation_lastpage=507; citation_doi=33169797; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses.;  citation_author=S Capella-Guti&#xe9;rrez; citation_author=JM Silla-Mart&#xed;nez; citation_author=T Gabald&#xf3;n;   citation_volume=25;  citation_firstpage=1972; citation_lastpage=1973; citation_doi=19505945; ">
			<meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=The phylogeny and biogeography of Hakea (Proteaceae) reveals the role of biome shifts in a continental plant radiation.;  citation_author=M Cardillo; citation_author=PH Weston; citation_author=ZKM Reynolds; citation_author=PM Olde; citation_author=AR Mast; citation_author=EM Lemmon; citation_author=AR Lemmon; citation_author=L Bromham;   citation_volume=71;  citation_firstpage=1928; citation_lastpage=1943; citation_doi=28548206; ">
			<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=iGTP: a software package for large-scale gene tree parsimony analysis.;  citation_author=R Chaudhary; citation_author=MS Bansal; citation_author=A Wehe; citation_author=D Fern&#xe1;ndez-Baca; citation_author=O Eulenstein;   citation_volume=11;    citation_doi=21092314; ">
			<meta name="citation_reference" content="citation_journal_title=Trends in Plant Science; citation_title=Whole-genome duplication and plant macroevolution.;  citation_author=JW Clark; citation_author=PCJ Donoghue;   citation_volume=23;  citation_firstpage=933; citation_lastpage=945; citation_doi=30122372; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=PhylteR: efficient identification of outlier sequences in phylogenomic datasets.;  citation_author=A Comte; citation_author=T Tricou; citation_author=E Tannier; citation_author=J Joseph; citation_author=A Siberchicot; citation_author=S Penel; citation_author=R Allio; citation_author=F Delsuc; citation_author=S Dray; citation_author=DM De Vienne;   citation_volume=40;    citation_doi=37879113; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Journal of Taxonomy; citation_title=Aglaia fellii W.E.Cooper &amp; Joyce (Meliaceae), a new species for Cape York Peninsula.;  citation_author=WE Cooper; citation_author=DM Crayn; citation_author=EM Joyce;   citation_volume=16;  citation_firstpage=1; citation_lastpage=9; citation_doi=10.54102/ajt.p8to6; ">
			<meta name="citation_reference" content="citation_journal_title=Journal of Systematics and Evolution jse; citation_title=Perianth evolution and implications for generic delimitation in the eucalypts (Myrtaceae), including the description of the new genus, Blakella.;  citation_author=MD Crisp; citation_author=BQ Minh; citation_author=B Choi; citation_author=RD Edwards; citation_author=J Hereward; citation_author=C Kulheim; citation_author=YP Lin; citation_author=K Meusemann; citation_author=AH Thornhill; citation_author=A Toon; citation_author=LG Cook;   citation_volume=62;  citation_firstpage=942; citation_lastpage=962; citation_doi=10.1111/jse.13047; ">
			<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution; citation_title=Genome duplication and the origin of angiosperms.;  citation_author=S De Bodt; citation_author=S Maere; citation_author=Y Van de Peer;   citation_volume=20;  citation_firstpage=591; citation_lastpage=597; citation_doi=16701441; ">
			<meta name="citation_reference" content="citation_journal_title=Journal of Experimental Botany; citation_title=Whole genome duplications in plants: an overview from Arabidopsis.;  citation_author=JC del Pozo; citation_author=E Ramirez-Parra;   citation_volume=66;  citation_firstpage=6991; citation_lastpage=7003; citation_doi=26417017; ">
			<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution; citation_title=Speciation and phylogenetic resolution.;  citation_author=R DeSalle; citation_author=R Absher; citation_author=G Amato;   citation_volume=9;  citation_firstpage=297; citation_lastpage=298; citation_doi=21236859; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Simultaneous speciation in the European high mountain flowering plant genus Facchinia (Minuartia sl, Caryophyllaceae) revealed by genotyping-by-sequencing.;  citation_author=MS Dillenberger; citation_author=JW Kadereit;   citation_volume=112;  citation_firstpage=23; citation_lastpage=35; citation_doi=28433621; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=StarBeast3: adaptive parallelized Bayesian inference under the multispecies coalescent.;  citation_author=J Douglas; citation_author=CL Jim&#xe9;nez-Silva; citation_author=R Bouckaert;   citation_volume=71;  citation_firstpage=901; citation_lastpage=916; citation_doi=35176772; ">
			<meta name="citation_reference" content="citation_journal_title=Annual Review of Genetics; citation_title=Prevalence and adaptive impact of introgression.;  citation_author=NB Edelman; citation_author=J Mallet;   citation_volume=55;  citation_firstpage=265; citation_lastpage=283; citation_doi=34579539; ">
			<meta name="citation_reference" content="citation_journal_title=Science; citation_title=Genomic architecture and introgression shape a butterfly radiation.;  citation_author=NB Edelman; citation_author=PB Frandsen; citation_author=M Miyagi; citation_author=B Clavijo; citation_author=J Davey; citation_author=RB Dikow; citation_author=G Garc&#xed;a-Accinelli; citation_author=SM Van Belleghem; citation_author=N Patterson; citation_author=DE Neafsey; citation_author=R Challis; citation_author=S Kumar; citation_author=GRP Moreira; citation_author=C Salazar; citation_author=M Chouteau; citation_author=BA Counterman; citation_author=R Papa; citation_author=M Blaxter; citation_author=RD Reed; citation_author=KK Dasmahapatra; citation_author=M Kronforst; citation_author=M Joron; citation_author=CD Jiggins; citation_author=WO McMillan; citation_author=F Di Palma; citation_author=AJ Blumberg; citation_author=J Wakeley; citation_author=D Jaffe; citation_author=J Mallet;   citation_volume=366;  citation_firstpage=594; citation_lastpage=599; citation_doi=31672890; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=PHYLUCE is a software package for the analysis of conserved genomic loci.;  citation_author=BC Faircloth;   citation_volume=32;  citation_firstpage=786; citation_lastpage=788; citation_doi=26530724; ">
			<meta name="citation_reference" content="citation_journal_title=Evolutionary Bioinformatics Online; citation_title=HybPhyloMaker: target enrichment data analysis from raw reads to species trees.;  citation_author=T F&#xe9;r; citation_author=RE Schmickl;   citation_volume=14;    citation_doi=29348708; ">
			<meta name="citation_reference" content="citation_journal_title=The New Phytologist; citation_title=Gene duplication and evolutionary novelty in plants.;  citation_author=LE Flagel; citation_author=JF Wendel;   citation_volume=183;  citation_firstpage=557; citation_lastpage=564; citation_doi=19555435; ">
			<meta name="citation_reference" content="citation_journal_title=Plant Systematics and Evolution; citation_title=Plastid phylogenomic analysis of tribe Myoporeae (Scrophulariaceae).;  citation_author=RM Fowler; citation_author=TGB McLay; citation_author=TM Schuster; citation_author=BJ Buirchell; citation_author=DJ Murphy; citation_author=MJ Bayly;   citation_volume=306;    citation_doi=10.1007/s00606-020-01678-4; ">
			<meta name="citation_reference" content="citation_journal_title=bioRxiv; citation_title=Summary tests of introgression are highly sensitive to rate variation across lineages.;  citation_author=LE Frankel; citation_author=C An&#xe9;;  citation_publication_date=2023; citation_volume=2023;    citation_doi=10.1101/2023.01.26.525396; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Summary tests of introgression are highly sensitive to rate variation across lineages.;  citation_author=LE Frankel; citation_author=C An&#xe9;;  citation_publication_date=2023; citation_volume=72; citation_issue=6; citation_firstpage=1357; citation_lastpage=1369; citation_doi=37698548; ">
			<meta name="citation_reference" content="citation_journal_title=Methods in Ecology and Evolution; citation_title=homologizer: phylogenetic phasing of gene copies into polyploid subgenomes.;  citation_author=WA Freyman; citation_author=MG Johnson; citation_author=CJ Rothfels;   citation_volume=14;  citation_firstpage=1230; citation_lastpage=1244; citation_doi=10.1111/2041-210X.14072; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Artifactual orthologs and the need for diligent data exploration in complex phylogenomic datasets: a museomic case study from the Andean Flora.;  citation_author=LA Frost; citation_author=AM Bedoya; citation_author=LP Lagomarsino;   citation_volume=73;  citation_firstpage=308; citation_lastpage=322; citation_doi=38170162; ">
			<meta name="citation_reference" content="citation_journal_title=Peer Community Journal; citation_title=An approximate likelihood method reveals ancient gene flow between human, chimpanzee and gorilla.;  citation_author=N Galtier;   citation_volume=4;    citation_doi=10.24072/pcjournal.359; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Fitting the gene lineage into its species lineage, a parsimony strategy illustrated by cladograms constructed from globin sequences.;  citation_author=M Goodman; citation_author=J Czelusniak; citation_author=GW Moore; citation_author=AE Romero-Herrera; citation_author=G Matsuda;   citation_volume=28;  citation_firstpage=132; citation_lastpage=163; citation_doi=10.1093/sysbio/28.2.132; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Evolution of Lomandroideae: multiple origins of polyploidy and biome occupancy in Australia.;  citation_author=BF Gunn; citation_author=DJ Murphy; citation_author=NG Walsh; citation_author=JG Conran; citation_author=JC Pires; citation_author=TD Macfarlane; citation_author=JL Birch;   citation_volume=149;    citation_doi=32304826; ">
			<meta name="citation_reference" content="citation_journal_title=Botanical Journal of the Linnean Society; citation_title=Genomic data resolve phylogenetic relationships of Australian mat-rushes, Lomandra (Asparagaceae: Lomandroideae).;  citation_author=BF Gunn; citation_author=DJ Murphy; citation_author=NG Walsh; citation_author=JG Conran; citation_author=JC Pires; citation_author=TD Macfarlane; citation_author=MD Crisp; citation_author=LG Cook; citation_author=JL Birch;   citation_volume=204;  citation_firstpage=1; citation_lastpage=22; citation_doi=10.1093/botlinnean/boad034; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=A framework phylogeny of the diverse guinea-flowers (Hibbertia, Dilleniaceae) using high-throughput sequence data.;  citation_author=TA Hammer; citation_author=E Biffin; citation_author=K van Dijk; citation_author=KR Thiele; citation_author=M Waycott;   citation_volume=38; citation_issue=2;   citation_doi=10.1071/SB24009; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Phylogeny, evolution, and biogeographic history of Calandrinia (Montiaceae).;  citation_author=LP Hancock; citation_author=F Obbens; citation_author=AJ Moore; citation_author=K Thiele; citation_author=JM De Vos; citation_author=J West; citation_author=JAM Holtum; citation_author=EJ Edwards;   citation_volume=105;  citation_firstpage=1021; citation_lastpage=1034; citation_doi=29995314; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Ecology Resources; citation_title=SEGUL: ultrafast, memory‐efficient and mobile‐friendly software for manipulating and summarizing phylogenomic datasets.;  citation_author=H Handika; citation_author=JA Esselstyn;   citation_volume=24; citation_issue=7; citation_firstpage=e13964;  citation_doi=10.1111/1755-0998.13964; ">
			<meta name="citation_reference" content="citation_journal_title=TAXON; citation_title=Retrieval of hundreds of nuclear loci from herbarium specimens.;  citation_author=ML Hart; citation_author=LL Forrest; citation_author=JA Nicholls; citation_author=CA Kidner;   citation_volume=65;  citation_firstpage=1081; citation_lastpage=1092; citation_doi=10.12705/655.9; ">
			<meta name="citation_reference" content="citation_journal_title=Current Biology; citation_title=Global Brassicaceae phylogeny based on filtering of 1,000-gene dataset.;  citation_author=KP Hendriks; citation_author=C Kiefer; citation_author=IA Al-Shehbaz; citation_author=CD Bailey; citation_author=A Hooft van Huysduynen; citation_author=LA Nikolov; citation_author=L Nauheimer; citation_author=AR Zuntini; citation_author=DA German; citation_author=A Franzke; citation_author=MA Koch; citation_author=MA Lysak; citation_author=&#xd3; Toro-N&#xfa;&#xf1;ez; citation_author=B &#xd6;z&#xfc;do&#x11f;ru; citation_author=VR Invern&#xf3;n; citation_author=N Walden; citation_author=O Maurin; citation_author=NM Hay; citation_author=P Shushkov; citation_author=T Mand&#xe1;kov&#xe1;; citation_author=ME Schranz; citation_author=M Thulin; citation_author=MD Windham; citation_author=I Re&#x161;etnik; citation_author=S &#x160;paniel; citation_author=E Ly; citation_author=JC Pires; citation_author=A Harkess; citation_author=B Neuffer; citation_author=R Vogt; citation_author=C Br&#xe4;uchler; citation_author=H Rainer; citation_author=SB Janssens; citation_author=M Schmull; citation_author=A Forrest; citation_author=A Guggisberg; citation_author=S Zmarzty; citation_author=BJ Lepschi; citation_author=N Scarlett; citation_author=FW Stauffer; citation_author=I Sch&#xf6;nberger; citation_author=P Heenan; citation_author=WJ Baker; citation_author=F Forest; citation_author=K Mummenhoff; citation_author=F Lens;   citation_volume=33;  citation_firstpage=4052; citation_lastpage=4068; citation_doi=37659415; ">
			<meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Phylogenomic approaches to detecting and characterizing introgression.;  citation_author=MS Hibbins; citation_author=MW Hahn;   citation_volume=220;    citation_doi=34788444; ">
			<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution; citation_title=Patterns of speciation and limits to phylogenetic resolution.;  citation_author=GA Hoelzer; citation_author=DJ Meinick;   citation_volume=9;  citation_firstpage=104; citation_lastpage=107; citation_doi=21236789; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SplitsTree: analyzing and visualizing evolutionary data.;  citation_author=DH Huson;   citation_volume=14;  citation_firstpage=68; citation_lastpage=73; citation_doi=9520503; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Application of phylogenetic networks in evolutionary studies.;  citation_author=DH Huson; citation_author=D Bryant;   citation_volume=23; citation_issue=2; citation_firstpage=254; citation_lastpage=267; citation_doi=10.1093/molbev/msj030; ">
			<meta name="citation_reference" content="citation_journal_title=Applications in Plant Sciences; citation_title=hybpiper-nf and paragone-nf: containerization and additional options for target capture assembly and paralog resolution.;  citation_author=C Jackson; citation_author=T McLay; citation_author=AN Schmidt-Lebuhn;   citation_volume=11;    citation_doi=37601313; ">
			<meta name="citation_reference" content="citation_journal_title=Applications in Plant Sciences; citation_title=HybPiper: extracting coding sequence and introns for phylogenetics from high-throughput sequencing reads using target enrichment.;  citation_author=MG Johnson; citation_author=EM Gardner; citation_author=Y Liu; citation_author=R Medina; citation_author=B Goffinet; citation_author=AJ Shaw; citation_author=NJ Zerega; citation_author=NJ Wickett;   citation_volume=4;    citation_doi=27437175; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=A universal probe set for targeted sequencing of 353 nuclear genes from any flowering plant designed using k-medoids clustering.;  citation_author=MG Johnson; citation_author=L Pokorny; citation_author=S Dodsworth; citation_author=LR Botigu&#xe9;; citation_author=RS Cowan; citation_author=A Devault; citation_author=WL Eiserhardt; citation_author=N Epitawalage; citation_author=F Forest; citation_author=JT Kim; citation_author=JH Leebens-Mack; citation_author=IJ Leitch; citation_author=O Maurin; citation_author=DE Soltis; citation_author=PS Soltis; citation_author=GK Wong; citation_author=WJ Baker; citation_author=NJ Wickett;   citation_volume=68;  citation_firstpage=594; citation_lastpage=606; citation_doi=30535394; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Ecology Resources; citation_title=JML: testing hybridization from species trees.;  citation_author=S Joly;   citation_volume=12;  citation_firstpage=179; citation_lastpage=184; citation_doi=21899723; ">
			<meta name="citation_reference" content="citation_journal_title=The American Naturalist; citation_title=A statistical approach for distinguishing hybridization and incomplete lineage sorting.;  citation_author=S Joly; citation_author=PA McLenachan; citation_author=PJ Lockhart;   citation_volume=174;  citation_firstpage=54; citation_lastpage=70; citation_doi=19519219; ">
			<meta name="citation_reference" content="citation_journal_title=Frontiers in Plant Science; citation_title=Phylogenomic analyses of Sapindales support new family relationships, rapid Mid-Cretaceous Hothouse diversification, and heterogeneous histories of gene duplication.;  citation_author=EM Joyce; citation_author=MS Appelhans; citation_author=S Buerki; citation_author=M Cheek; citation_author=JM de Vos; citation_author=JR Pirani; citation_author=AR Zuntini; citation_author=JB Bachelier; citation_author=MJ Bayly; citation_author=MW Callmander; citation_author=MF Devecchi; citation_author=SK Pell; citation_author=M Groppo; citation_author=PP Lowry; citation_author=J Mitchell; citation_author=CM Siniscalchi; citation_author=J Munzinger; citation_author=HK Orel; citation_author=CM Pannell; citation_author=L Nauheimer; citation_author=H Sauquet; citation_author=A Weeks; citation_author=AN Muellner-Riehl; citation_author=IJ Leitch; citation_author=O Maurin; citation_author=F Forest; citation_author=K Nargar; citation_author=KR Thiele; citation_author=WJ Baker; citation_author=DM Crayn;   citation_volume=14;    citation_doi=36959945; ">
			<meta name="citation_reference" content="citation_journal_title=EcoEvoRxiv; citation_title=Navigating phylogenetic conflict and evolutionary inference in plants with target capture data.;  citation_author=EM Joyce; citation_author=AN Schmidt-Lebuhn; citation_author=HK Orel; citation_author=FJ Nge; citation_author=BM Anderson; citation_author=TA Hammer; citation_author=TGB McLay;   citation_volume=2024;    citation_doi=10.32942/X2WP6V; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Reticulate evolution helps explain apparent homoplasy in floral biology and pollination in baobabs (Adansonia; Bombacoideae; Malvaceae).;  citation_author=N Karimi; citation_author=CE Grover; citation_author=JP Gallagher; citation_author=JF Wendel; citation_author=C An&#xe9;; citation_author=DA Baum;   citation_volume=69;  citation_firstpage=462; citation_lastpage=478; citation_doi=31693158; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Allele phasing has minimal impact on phylogenetic reconstruction from targeted nuclear gene sequences in a case study of Artocarpus.;  citation_author=HR Kates; citation_author=MG Johnson; citation_author=EM Gardner; citation_author=NJC Zerega; citation_author=NJ Wickett;   citation_volume=105;  citation_firstpage=404; citation_lastpage=416; citation_doi=29729187; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference.;  citation_author=AM Kozlov; citation_author=D Darriba; citation_author=T Flouri; citation_author=B Morel; citation_author=A Stamatakis;   citation_volume=35;  citation_firstpage=4453; citation_lastpage=4455; citation_doi=31070718; ">
			<meta name="citation_reference" content="citation_journal_title=NAR Genomics and Bioinformatics; citation_title=Pitfalls of the site-concordance factor (sCF) as measure of phylogenetic branch support.;  citation_author=P K&#xfc;ck; citation_author=J Romahn; citation_author=K Meusemann;   citation_volume=4;    citation_doi=36128424; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Statistics and truth in phylogenomics.;  citation_author=S Kumar; citation_author=AJ Filipski; citation_author=FU Battistuzzi; citation_author=SL Kosakovsky Pond; citation_author=K Tamura;   citation_volume=29;  citation_firstpage=457; citation_lastpage=472; citation_doi=21873298; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Impact of whole‐genome duplication events on diversification rates in angiosperms.;  citation_author=JB Landis; citation_author=DE Soltis; citation_author=Z Li; citation_author=HE Marx; citation_author=MS Barker; citation_author=DC Tank; citation_author=PS Soltis;   citation_volume=105;  citation_firstpage=348; citation_lastpage=363; citation_doi=29719043; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=The meaning and measure of concordance factors in phylogenomics.;  citation_author=R Lanfear; citation_author=MW Hahn;   citation_volume=41;    citation_doi=39418118; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=BUCKy: gene tree/species tree reconciliation with Bayesian concordance analysis.;  citation_author=BR Larget; citation_author=SK Kotha; citation_author=CN Dewey; citation_author=C An&#xe9;;   citation_volume=26;  citation_firstpage=2910; citation_lastpage=2911; citation_doi=20861028; ">
			<meta name="citation_reference" content="citation_journal_title=Botanical Journal of the Linnean Society; citation_title=Resolving generic limits in Cyperaceae tribe Abildgaardieae using targeted sequencing.;  citation_author=I Larridon; citation_author=AR Zuntini; citation_author=RL Barrett; citation_author=KL Wilson; citation_author=JJ Bruhl; citation_author=P Goetghebeur; citation_author=WJ Baker; citation_author=GE Brewer; citation_author=N Epitawalage; citation_author=I Fairlie; citation_author=F Forest; citation_author=IAB Sabino Kikuchi; citation_author=L Pokorny; citation_author=I Semmouri; citation_author=D Spalink; citation_author=DA Simpson; citation_author=AM Muasya; citation_author=EH Roalson;   citation_volume=196;  citation_firstpage=163; citation_lastpage=187; citation_doi=10.1093/botlinnean/boaa099; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=The accuracy of species tree estimation under simulation: a comparison of methods.;  citation_author=AD Leach&#xe9;; citation_author=B Rannala;   citation_volume=60;  citation_firstpage=126; citation_lastpage=137; citation_doi=21088009; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Anchored hybrid enrichment for massively high-throughput phylogenomics.;  citation_author=AR Lemmon; citation_author=SA Emme; citation_author=EM Lemmon;   citation_volume=61; citation_issue=5; citation_firstpage=727; citation_lastpage=744; citation_doi=22605266; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Polytomies and Bayesian phylogenetic inference.;  citation_author=PO Lewis; citation_author=MT Holder; citation_author=KE Holsinger;   citation_volume=54;  citation_firstpage=241; citation_lastpage=253; citation_doi=16012095; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.;  citation_author=D Li; citation_author=C-M Liu; citation_author=R Luo; citation_author=K Sadakane; citation_author=T-W Lam;   citation_volume=31;  citation_firstpage=1674; citation_lastpage=1676; citation_doi=25609793; ">
			<meta name="citation_reference" content="citation_journal_title=Quantitative Biology; citation_title=Current challenges and solutions of de novo assembly.;  citation_author=X Liao; citation_author=M Li; citation_author=Y Zou; citation_author=F Wu; citation_author= Yi&#x2010;Pan; citation_author=J Wang;   citation_volume=7;  citation_firstpage=90; citation_lastpage=109; citation_doi=10.1007/s40484-019-0166-9; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Incongruence between chloroplast and species phylogenies in Eucalyptus subgenus Monocalyptus (Myrtaceae).;  citation_author=GE McKinnon; citation_author=DA Steane; citation_author=BM Potts; citation_author=RE Vaillancourt;   citation_volume=86;  citation_firstpage=1038; citation_lastpage=1046; citation_doi=10406727; ">
			<meta name="citation_reference" content="citation_journal_title=Applications in Plant Sciences; citation_title=New targets acquired: improving locus recovery from the Angiosperms353 probe set.;  citation_author=TGB McLay; citation_author=JL Birch; citation_author=BF Gunn; citation_author=W Ning; citation_author=JA Tate; citation_author=L Nauheimer; citation_author=EM Joyce; citation_author=L Simpson; citation_author=AN Schmidt-Lebuhn; citation_author=WJ Baker; citation_author=F Forest; citation_author=CJ Jackson;   citation_volume=9;    citation_doi=34336399; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Phylogenomics reveals extreme gene tree discordance in a lineage of dominant trees: hybridization, introgression, and incomplete lineage sorting blur deep evolutionary relationships despite clear species groupings in Eucalyptus subgenus Eudesmia.;  citation_author=TGB McLay; citation_author=RM Fowler; citation_author=PS Fahey; citation_author=DJ Murphy; citation_author=F Udovicic; citation_author=DJ Cantrill; citation_author=MJ Bayly;   citation_volume=187;    citation_doi=37423562; ">
			<meta name="citation_reference" content="citation_journal_title=Cladistics; citation_title=Reconstructing character evolution on polytomous cladograms.;  citation_author=W Maddison;   citation_volume=5;  citation_firstpage=365; citation_lastpage=377; citation_doi=34933477; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Gene trees in species trees.;  citation_author=WP Maddison;   citation_volume=46;  citation_firstpage=523; citation_lastpage=536; citation_doi=10.1093/sysbio/46.3.523; ">
			<meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=TreeShrink: fast and accurate detection of outlier long branches in collections of phylogenetic trees.;  citation_author=U Mai; citation_author=S Mirarab;   citation_volume=19;    citation_doi=29745847; ">
			<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution; citation_title=A combinatorial view on speciation and adaptive radiation.;  citation_author=DA Marques; citation_author=JI Meier; citation_author=O Seehausen;   citation_volume=34;  citation_firstpage=531; citation_lastpage=544; citation_doi=30885412; ">
			<meta name="citation_reference" content="citation_journal_title=Frontiers in Genetics; citation_title=Homoeologous exchanges, segmental allopolyploidy, and polyploid genome evolution.;  citation_author=AS Mason; citation_author=JF Wendel;   citation_volume=11;    citation_doi=33005183; ">
			<meta name="citation_reference" content="citation_journal_title=Population Ecology; citation_title=The speciation view: disentangling multiple causes of adaptive and non-adaptive radiation in terms of speciation.;  citation_author=KW Matsubayashi; citation_author=R Yamaguchi;   citation_volume=64;  citation_firstpage=95; citation_lastpage=107; citation_doi=10.1002/1438-390X.12103; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=New methods to calculate concordance factors for phylogenomic datasets.;  citation_author=BQ Minh; citation_author=MW Hahn; citation_author=R Lanfear;  citation_publication_date=2020; citation_volume=37;  citation_firstpage=2727; citation_lastpage=2733; citation_doi=32365179; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era.;  citation_author=BQ Minh; citation_author=HA Schmidt; citation_author=O Chernomor; citation_author=D Schrempf; citation_author=MD Woodhams; citation_author=A von Haeseler; citation_author=R Lanfear;  citation_publication_date=2020; citation_volume=37;  citation_firstpage=1530; citation_lastpage=1534; citation_doi=32011700; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ASTRAL: genome-scale coalescent-based species tree estimation.;  citation_author=S Mirarab; citation_author=R Reaz; citation_author=MdS Bayzid; citation_author=T Zimmermann; citation_author=MS Swenson; citation_author=T Warnow;   citation_volume=30;  citation_firstpage=541; citation_lastpage=8; citation_doi=25161245; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Evaluating summary methods for multilocus species tree estimation in the presence of incomplete lineage sorting.;  citation_author=S Mirarab; citation_author=MS Bayzid; citation_author=T Warnow;   citation_volume=65;  citation_firstpage=366; citation_lastpage=380; citation_doi=25164915; ">
			<meta name="citation_reference" content="citation_journal_title=bioRxiv; citation_title=reconcILS: a gene tree-species tree reconciliation algorithm that allows for incomplete lineage sorting.;  citation_author=S Mishra; citation_author=ML Smith; citation_author=MW Hahn;   citation_volume=2023;    citation_doi=10.1101/2023.11.03.565544; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Updated site concordance factors minimize effects of homoplasy and taxon sampling.;  citation_author=YK Mo; citation_author=R Lanfear; citation_author=MW Hahn; citation_author=BQ Minh;   citation_volume=39;    citation_doi=36383168; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Applications of machine learning in phylogenetics.;  citation_author=YK Mo; citation_author=MW Hahn; citation_author=ML Smith;   citation_volume=196;    citation_doi=38565358; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=To include or not to include: the impact of gene filtering on species tree estimation methods.;  citation_author=EK Molloy; citation_author=T Warnow;   citation_volume=67;  citation_firstpage=285; citation_lastpage=303; citation_doi=29029338; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=FastMulRFS: fast and accurate species tree estimation under generic gene duplication and loss models.;  citation_author=EK Molloy; citation_author=T Warnow;   citation_volume=36;  citation_firstpage=i57; citation_lastpage=i65; citation_doi=32657396; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Phylogenomic subsampling and the search for phylogenetically reliable loci.;  citation_author=N Mongiardino Koch;   citation_volume=38;  citation_firstpage=4025; citation_lastpage=4038; citation_doi=33983409; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Analysis of paralogs in target enrichment data pinpoints multiple ancient polyploidy events in Alchemilla s.l. (Rosaceae).;  citation_author=DF Morales-Briones; citation_author=B Gehrke; citation_author=C-H Huang; citation_author=A Liston; citation_author=H Ma; citation_author=HE Marx; citation_author=DC Tank; citation_author=Y Yang;   citation_volume=71;  citation_firstpage=190; citation_lastpage=207; citation_doi=33978764; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=SpeciesRax: a tool for maximum likelihood species tree inference from gene family trees under duplication, transfer, and loss.;  citation_author=B Morel; citation_author=P Schade; citation_author=S Lutteropp; citation_author=TA Williams; citation_author=GJ Sz&#xf6;ll&#x151;si; citation_author=A Stamatakis;   citation_volume=39;    citation_doi=35021210; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=AleRax: a tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer, and loss.;  citation_author=B Morel; citation_author=TA Williams; citation_author=A Stamatakis; citation_author=GJ Sz&#xf6;ll&#x151;si;   citation_volume=40;    citation_doi=38514421; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=Genome skimming provides well resolved plastid and nuclear phylogenies, showing patterns of deep reticulate evolution in the tropical carnivorous plant genus Nepenthes (Caryophyllales).;  citation_author=L Nauheimer; citation_author=L Cui; citation_author=C Clarke; citation_author=DM Crayn; citation_author=G Bourke; citation_author=K Nargar;   citation_volume=32;  citation_firstpage=243; citation_lastpage=254; citation_doi=10.1071/SB18057; ">
			<meta name="citation_reference" content="citation_journal_title=Applications in Plant Sciences; citation_title=HybPhaser: a workflow for the detection and phasing of hybrids in target capture data sets.;  citation_author=L Nauheimer; citation_author=N Weigner; citation_author=E Joyce; citation_author=D Crayn; citation_author=C Clarke; citation_author=K Nargar;   citation_volume=9;    citation_doi=34336402; ">
			<meta name="citation_reference" content="citation_journal_title=Tree Genetics &amp; Genomes; citation_title=Shared phylogeographic patterns and widespread chloroplast haplotype sharing in Eucalyptus species with different ecological tolerances.;  citation_author=PG Nevill; citation_author=T Despr&#xe9;s; citation_author=MJ Bayly; citation_author=G Bossinger; citation_author=PK Ades;   citation_volume=10;  citation_firstpage=1079; citation_lastpage=1092; citation_doi=10.1007/s11295-014-0744-y; ">
			<meta name="citation_reference" content="citation_journal_title=Frontiers in Ecology and Evolution; citation_title=Reticulate evolution, ancient chloroplast haplotypes, and rapid radiation of the Australian plant genus Adenanthos (Proteaceae).;  citation_author=FJ Nge; citation_author=E Biffin; citation_author=KR Thiele; citation_author=M Waycott;  citation_publication_date=2021; citation_volume=8;    citation_doi=10.3389/fevo.2020.616741; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Historical biogeography of Pomaderris (Rhamnaceae): continental vicariance in Australia and repeated independent dispersals to New Zealand.;  citation_author=FJ Nge; citation_author=J Kellermann; citation_author=E Biffin; citation_author=M Waycott; citation_author=KR Thiele;  citation_publication_date=2021; citation_volume=158;    citation_doi=33540078; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Phylogenomics and continental biogeographic disjunctions: insight from the Australian starflowers (Calytrix).;  citation_author=FJ Nge; citation_author=E Biffin; citation_author=M Waycott; citation_author=KR Thiele;   citation_volume=109;  citation_firstpage=291; citation_lastpage=308; citation_doi=34671970; ">
			<meta name="citation_reference" content="citation_journal_title=Botanical Journal of the Linnean Society; citation_title=Rise and fall of a continental mesic radiation in Australia: spine evolution, biogeography, and diversification of Cryptandra (Rhamnaceae: Pomaderreae).;  citation_author=FJ Nge; citation_author=J Kellermann; citation_author=E Biffin; citation_author=KR Thiele; citation_author=M Waycott;   citation_volume=204;  citation_firstpage=327; citation_lastpage=342; citation_doi=10.1093/botlinnean/boad051; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=Australian biogeography, climate-dependent diversification and phylogenomics of the spectacular Chamelaucieae tribe (Myrtaceae).;  citation_author=FJ Nge; citation_author=E Biffin; citation_author=BL Rye; citation_author=PG Wilson; citation_author=K van Dijk; citation_author=KR Thiele; citation_author=M Waycott; citation_author=MD Barrett;   citation_volume=38; citation_issue=1;   citation_doi=10.1071/SB24014; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.;  citation_author=L-T Nguyen; citation_author=HA Schmidt; citation_author=A von Haeseler; citation_author=BQ Minh;   citation_volume=32;  citation_firstpage=268; citation_lastpage=274; citation_doi=25371430; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Phylogenomics and morphology of Celmisiinae (Asteraceae: Astereae): taxonomic and evolutionary implications.;  citation_author=DA Nicol; citation_author=P Saldivia; citation_author=TC Summerfield; citation_author=M Heads; citation_author=JM Lord; citation_author=EP Khaing; citation_author=MJ Larcombe;   citation_volume=195;    citation_doi=38508479; ">
			<meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=The performance of coalescent-based species tree estimation methods under models of missing data.;  citation_author=M Nute; citation_author=J Chou; citation_author=EK Molloy; citation_author=T Warnow;   citation_volume=19;    citation_doi=29745854; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=Plastid phylogenomics of the Eriostemon group (Rutaceae; Zanthoxyloideae): support for major clades and investigation of a backbone polytomy.;  citation_author=HK Orel; citation_author=TGB McLay; citation_author=WC Neal; citation_author=PI Forster; citation_author=MJ Bayly;   citation_volume=36;  citation_firstpage=355; citation_lastpage=385; citation_doi=10.1071/SB23011; ">
			<meta name="citation_reference" content="citation_journal_title=Botanical Journal of the Linnean Society; citation_title=Genomic data inform taxonomy and conservation of critically endangered shrubs: a case study of Zieria (Rutaceae) species from eastern Australia.;  citation_author=HK Orel; citation_author=TG McLay; citation_author=LK Guja; citation_author=MF Duretto; citation_author=MJ Bayly;   citation_volume=205;  citation_firstpage=292; citation_lastpage=312; citation_doi=10.1093/botlinnean/boad069; ">
			<meta name="citation_reference" content="citation_journal_title=TAXON.; citation_title=Target capture sequencing clarifies key relationships in the Eriostemon group (Rutaceae: Zanthoxyloideae) and supports a reclassification of Philotheca, including the recognition of two new genera.;  citation_author=HK Orel; citation_author=TGB McLay; citation_author=PI Forster; citation_author=MJ Bayly;       citation_doi=10.1002/tax.13308; ">
			<meta name="citation_reference" content="citation_journal_title=bioRxiv; citation_title=A novel phylogenomics pipeline reveals complex pattern of reticulate evolution in Cucurbitales.;  citation_author=EM Ortiz; citation_author=A H&#xf6;wener; citation_author=G Shigita; citation_author=M Raza; citation_author=O Maurin; citation_author=A Zuntini; citation_author=F Forest; citation_author=WJ Baker; citation_author=H Schaefer;   citation_volume=2023;    citation_doi=10.1101/2023.10.27.564367; ">
			<meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Multiple reproductive barriers separate recently diverged sunflower ecotypes.;  citation_author=KL Ostevik; citation_author=RL Andrew; citation_author=SP Otto; citation_author=LH Rieseberg;   citation_volume=70;  citation_firstpage=2322; citation_lastpage=2335; citation_doi=27479368; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Maps between trees and cladistic analysis of historical associations among genes, organisms, and areas.;  citation_author=RD Page;   citation_volume=43;  citation_firstpage=58; citation_lastpage=77; citation_doi=10.1093/sysbio/43.1.58; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Relationships between gene trees and species trees.;  citation_author=P Pamilo; citation_author=M Nei;   citation_volume=5;  citation_firstpage=568; citation_lastpage=583; citation_doi=3193878; ">
			<meta name="citation_reference" content="citation_journal_title=Plant Physiology; citation_title=Evolution of gene duplication in plants.;  citation_author=N Panchy; citation_author=M Lehti-Shiu; citation_author=S-H Shiu;   citation_volume=171;  citation_firstpage=2294; citation_lastpage=2316; citation_doi=27288366; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Molecular dating of phylogenies by likelihood methods: a comparison of models and a new information criterion.;  citation_author=E Paradis;   citation_volume=67;  citation_firstpage=436; citation_lastpage=444; citation_doi=23454091; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R.;  citation_author=E Paradis; citation_author=K Schliep;   citation_volume=35;  citation_firstpage=526; citation_lastpage=528; citation_doi=30016406; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Ecology Resources; citation_title=A multitiered sequence capture strategy spanning broad evolutionary scales: application for phylogenetic and phylogeographic studies of orchids.;  citation_author=R Peakall; citation_author=DCJ Wong; citation_author=RD Phillips; citation_author=M Ruibal; citation_author=R Eyles; citation_author=C Rodriguez-Delgado; citation_author=CC Linde;   citation_volume=21;  citation_firstpage=1118; citation_lastpage=1140; citation_doi=33453072; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Quartet Sampling distinguishes lack of support from conflicting support in the green plant tree of life.;  citation_author=JB Pease; citation_author=JW Brown; citation_author=JF Walker; citation_author=CE Hinchliff; citation_author=SA Smith;   citation_volume=105;  citation_firstpage=385; citation_lastpage=403; citation_doi=29746719; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Phylogenomics and biogeography of Cunoniaceae (Oxalidales) with complete generic sampling and taxonomic realignments.;  citation_author=Y Pillon; citation_author=HCF Hopkins; citation_author=O Maurin; citation_author=N Epitawalage; citation_author=J Bradford; citation_author=ZS Rogers; citation_author=WJ Baker; citation_author=F Forest;   citation_volume=108;  citation_firstpage=1181; citation_lastpage=1200; citation_doi=34278558; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Constructing phylogenies in the presence of intra-individual site polymorphisms (2ISPs) with a focus on the nuclear ribosomal cistron.;  citation_author=AJ Potts; citation_author=TA Hedderson; citation_author=GW Grimm;   citation_volume=63;  citation_firstpage=1; citation_lastpage=16; citation_doi=23939193; ">
			<meta name="citation_reference" content="Rannala B, Edwards VS, Leaché A, Yang Z (2020) The multi-species coalescent model and species tree inference. In ‘Phylogenetics in the Genomic Era)’. pp. 3.3:1–3.3:21. (Published by the authors, CC BY-NC-ND) Available at">
			<meta name="citation_reference" content="citation_journal_title=BMC Ecology and Evolution; citation_title=Resolving the phylogeny of Thladiantha (Cucurbitaceae) with three different target capture pipelines.;  citation_author=M Raza; citation_author=EM Ortiz; citation_author=L Schwung; citation_author=G Shigita; citation_author=H Schaefer;   citation_volume=23;    citation_doi=38087247; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Plant; citation_title=Widespread whole genome duplications contribute to genome complexity and species diversity in angiosperms.;  citation_author=R Ren; citation_author=H Wang; citation_author=C Guo; citation_author=N Zhang; citation_author=L Zeng; citation_author=Y Chen; citation_author=H Ma; citation_author=J Qi;   citation_volume=11;  citation_firstpage=414; citation_lastpage=428; citation_doi=29317285; ">
			<meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Inferring ancient divergences requires genes with strong phylogenetic signals.;  citation_author=L Salichos; citation_author=A Rokas;   citation_volume=497;  citation_firstpage=327; citation_lastpage=331; citation_doi=23657258; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Estimating absolute rates of molecular evolution and divergence times: a penalized likelihood approach.;  citation_author=MJ Sanderson;   citation_volume=19;  citation_firstpage=101; citation_lastpage=109; citation_doi=11752195; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock.;  citation_author=MJ Sanderson;   citation_volume=19;  citation_firstpage=301; citation_lastpage=302; citation_doi=12538260; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Fast coalescent-based computation of local branch support from quartet frequencies.;  citation_author=E Sayyari; citation_author=S Mirarab;   citation_volume=33;  citation_firstpage=1654; citation_lastpage=1668; citation_doi=27189547; ">
			<meta name="citation_reference" content="citation_journal_title=Genes; citation_title=Testing for polytomies in phylogenetic species trees using quartet frequencies.;  citation_author=E Sayyari; citation_author=S Mirarab;   citation_volume=9;    citation_doi=29495636; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=DiscoVista: interpretable visualizations of gene tree discordance.;  citation_author=E Sayyari; citation_author=JB Whitfield; citation_author=S Mirarab;   citation_volume=122;  citation_firstpage=110; citation_lastpage=115; citation_doi=29421312; ">
			<meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts.;  citation_author=DR Scannell; citation_author=KP Byrne; citation_author=JL Gordon; citation_author=S Wong; citation_author=KH Wolfe;   citation_volume=440;  citation_firstpage=341; citation_lastpage=345; citation_doi=16541074; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=Sequence capture data support the taxonomy of Pogonolepis (Asteraceae: Gnaphalieae) and show unexpected genetic structure.;  citation_author=AN Schmidt-Lebuhn;   citation_volume=35; citation_issue=4; citation_firstpage=317; citation_lastpage=325; citation_doi=10.1071/SB22010; ">
			<meta name="citation_reference" content="citation_journal_title=TAXON; citation_title=Phylogenomic data reveal four major clades of Australian Gnaphalieae (Asteraceae).;  citation_author=AN Schmidt-Lebuhn; citation_author=J Bovill;   citation_volume=70;  citation_firstpage=1020; citation_lastpage=1034; citation_doi=10.1002/tax.12510; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=Transfer of Cotula alpina to the genus Leptinella (Asteraceae: Anthemideae).;  citation_author=AN Schmidt-Lebuhn; citation_author=A Grealy;   citation_volume=37;    citation_doi=10.1071/SB23012; ">
			<meta name="citation_reference" content="citation_journal_title=New Zealand Journal of Botany; citation_title=Genetic data confirm the presence of Senecio madagascariensis in New Zealand.;  citation_author=AN Schmidt-Lebuhn; citation_author=D Egli; citation_author=A Grealy; citation_author=JA Nicholls; citation_author=A Zwick; citation_author=JJ Dymock; citation_author=B Gooden;  citation_publication_date=2024; citation_volume=62;  citation_firstpage=1; citation_lastpage=13; citation_doi=10.1080/0028825X.2022.2148544; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=Elachanthus, Isoetopsis and Kippistia are nested in the genus Minuria (Asteraceae: Astereae).;  citation_author=AN Schmidt-Lebuhn; citation_author=SH Chen; citation_author=A Grealy;  citation_publication_date=2024; citation_volume=37; citation_issue=4;   citation_doi=10.1071/SB23028; ">
			<meta name="citation_reference" content="citation_journal_title=Frontiers in Plant Science; citation_title=Reconstructing the complex evolutionary history of the Papuasian Schefflera radiation through herbariomics.;  citation_author=ZQ Shee; citation_author=DG Frodin; citation_author=R C&#xe1;mara-Leret; citation_author=L Pokorny;   citation_volume=11;    citation_doi=32265950; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Coalescence vs. concatenation: sophisticated analyses vs. first principles applied to rooting the angiosperms.;  citation_author=MP Simmons; citation_author=J Gatesy;   citation_volume=91;  citation_firstpage=98; citation_lastpage=122; citation_doi=26002829; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Collapsing dubiously resolved gene-tree branches in phylogenomic coalescent analyses.;  citation_author=MP Simmons; citation_author=J Gatesy;   citation_volume=158;    citation_doi=33545272; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=The Crinum flaccidum (Amaryllidaceae) species complex in Australia.;  citation_author=J Simpson; citation_author=JG Conran; citation_author=E Biffin; citation_author=K Van Dijk; citation_author=M Waycott;   citation_volume=35;  citation_firstpage=395; citation_lastpage=402; citation_doi=10.1071/SB21038; ">
			<meta name="citation_reference" content="citation_journal_title=Applications in Plant Sciences; citation_title=Lineage‐specific vs. universal: a comparison of the Compositae1061 and Angiosperms353 enrichment panels in the sunflower family.;  citation_author=CM Siniscalchi; citation_author=O Hidalgo; citation_author=L Palazzesi; citation_author=J Pellicer; citation_author=L Pokorny; citation_author=O Maurin; citation_author=IJ Leitch; citation_author=F Forest; citation_author=WJ Baker; citation_author=JR Mandel;   citation_volume=9;    citation_doi=10.1002/aps3.11422; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Phylogeny estimation given sequence length heterogeneity.;  citation_author=V Smirnov; citation_author=T Warnow;   citation_volume=70; citation_issue=2; citation_firstpage=268; citation_lastpage=282; citation_doi=10.1093/sysbio/syaa058; ">
			<meta name="citation_reference" content="citation_journal_title=Trends in Genetics; citation_title=New approaches for inferring phylogenies in the presence of paralogs.;  citation_author=ML Smith; citation_author=MW Hahn;   citation_volume=37;  citation_firstpage=174; citation_lastpage=187; citation_doi=32921510; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=The frequency and topology of pseudoorthologs.;  citation_author=ML Smith; citation_author=MW Hahn;   citation_volume=71;  citation_firstpage=649; citation_lastpage=659; citation_doi=34951639; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=treePL: divergence time estimation using penalized likelihood for large phylogenies.;  citation_author=SA Smith; citation_author=BC O&#x2019;Meara;   citation_volume=28;  citation_firstpage=2689; citation_lastpage=2690; citation_doi=22908216; ">
			<meta name="citation_reference" content="citation_journal_title=BMC Evolutionary Biology; citation_title=Analysis of phylogenomic datasets reveals conflict, concordance, and gene duplications with examples from animals and plants.;  citation_author=SA Smith; citation_author=MJ Moore; citation_author=JW Brown; citation_author=Y Yang;   citation_volume=15;    citation_doi=26239519; ">
			<meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=So many genes, so little time: a practical approach to divergence-time estimation in the genomic era.;  citation_author=SA Smith; citation_author=JW Brown; citation_author=JF Walker;   citation_volume=13;    citation_doi=29772020; ">
			<meta name="citation_reference" content="citation_journal_title=Genome Biology and Evolution; citation_title=Uneven missing data skew phylogenomic relationships within the lories and lorikeets.;  citation_author=BT Smith; citation_author=WM Mauck; citation_author=BW Benz; citation_author=MJ Andersen;   citation_volume=12;  citation_firstpage=1131; citation_lastpage=1147; citation_doi=32470111; ">
			<meta name="citation_reference" content="citation_journal_title=PLoS Genetics; citation_title=Inferring phylogenetic networks with maximum pseudolikelihood under incomplete lineage sorting.;  citation_author=C Sol&#xed;s-Lemus; citation_author=C An&#xe9;;   citation_volume=12;    citation_doi=26950302; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=PhyloNetworks: a package for phylogenetic networks.;  citation_author=C Sol&#xed;s-Lemus; citation_author=P Bastide; citation_author=C An&#xe9;;   citation_volume=34; citation_issue=12; citation_firstpage=3292; citation_lastpage=3298; citation_doi=10.1093/molbev/msx235; ">
			<meta name="citation_reference" content="citation_journal_title=American Journal of Botany; citation_title=Polyploidy and angiosperm diversification.;  citation_author=DE Soltis; citation_author=VA Albert; citation_author=J Leebens-Mack; citation_author=CD Bell; citation_author=AH Paterson; citation_author=C Zheng; citation_author=D Sankoff; citation_author=CW de Pamphilis; citation_author=PK Wall; citation_author=PS Soltis;   citation_volume=96;  citation_firstpage=336; citation_lastpage=348; citation_doi=21628192; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;  citation_author=A Stamatakis;   citation_volume=30;  citation_firstpage=1312; citation_lastpage=1313; citation_doi=24451623; ">
			<meta name="citation_reference" content="citation_journal_title=PLoS Biology; citation_title=ClipKIT: a multiple sequence alignment trimming software for accurate phylogenomic inference.;  citation_author=JL Steenwyk; citation_author=TJ Buida; citation_author=Y Li; citation_author=X-X Shen; citation_author=A Rokas;   citation_volume=18;    citation_doi=33264284; ">
			<meta name="citation_reference" content="citation_journal_title=Nature Reviews Genetics; citation_title=Incongruence in the phylogenomics era.;  citation_author=JL Steenwyk; citation_author=Y Li; citation_author=X Zhou; citation_author=X-X Shen; citation_author=A Rokas;   citation_volume=24;  citation_firstpage=834; citation_lastpage=850; citation_doi=37369847; ">
			<meta name="citation_reference" content="citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_title=Horizontal transfer of chloroplast genomes between plant species.;  citation_author=S Stegemann; citation_author=M Keuthe; citation_author=S Greiner; citation_author=R Bock;   citation_volume=109;  citation_firstpage=2434; citation_lastpage=2438; citation_doi=22308367; ">
			<meta name="citation_reference" content="citation_journal_title=Zenodo; citation_title=Pold87/academic-keyword-occurrence: first release.;  citation_author=V Strobel;   citation_volume=2018;    citation_doi=10.5281/ZENODO.1218409; ">
			<meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=The impact of paralogy on phylogenomic studies – a case study on annelid relationships.;  citation_author=TH Struck;   citation_volume=8;    citation_doi=23667537; ">
			<meta name="citation_reference" content="citation_journal_title=The Plant Journal; citation_title=Deep reticulation: the long legacy of hybridization in vascular plant evolution.;  citation_author=GW Stull; citation_author=KK Pham; citation_author=PS Soltis; citation_author=DE Soltis;   citation_volume=114;  citation_firstpage=743; citation_lastpage=766; citation_doi=36775995; ">
			<meta name="citation_reference" content="citation_journal_title=Biology Letters; citation_title=Adaptive introgression: a plant perspective.;  citation_author=A Suarez-Gonzalez; citation_author=C Lexer; citation_author=QCB Cronk;   citation_volume=14;    citation_doi=29540564; ">
			<meta name="citation_reference" content="Swofford DL, Olsen GJ, Waddell PJ, Hillis DM (1996) Phylogenetic inference. In ‘Molecular Systematics’. (Eds DM Hillis, C Moritz, BK Mable) pp. 407–514. (Sinauer Associates: Sunderland, MA, USA)">
			<meta name="citation_reference" content="citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_title=Estimating divergence times in large molecular phylogenies.;  citation_author=K Tamura; citation_author=FU Battistuzzi; citation_author=P Billing-Ross; citation_author=O Murillo; citation_author=A Filipski; citation_author=S Kumar;   citation_volume=109;  citation_firstpage=19333; citation_lastpage=19338; citation_doi=23129628; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Theoretical foundation of the RelTime method for estimating divergence times from variable evolutionary rates.;  citation_author=K Tamura; citation_author=Q Tao; citation_author=S Kumar;   citation_volume=35;  citation_firstpage=1770; citation_lastpage=1782; citation_doi=29893954; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Phylogenomic analysis of concatenated ultraconserved elements reveals the recent evolutionary radiation of the fairy wrasses (Teleostei: Labridae: Cirrhilabrus).;  citation_author=Y-K Tea; citation_author=X Xu; citation_author=JD DiBattista; citation_author=N Lo; citation_author=PF Cowman; citation_author=SYW Ho;   citation_volume=71;  citation_firstpage=1; citation_lastpage=12; citation_doi=33620490; ">
			<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=PhyloNet: a software package for analyzing and reconstructing reticulate evolutionary relationships.;  citation_author=C Than; citation_author=D Ruths; citation_author=L Nakhleh;   citation_volume=9;    citation_doi=18662388; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=On the need for new measures of phylogenomic support.;  citation_author=RC Thomson; citation_author=JM Brown;   citation_volume=71;  citation_firstpage=917; citation_lastpage=920; citation_doi=35088868; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Gene-tree reconciliation with MUL-trees to resolve polyploidy events.;  citation_author=GWC Thomas; citation_author=SH Ather; citation_author=MW Hahn;   citation_volume=66;  citation_firstpage=1007; citation_lastpage=1018; citation_doi=28419377; ">
			<meta name="citation_reference" content="citation_journal_title=PeerJ; citation_title=CIAlign: a highly customisable command line tool to clean, interpret and visualise multiple sequence alignments.;  citation_author=C Tumescheit; citation_author=AE Firth; citation_author=K Brown;   citation_volume=10;    citation_doi=35310163; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Ecology Resources; citation_title=Utilizing paralogues for phylogenetic reconstruction has the potential to increase species tree support and reduce gene tree discordance in target enrichment data.;  citation_author=R Ufimov; citation_author=JM Gorospe; citation_author=T F&#xe9;r; citation_author=M Kandziora; citation_author=L Salomon; citation_author=M van Loo; citation_author=R Schmickl;   citation_volume=22;  citation_firstpage=3018; citation_lastpage=3034; citation_doi=35796729; ">
			<meta name="citation_reference" content="citation_journal_title=Diversity; citation_title=Phylogenomic insights into the phylogeography of Halophila baillonii Asch.;  citation_author=K Van Dijk; citation_author=M Waycott; citation_author=E Biffin; citation_author=JC Creed; citation_author=FJ Albertazzi; citation_author=J Samper-Villarreal;   citation_volume=15;    citation_doi=10.3390/d15010111; ">
			<meta name="citation_reference" content="citation_journal_title=Applications in Plant Sciences; citation_title=Targeting legume loci: a comparison of three methods for target enrichment bait design in Leguminosae phylogenomics.;  citation_author=M Vatanparast; citation_author=A Powell; citation_author=JJ Doyle; citation_author=AN Egan;   citation_volume=6;    citation_doi=29732266; ">
			<meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Polytomies and the power of phylogenetic inference.;  citation_author=HE Walsh; citation_author=MG Kidd; citation_author=T Moum; citation_author=VL Friesen;   citation_volume=53;  citation_firstpage=932; citation_lastpage=937; citation_doi=28565639; ">
			<meta name="citation_reference" content="citation_journal_title=bioRxiv; citation_title=A hybrid capture RNA bait set for resolving genetic and evolutionary relationships in angiosperms from deep phylogeny to intraspecific lineage hybridization.;  citation_author=M Waycott; citation_author=K Van Dijk; citation_author=E Biffin;   citation_volume=2021;    citation_doi=10.1101/2021.09.06.456727; ">
			<meta name="citation_reference" content="citation_journal_title=PLoS Genetics; citation_title=Bayesian inference of reticulate phylogenies under the multispecies network coalescent.;  citation_author=D Wen; citation_author=Y Yu; citation_author=L Nakhleh;   citation_volume=12;    citation_doi=27144273; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Inferring phylogenetic networks using PhyloNet.;  citation_author=D Wen; citation_author=Y Yu; citation_author=J Zhu; citation_author=L Nakhleh;   citation_volume=67; citation_issue=4; citation_firstpage=735; citation_lastpage=740; citation_doi=10.1093/sysbio/syy015; ">
			<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution; citation_title=Deciphering ancient rapid radiations.;  citation_author=JB Whitfield; citation_author=PJ Lockhart;   citation_volume=22;  citation_firstpage=258; citation_lastpage=265; citation_doi=17300853; ">
			<meta name="citation_reference" content="citation_journal_title=Australian Systematic Botany; citation_title=Phylogenomics of Australian sundews (Drosera: Droseraceae).;  citation_author=L Williamson; citation_author=E Biffin; citation_author=T Hammer; citation_author=K van Dijk; citation_author=J Conran; citation_author=M Waycott;   citation_volume=38;    citation_doi=10.1071/SB24016; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=DISCO: species tree inference using multicopy gene family tree decomposition.;  citation_author=J Willson; citation_author=MS Roddur; citation_author=B Liu; citation_author=P Zaharias; citation_author=T Warnow;   citation_volume=71;  citation_firstpage=610; citation_lastpage=629; citation_doi=34450658; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=The impact of missing data on species tree estimation.;  citation_author=Z Xi; citation_author=L Liu; citation_author=CC Davis;   citation_volume=33;  citation_firstpage=838; citation_lastpage=860; citation_doi=26589995; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Species tree estimation and the impact of gene loss following whole-genome duplication.;  citation_author=H Xiong; citation_author=D Wang; citation_author=C Shao; citation_author=X Yang; citation_author=J Yang; citation_author=T Ma; citation_author=CC Davis; citation_author=L Liu; citation_author=Z Xi;   citation_volume=71;  citation_firstpage=1348; citation_lastpage=1361; citation_doi=35689633; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Species tree inference methods intended to deal with incomplete lineage sorting are robust to the presence of paralogs.;  citation_author=Z Yan; citation_author=ML Smith; citation_author=P Du; citation_author=MW Hahn; citation_author=L Nakhleh;   citation_volume=71;  citation_firstpage=367; citation_lastpage=381; citation_doi=34245291; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=PAML 4: phylogenetic analysis by maximum likelihood.;  citation_author=Z Yang;   citation_volume=24;  citation_firstpage=1586; citation_lastpage=1591; citation_doi=17483113; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Orthology inference in nonmodel organisms using transcriptomes and low-coverage genomes: improving accuracy and matrix occupancy for phylogenomics.;  citation_author=Y Yang; citation_author=SA Smith;   citation_volume=31;  citation_firstpage=3081; citation_lastpage=3092; citation_doi=25158799; ">
			<meta name="citation_reference" content="citation_journal_title=New Phytologist; citation_title=Improved transcriptome sampling pinpoints 26 ancient and more recent polyploidy events in Caryophyllales, including two allopolyploidy events.;  citation_author=Y Yang; citation_author=MJ Moore; citation_author=SF Brockington; citation_author=J Mikenas; citation_author=J Olivieri; citation_author=JF Walker; citation_author=SA Smith;   citation_volume=217;  citation_firstpage=855; citation_lastpage=870; citation_doi=28944472; ">
			<meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=A maximum pseudo-likelihood approach for phylogenetic networks.;  citation_author=Y Yu; citation_author=L Nakhleh;   citation_volume=16;    citation_doi=26450642; ">
			<meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ASTRAL-Pro 2: ultrafast species tree reconstruction from multi-copy gene family trees.;  citation_author=C Zhang; citation_author=S Mirarab;   citation_volume=38;  citation_firstpage=4949; citation_lastpage=4950; citation_doi=36094339; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Bayesian inference of species networks from multilocus sequence data.;  citation_author=C Zhang; citation_author=HA Ogilvie; citation_author=AJ Drummond; citation_author=T Stadler;  citation_publication_date=2018; citation_volume=35;  citation_firstpage=504; citation_lastpage=517; citation_doi=10.1093/molbev/msx307; ">
			<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees.;  citation_author=C Zhang; citation_author=M Rabiee; citation_author=E Sayyari; citation_author=S Mirarab;  citation_publication_date=2018; citation_volume=19;    citation_doi=10.1186/s12859-018-2129-y; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=ASTRAL-Pro: quartet-based species-tree inference despite paralogy.;  citation_author=C Zhang; citation_author=C Scornavacca; citation_author=EK Molloy; citation_author=S Mirarab;   citation_volume=37;  citation_firstpage=3292; citation_lastpage=3307; citation_doi=32886770; ">
			<meta name="citation_reference" content="citation_journal_title=Methods in Ecology and Evolution; citation_title=TAPER: pinpointing errors in multiple sequence alignments despite varying rates of evolution.;  citation_author=C Zhang; citation_author=Y Zhao; citation_author=EL Braun; citation_author=S Mirarab;   citation_volume=12; citation_issue=11; citation_firstpage=2145; citation_lastpage=2158; citation_doi=10.1111/2041-210x.13696; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Phylogenetics and Evolution; citation_title=Phylotranscriptomic analyses reveal deep gene tree discordance in Camellia (Theaceae).;  citation_author=Q Zhang; citation_author=RA Folk; citation_author=Z-Q Mo; citation_author=H Ye; citation_author=Z-Y Zhang; citation_author=H Peng; citation_author=J-L Zhao; citation_author=S-X Yang; citation_author=X-Q Yu;   citation_volume=188;    citation_doi=37648181; ">
			<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution; citation_title=Easy353: a tool to get Angiosperms353 genes for phylogenomic research.;  citation_author=Z Zhang; citation_author=P Xie; citation_author=Y Guo; citation_author=W Zhou; citation_author=E Liu; citation_author=Y Yu;   citation_volume=39;    citation_doi=36458838; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=A new pipeline for removing paralogs in target enrichment data.;  citation_author=W Zhou; citation_author=J Soghigian; citation_author=QJ Xiang;   citation_volume=71;  citation_firstpage=410; citation_lastpage=425; citation_doi=34146111; ">
			<meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=Quartet-based computations of internode certainty provide robust measures of phylogenetic incongruence.;  citation_author=X Zhou; citation_author=S Lutteropp; citation_author=L Czech; citation_author=A Stamatakis; citation_author=MV Looz; citation_author=A Rokas;   citation_volume=69;  citation_firstpage=308; citation_lastpage=324; citation_doi=31504977; ">
			<meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Phylogenomics and the rise of the angiosperms.;  citation_author=A Zuntini; citation_author=T Carruthers; citation_author=O Maurin; citation_author=PC Bailey; citation_author=K Leempoel; citation_author=GE Brewer; citation_author=N Epitawalage; citation_author=E Fran&#xe7;oso; citation_author=B Gallego-Paramo; citation_author=C McGinnie; citation_author=R Negr&#xe3;o; citation_author=SR Roy; citation_author=L Simpson; citation_author=E Toledo Romero; citation_author=VMA Barber; citation_author=L Botigu&#xe9;; citation_author=JJ Clarkson; citation_author=RS Cowan; citation_author=S Dodsworth; citation_author=MG Johnson; citation_author=JT Kim; citation_author=L Pokorny; citation_author=NJ Wickett; citation_author=GM Antar; citation_author=L DeBolt; citation_author=K Gutierrez; citation_author=KP Hendriks; citation_author=A Hoewener; citation_author=A-Q Hu; citation_author=EM Joyce; citation_author=IABS Kikuchi; citation_author=I Larridon; citation_author=DA Larson; citation_author=EJ de L&#xed;rio; citation_author=J-X Liu; citation_author=P Malakasi; citation_author=NAS Przelomska; citation_author=T Shah; citation_author=J Viruel; citation_author=TR Allnutt; citation_author=GK Ameka; citation_author=RL Andrew; citation_author=MS Appelhans; citation_author=M Arista; citation_author=MJ Ariza; citation_author=J Arroyo; citation_author=W Arthan; citation_author=JB Bachelier; citation_author=CD Bailey; citation_author=HF Barnes; citation_author=MD Barrett; citation_author=RL Barrett; citation_author=RJ Bayer; citation_author=MJ Bayly; citation_author=E Biffin; citation_author=N Biggs; citation_author=JL Birch; citation_author=D Bogar&#xed;n; citation_author=R Borosova; citation_author=AMC Bowles; citation_author=PC Boyce; citation_author=GLC Bramley; citation_author=M Briggs; citation_author=L Broadhurst; citation_author=GK Brown; citation_author=JJ Bruhl; citation_author=A Bruneau; citation_author=S Buerki; citation_author=E Burns; citation_author=M Byrne; citation_author=S Cable; citation_author=A Calladine; citation_author=MW Callmander; citation_author=&#xc1; Cano; citation_author=DJ Cantrill; citation_author=WM Cardinal-McTeague; citation_author=MM Carlsen; citation_author=AJA Carruthers; citation_author=A de Castro Mateo; citation_author=MW Chase; citation_author=LW Chatrou; citation_author=M Cheek; citation_author=S Chen; citation_author=MJM Christenhusz; citation_author=P-A Christin; citation_author=MA Clements; citation_author=SC Coffey; citation_author=JG Conran; citation_author=X Cornejo; citation_author=TLP Couvreur; citation_author=ID Cowie; citation_author=L Csiba; citation_author=I Darbyshire; citation_author=G Davidse; citation_author=NMJ Davies; citation_author=AP Davis; citation_author=K van Dijk; citation_author=SR Downie; citation_author=MF Duretto; citation_author=MR Duvall; citation_author=SL Edwards; citation_author=U Eggli; citation_author=RHJ Erkens; citation_author=M Escudero; citation_author=M de la Estrella; citation_author=F Fabriani; citation_author=MF Fay; citation_author=L Ferreira Pde; citation_author=SZ Ficinski; citation_author=RM Fowler; citation_author=S Frisby; citation_author=L Fu; citation_author=T Fulcher; citation_author=M Galbany-Casals; citation_author=EM Gardner; citation_author=DA German; citation_author=A Giaretta; citation_author=M Gibernau; citation_author=LJ Gillespie; citation_author=CC Gonz&#xe1;lez; citation_author=DJ Goyder; citation_author=SW Graham; citation_author=A Grall; citation_author=L Green; citation_author=BF Gunn; citation_author=DG Guti&#xe9;rrez; citation_author=J Hackel; citation_author=T Haevermans; citation_author=A Haigh; citation_author=JC Hall; citation_author=T Hall; citation_author=MJ Harrison; citation_author=SA Hatt; citation_author=O Hidalgo; citation_author=TR Hodkinson; citation_author=GD Holmes; citation_author=HCF Hopkins; citation_author=CJ Jackson; citation_author=SA James; citation_author=RW Jobson; citation_author=G Kadereit; citation_author=IM Kahandawala; citation_author=K Kainulainen; citation_author=M Kato; citation_author=EA Kellogg; citation_author=GJ King; citation_author=B Klejevskaja; citation_author=BB Klitgaard; citation_author=RR Klopper; citation_author=S Knapp; citation_author=MA Koch; citation_author=JH Leebens-Mack; citation_author=F Lens; citation_author=CJ Leon; citation_author=&#xc9; L&#xe9;veill&#xe9;-Bourret; citation_author=GP Lewis; citation_author=D-Z Li; citation_author=L Li; citation_author=S Liede-Schumann; citation_author=T Livshultz; citation_author=D Lorence; citation_author=M Lu; citation_author=P Lu-Irving; citation_author=J Luber; citation_author=EJ Lucas; citation_author=M Luj&#xe1;n; citation_author=M Lum; citation_author=TD Macfarlane; citation_author=C Magdalena; citation_author=VF Mansano; citation_author=LE Masters; citation_author=SJ Mayo; citation_author=K McColl; citation_author=AJ McDonnell; citation_author=AE McDougall; citation_author=TGB McLay; citation_author=H McPherson; citation_author=RI Meneses; citation_author=VSFT Merckx; citation_author=FA Michelangeli; citation_author=JD Mitchell; citation_author=AK Monro; citation_author=MJ Moore; citation_author=TL Mueller; citation_author=K Mummenhoff; citation_author=J Munzinger; citation_author=P Muriel; citation_author=DJ Murphy; citation_author=K Nargar; citation_author=L Nauheimer; citation_author=FJ Nge; citation_author=R Nyffeler; citation_author=A Orejuela; citation_author=EM Ortiz; citation_author=L Palazzesi; citation_author=AL Peixoto; citation_author=SK Pell; citation_author=J Pellicer; citation_author=DS Penneys; citation_author=OA Perez-Escobar; citation_author=C Persson; citation_author=M Pignal; citation_author=Y Pillon; citation_author=JR Pirani; citation_author=GM Plunkett; citation_author=RF Powell; citation_author=GT Prance; citation_author=C Puglisi; citation_author=M Qin; citation_author=RK Rabeler; citation_author=PEJ Rees; citation_author=M Renner; citation_author=EH Roalson; citation_author=M Rodda; citation_author=ZS Rogers; citation_author=S Rokni; citation_author=R Rutishauser; citation_author=MF de Salas; citation_author=H Schaefer; citation_author=RJ Schley; citation_author=A Schmidt-Lebuhn; citation_author=A Shapcott; citation_author=I Al-Shehbaz; citation_author=KA Shepherd; citation_author=MP Simmons; citation_author=AO Sim&#xf5;es; citation_author=A Sim&#xf5;es; citation_author=G Rita; citation_author=M Siros; citation_author=EC Smidt; citation_author=JF Smith; citation_author=N Snow; citation_author=DE Soltis; citation_author=PS Soltis; citation_author=RJ Soreng; citation_author=CA Sothers; citation_author=JR Starr; citation_author=PF Stevens; citation_author=SCK Straub; citation_author=L Struwe; citation_author=JM Taylor; citation_author=IRH Telford; citation_author=AH Thornhill; citation_author=I Tooth; citation_author=A Trias-Blasi; citation_author=F Udovicic; citation_author=TMA Utteridge; citation_author=JC Del Valle; citation_author=GA Verboom; citation_author=HP Vonow; citation_author=MS Vorontsova; citation_author=JM de Vos; citation_author=N Al-Wattar; citation_author=M Waycott; citation_author=CAD Welker; citation_author=AJ White; citation_author=JJ Wieringa; citation_author=LT Williamson; citation_author=TC Wilson; citation_author=SY Wong; citation_author=LA Woods; citation_author=R Woods; citation_author=S Worboys; citation_author=M Xanthos; citation_author=Y Yang; citation_author=Y-X Zhang; citation_author=M-Y Zhou; citation_author=S Zmarzty; citation_author=FO Zuloaga; citation_author=A Antonelli; citation_author=S Bellot; citation_author=DM Crayn; citation_author=OM Grace; citation_author=PJ Kersey; citation_author=IJ Leitch; citation_author=H Sauquet; citation_author=SA Smith; citation_author=WL Eiserhardt; citation_author=F Forest; citation_author=WJ Baker;   citation_volume=629;  citation_firstpage=843; citation_lastpage=850; citation_doi=38658746; ">
			<meta name="citation_abstract" content="Target capture has rapidly become a preferred approach for plant systematic and evolutionary research, marking a step change in the generation of data for phylogenetic inference. Although this advancement has facilitated the resolution of many relationships, phylogenetic conflict continues to be reported and is often attributed to genome duplication, reticulation, incomplete lineage sorting or rapid speciation &#x2013; common processes in plant evolution. The proliferation of methods for analysing target-capture data in the presence of these processes can be overwhelming for many researchers, especially students. In this review, we break down the causes of conflict and guide researchers through a target-capture bioinformatic workflow, with a particular focus on robust phylogenetic inference in the presence of conflict. Through the workflow, we highlight key considerations for reducing artefactual conflict, managing paralogs and assessing conflict, and discuss current methods for investigating causes of conflict. Although we draw from examples in the Australian flora, this review is broadly relevant for any researcher working with target-capture data. We conclude that conflict is often inherent in plant phylogenomic datasets, and, although further methodological development is needed, when conflict is carefully investigated, target-capture data can provide unprecedented insight into the extraordinary evolutionary histories of plants.">

			<meta property="og:site_name" content="CSIRO PUBLISHING"/>
			<meta property="og:type" content="Journal"/>
			<meta property="og:title" content="Navigating phylogenetic conflict and evolutionary inference in plants with target-capture data"/>
			<meta property="og:image" content="https://www.publish.csiro.au/covers/SB_generic.jpg"/>
			<meta property="og:url" content="https://www.publish.csiro.au/sb/SB24011"/>
			<meta property="og:description" content="Target capture has rapidly become a preferred approach for plant systematic and evolutionary research, marking a step change in the generation of data for phylogenetic inference. Although this advancement has facilitated the resolution of many relationships, phylogenetic conflict continues to be reported and is often attributed to genome duplication, reticulation, incomplete lineage sorting or rapid speciation &#x2013; common processes in plant evolution. The proliferation of methods for analysing target-capture data in the presence of these processes can be overwhelming for many researchers, especially students. In this review, we break down the causes of conflict and guide researchers through a target-capture bioinformatic workflow, with a particular focus on robust phylogenetic inference in the presence of conflict. Through the workflow, we highlight key considerations for reducing artefactual conflict, managing paralogs and assessing conflict, and discuss current methods for investigating causes of conflict. Although we draw from examples in the Australian flora, this review is broadly relevant for any researcher working with target-capture data. We conclude that conflict is often inherent in plant phylogenomic datasets, and, although further methodological development is needed, when conflict is carefully investigated, target-capture data can provide unprecedented insight into the extraordinary evolutionary histories of plants."> 
</head>

<body class="not-front journals-page multi-sidebars">
	

	<header>

<div class="row login-register">
	
		<div>
			<a href="/account/register"><i class="fi-pencil"></i> Register</a>
			&nbsp;&nbsp;&nbsp;&nbsp;
			<a href="/account/login"><i class="fi-torso"></i> Login</a>
		</div>
	
</div>

      <div class="row">
        <div class="large-4 small-5 columns logo-wrap">
        <a id="logo" href="/">CSIRO Publishing</a>
        </div><div class="large-8 small-7 columns nav-wrap">
          <nav id="nav-main" class="top-bar okayNav" data-topbar>
             <ul class="menu" tabindex=0>
				<li><a href="/">Home</a></li>
				<li><a href="/books">Books</a></li>
				<li class="active"><a href="/journals">Journals</a></li>				
				<li><a href="/learning">LEARNING</a></li>
				<li><a href="https://blog.publish.csiro.au/">Blog</a></li>
			</ul>
          </nav>
        </div>
      </div>

	</header>

<div class="journal-banner">
	
		<a href="/sb" class="journal-banner-left"><img src="/images/journals/banners/ban_sb_1.png" alt="Australian Systematic Botany" width="600" height="89"></a>
		
			<span class="journal-banner-right"><img src="/images/journals/banners/ban_sb_3.png" alt="Australian Systematic Botany Society" width="400" height="89"></span>
		
			

    </div>

    <div class="tag-line">
      <div class="row">
	      <div class="small-12 columns">
		      Taxonomy, biogeography and evolution of plants 
	      </div>
      </div>
    </div>
	


	<div class="cart-search">
		<div class="row">


<div class="large-9 small-6 shopping-cart columns no-pad-left">
	<a href="/cart"><i class="shopping-cart-icon icon" aria-hidden="true"></i>Shopping Cart: ( empty )</a>
	
</div>
<div class="large-3 small-6 search-wrap columns no-pad-right">
	<form class="search" action="https://www.publish.csiro.au/sb/search" method="get" id="cpgstp" name="cpgstp" title="Search">
		<label class="show-for-sr" for="search-input" >Search our journals</label>
		<input name="q" type="search" id="search-input" placeholder="Search This Journal...">
		<input type="hidden" id="sjournal" name="sjournal" value="on">
		
		<input type="submit" id="btnSearch" value="search" class="button">
	</form>
</div>
		</div>
	</div>
	<div class="breadcrumb-wrap">
		<div class="row">
<div class="breadcrumb small-12 large-6 columns no-pad-left"><span class="">You are here:</span> <a href="/" class="crumb">Home</a> &gt; <a href="/journals" class="crumb">Journals</a> &gt; <a href="/sb" class="crumb">SB</a> &gt; SB24011 </div><div class="brought-to small-12 large-6 columns text-right no-pad-right">

				</div> 
		</div>
	</div>
	<div class="row content-bg collapse">


		<div class="region-main small-12 large-9 large-push-3 medium-9 medium-push-3 columns no-pad">
		    <main id="mainContent">

			    
	<div class="article-top">
		<div class="small-12 large-4 medium-4 no-pad-left columns">
			
				<span><strong>RESEARCH ARTICLE</strong></span> <span style="padding-left:5px;">(Open Access)</span> 
		</div>

		<div class="small-12 large-8 medium-8 text-right columns">

			<a href="/sb/SB24022" class="previous-article">Previous</a>
				
			<a href="/sb/issue/12690" class="contents-vol">
				
						Contents Vol 38(3)
					
			</a>
		</div>
	</div>

			<section class="small-12 large-9 medium-7 no-pad region-main-content columns">
				
				<h1>Navigating phylogenetic conflict and evolutionary inference in plants with target-capture data</h1>
				 <span class="editors"> E. M. Joyce <a href="https://orcid.org/0000-0001-8291-8058" title="https://orcid.org/0000-0001-8291-8058" target="_blank" rel="nofollow noopener noreferrer"><img src="/media/client/orcid_16x16.png" style="vertical-align:baseline;" alt="https://orcid.org/0000-0001-8291-8058"></a></font> <sup>A</sup> <sup>*</sup> , A. N. Schmidt-Lebuhn <a href="https://orcid.org/0000-0002-7402-8941" title="https://orcid.org/0000-0002-7402-8941" target="_blank" rel="nofollow noopener noreferrer"><img src="/media/client/orcid_16x16.png" style="vertical-align:baseline;" alt="https://orcid.org/0000-0002-7402-8941"></a></font> <sup>B</sup> , H. K. Orel <a href="https://orcid.org/0000-0001-7971-8709" title="https://orcid.org/0000-0001-7971-8709" target="_blank" rel="nofollow noopener noreferrer"><img src="/media/client/orcid_16x16.png" style="vertical-align:baseline;" alt="https://orcid.org/0000-0001-7971-8709"></a></font> <sup>C</sup> , F. J. Nge <a href="https://orcid.org/0000-0002-0361-8709" title="https://orcid.org/0000-0002-0361-8709" target="_blank" rel="nofollow noopener noreferrer"><img src="/media/client/orcid_16x16.png" style="vertical-align:baseline;" alt="https://orcid.org/0000-0002-0361-8709"></a></font> <sup>D</sup> , B. M. Anderson <sup>E</sup> , T. A. Hammer <a href="https://orcid.org/0000-0003-3816-7933" title="https://orcid.org/0000-0003-3816-7933" target="_blank" rel="nofollow noopener noreferrer"><img src="/media/client/orcid_16x16.png" style="vertical-align:baseline;" alt="https://orcid.org/0000-0003-3816-7933"></a></font> <sup>F</sup> <sup>G</sup> and T. G. B. McLay <sup>C</sup> <sup>H</sup> <sup>I</sup> <sup>*</sup> </span> 
					 <div class="journal-editors">
					  
					   <span id="show-description">+ Author Affiliations</span>
					  
					  <div id="full-description">
					  
					   <span id="hide-description">- Author Affiliations</span>
					  <br/> <p><sup>A</sup> <institution content-type="university">Systematik, Biodiversität und Evolution der Pflanzen, Ludwig-Maximilians-Universität München</institution>, <addr-line>Menzinger Straße 67</addr-line>, <postal-code>D-80638</postal-code> <city>Munich</city>, <country>Germany</country>. </p>
							<p><sup>B</sup> <institution content-type="university">Centre for Australian National Biodiversity Research (a joint venture of Parks Australia and CSIRO)</institution>, <addr-line>Clunies Ross Street</addr-line>, <city>Canberra</city>, <state>ACT</state> <postal-code>2601</postal-code>, <country>Australia</country>. </p>
							<p><sup>C</sup> <institution content-type="university">School of BioSciences, The University of Melbourne</institution>, <city>Parkville</city>, <state>Vic.</state> <postal-code>3010</postal-code>, <country>Australia</country>. </p>
							<p><sup>D</sup> <institution content-type="university">National Herbarium of New South Wales, Botanic Gardens of Sydney, Locked Bag 6002</institution>, <city>Mount Annan</city>, <state>NSW</state> <postal-code>2567</postal-code>, <country>Australia</country>. </p>
							<p><sup>E</sup> <institution content-type="university">Western Australian Herbarium, Department of Biodiversity, Conservation and Attractions, Locked Bag 104, Bentley Delivery Centre</institution>, <city>Bentley</city>, <state>WA</state> <postal-code>6983</postal-code>, <country>Australia</country>. </p>
							<p><sup>F</sup> <institution content-type="university">School of Biological Sciences, The University of Adelaide</institution>, <city>Adelaide</city>, <state>SA</state> <postal-code>5005</postal-code>, <country>Australia</country>. </p>
							<p><sup>G</sup> <institution content-type="university">State Herbarium of South Australia, Botanic Gardens and State Herbarium</institution>, <addr-line>Hackney Road</addr-line>, <city>Adelaide</city>, <state>SA</state> <postal-code>5000</postal-code>, <country>Australia</country>. </p>
							<p><sup>H</sup> <institution content-type="university">National Biodiversity DNA Library, Environomics, CSIRO</institution>, <city>Parkville</city>, <state>Vic.</state> <postal-code>3010</postal-code>, <country>Australia</country>. </p>
							<p><sup>I</sup> <institution content-type="university">Royal Botanic Gardens Victoria</institution>, <city>Melbourne</city>, <state>Vic.</state> <postal-code>3004</postal-code>, <country>Australia</country>. </p>
							
					</div>
					</div> <div class="Journal_aff"> <div class="Journal_aff"><sup>*</sup> Correspondence to: <a href="mailto:e.joyce@lmu.de">e.joyce@lmu.de</a>, <a href="mailto:todd.mclay@csiro.au">todd.mclay@csiro.au</a></div> <div class="Journal_aff"><br/> Handling Editor: Caroline Puente-Lelievre </div> <br/></div> 
			<div class="journal_abstract">
				<i>Australian Systematic Botany</i> 38, SB24011 https://doi.org/10.1071/SB24011<div class="journal_editors">Submitted: 15 May 2024  Accepted: 7 March 2025  Published: 7 May 2025</div><div class="permissions"><p class="small">&#x000A9; 2025 The Author(s) (or their employer(s)). Published by CSIRO Publishing. This is an open access article distributed under the Creative Commons Attribution-NonCommercial 4.0 International License (<a target="_blank" rel="nofollow noopener noreferrer" href="https://creativecommons.org/licenses/by-nc/4.0/" id="d6e394">CC BY-NC</a>)</p></div><div class="ftlinks" style="margin:1rem 0;">
<div class="small-12 hide-for-medium hide-for-large text-left _columns">
<a href="#ftlinks"><button>Access Article</button></a>
</div>
</div>
<div class="journal-abstract green-item"><h2>Abstract</h2><p class="first" id="d6e404">Target capture has rapidly become a preferred approach for plant systematic and evolutionary research, marking a step change in the generation of data for phylogenetic inference. Although this advancement has facilitated the resolution of many relationships, phylogenetic conflict continues to be reported and is often attributed to genome duplication, reticulation, incomplete lineage sorting or rapid speciation &#x02013; common processes in plant evolution. The proliferation of methods for analysing target-capture data in the presence of these processes can be overwhelming for many researchers, especially students. In this review, we break down the causes of conflict and guide researchers through a target-capture bioinformatic workflow, with a particular focus on robust phylogenetic inference in the presence of conflict. Through the workflow, we highlight key considerations for reducing artefactual conflict, managing paralogs and assessing conflict, and discuss current methods for investigating causes of conflict. Although we draw from examples in the Australian flora, this review is broadly relevant for any researcher working with target-capture data. We conclude that conflict is often inherent in plant phylogenomic datasets, and, although further methodological development is needed, when conflict is carefully investigated, target-capture data can provide unprecedented insight into the extraordinary evolutionary histories of plants.</p></div><div class="journal_abstract"><p><strong>Keywords:</strong> Angiosperms353, deep coalescence, discordance, GAP, Genomics for Australian Plants, hybridisation, HybSeq, incomplete lineage sorting, incongruence, PAFTOL, paralogy, polyploidy, polytomy, target enrichment.</p></div> <div id="article1-back" class="journal-references"><div class="back-section"><a id="d4e5258"><!-- named anchor --></a><div class="section ref-list"><a id="d4e5258"><!-- named anchor --></a><a id="References"><!-- title anchor --></a><h2> References</h2><div class="ref-list table"><div class="row"><div class="ref-content cell"><p><a id="R1"><!-- named anchor --></a><span class="citation"><a id="d4e5265"><!-- named anchor --></a>Aberer AJ, Kobert K, Stamatakis A (2014) <i>ExaBayes</i>: massively parallel bayesian tree inference for the whole-genome era. <i>Molecular Biology and Evolution</i>  <b>31</b>, 2553-2556. <br/> | <a href="https://doi.org/10.1093/molbev/msu236" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5276">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=ExaBayes: massively parallel bayesian tree inference for the whole-genome era." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5279">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/25135941" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5292">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R2"><!-- named anchor --></a><span class="citation"><a id="d4e5299"><!-- named anchor --></a>A&#x000EF;nouche M, Wendel J (2014) Polyploid speciation and genome evolution: lessons from recent allopolyploids. In &#x02018;Evolutionary Biology: Genome Evolution, Speciation, Coevolution and Origin of Life&#x02019;. (Ed. P Pontarotti) pp. 87&#x02013;113. (Springer: Cham, Switzerland) <a href="https://doi.org/10.1007/978-3-319-07623-2_5" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5301">10.1007/978-3-319-07623-2_5</a></span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R3"><!-- named anchor --></a><span class="citation"><a id="d4e5308"><!-- named anchor --></a>Alix K, G&#x000E9;rard PR, Schwarzacher T, Heslop-Harrison J (2017) Polyploidy and interspecific hybridization: partners for adaptation, speciation and evolution in plants. <i>Annals of Botany</i>  <b>120</b>, 183-194. <br/> | <a href="https://doi.org/10.1093/aob/mcx079" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5316">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Polyploidy and interspecific hybridization: partners for adaptation, speciation and evolution in plants." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5319">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/28854567" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5332">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R4"><!-- named anchor --></a><span class="citation"><a id="d4e5339"><!-- named anchor --></a>Almeida-Silva F, Van de Peer Y (2023) Whole-genome duplications and the long-term evolution of gene regulatory networks in angiosperms. <i>Molecular Biology and Evolution</i>  <b>40</b>, msad141. <br/> | <a href="https://doi.org/10.1093/molbev/msad141" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5347">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Whole-genome duplications and the long-term evolution of gene regulatory networks in angiosperms." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5350">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37405949" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5363">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R5"><!-- named anchor --></a><span class="citation"><a id="d4e5371"><!-- named anchor --></a>Andermann T, Cano , Zizka A, Bacon C, Antonelli A (2018) <i>SECAPR</i>&#x02014;a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments. <i>PeerJ</i>  <b>6</b>, e5175. <br/> | <a href="https://doi.org/10.7717/peerj.5175" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5382">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=SECAPR&#x02014;a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5385">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R6"><!-- named anchor --></a><span class="citation"><a id="d4e5402"><!-- named anchor --></a>Andermann T, Torres Jim&#x000E9;nez MF, Matos-Marav&#x000ED; P, Batista R, Blanco-Pastor JL, Gustafsson ALS, Kistler L, Liberal IM, Oxelman B, Bacon CD, Antonelli A (2020) A guide to carrying out a phylogenomic target sequence capture project. <i>Frontiers in Genetics</i>  <b>10</b>, 1407. <br/> | <a href="https://doi.org/10.3389/fgene.2019.01407" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5410">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A guide to carrying out a phylogenomic target sequence capture project." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5413">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32153629" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5426">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R7"><!-- named anchor --></a><span class="citation"><a id="d4e5433"><!-- named anchor --></a>An&#x000E9; C, Larget B, Baum DA, Smith SD, Rokas A (2007) Bayesian estimation of concordance among gene trees. <i>Molecular Biology and Evolution</i>  <b>24</b>, 412-426. <br/> | <a href="https://doi.org/10.1093/molbev/msl170" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5441">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Bayesian estimation of concordance among gene trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5444">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/17095535" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5457">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R8"><!-- named anchor --></a><span class="citation"><a id="d4e5464"><!-- named anchor --></a>Anisimova M, Gascuel O (2006) Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative. <i>Systematic Biology</i>  <b>55</b>, 539-552. <br/> | <a href="https://doi.org/10.1080/10635150600755453" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5472">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5475">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/16785212" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5488">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R9"><!-- named anchor --></a><span class="citation"><a id="d4e5495"><!-- named anchor --></a>Baker WJ, Dodsworth S, Forest F, Graham SW, Johnson MG, McDonnell A, Pokorny L, Tate JA, Wicke S, Wickett NJ (2021) Exploring Angiosperms353: an open, community toolkit for collaborative phylogenomic research on flowering plants. <i>American Journal of Botany</i>  <b>108</b>, 1059-1065. <br/> | <a href="https://doi.org/10.1002/ajb2.1703" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5503">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Exploring Angiosperms353: an open, community toolkit for collaborative phylogenomic research on flowering plants." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5506">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34293179" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5519">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R10"><!-- named anchor --></a><span class="citation"><a id="d4e5526"><!-- named anchor --></a>Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA (2012) <i>SPAdes</i>: a new genome assembly algorithm and its applications to single-cell sequencing. <i>Journal of Computational Biology: A Journal of Computational Molecular Cell Biology</i>  <b>19</b>, 455-477. <br/> | <a href="https://doi.org/10.1089/cmb.2012.0021" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5537">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5540">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/22506599" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5553">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R11"><!-- named anchor --></a><span class="citation"><a id="d4e5561"><!-- named anchor --></a>Barba-Montoya J, Tao Q, Kumar S (2021) Assessing rapid relaxed-clock methods for phylogenomic dating. <i>Genome Biology and Evolution</i>  <b>13</b>, evab251. <br/> | <a href="https://doi.org/10.1093/gbe/evab251" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5569">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Assessing rapid relaxed-clock methods for phylogenomic dating." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5572">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34751377" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5585">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R12"><!-- named anchor --></a><span class="citation"><a id="d4e5592"><!-- named anchor --></a>Baum DA (2007) Concordance trees, concordance factors, and the exploration of reticulate genealogy. <i>TAXON</i>  <b>56</b>, 417-426. <br/> | <a href="https://doi.org/10.1002/tax.562013" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5600">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Concordance trees, concordance factors, and the exploration of reticulate genealogy." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5603">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R13"><!-- named anchor --></a><span class="citation"><a id="d4e5619"><!-- named anchor --></a>Blair C, An&#x000E9; C (2020) Phylogenetic trees and networks can serve as powerful and complementary approaches for analysis of genomic data. <i>Systematic Biology</i>  <b>69</b>, 593-601. <br/> | <a href="https://doi.org/10.1093/sysbio/syz056" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5627">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic trees and networks can serve as powerful and complementary approaches for analysis of genomic data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5630">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/31432090" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5643">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R14"><!-- named anchor --></a><span class="citation"><a id="d4e5650"><!-- named anchor --></a>Blischak PD, Chifman J, Wolfe AD, Kubatko LS (2018) <i>HyDe</i>: a Python package for genome-scale hybridization detection. <i>Systematic Biology</i>  <b>67</b>, 821-829. <br/> | <a href="https://doi.org/10.1093/sysbio/syy023" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5661">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=HyDe: a Python package for genome-scale hybridization detection." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5664">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29562307" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5677">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R15"><!-- named anchor --></a><span class="citation"><a id="d4e5684"><!-- named anchor --></a>Bloesch Z, Nauheimer L, Elias Almeida T, Crayn D, Field AR (2022) <i>HybPhaser</i> identifies hybrid evolution in Australian Thelypteridaceae. <i>Molecular Phylogenetics and Evolution</i>  <b>173</b>, 107526. <br/> | <a href="https://doi.org/10.1016/j.ympev.2022.107526" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5695">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=HybPhaser identifies hybrid evolution in Australian Thelypteridaceae." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5698">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35577295" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5711">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R16"><!-- named anchor --></a><span class="citation"><a id="d4e5718"><!-- named anchor --></a>Bolnick DI (2006) Multi-species outcomes in a common model of sympatric speciation. <i>Journal of Theoretical Biology</i>  <b>241</b>, 734-744. <br/> | <a href="https://doi.org/10.1016/j.jtbi.2006.01.009" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5726">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Multi-species outcomes in a common model of sympatric speciation." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5729">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/16483610" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5742">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R17"><!-- named anchor --></a><span class="citation"><a id="d4e5750"><!-- named anchor --></a>Bomblies K (2020) When everything changes at once: finding a new normal after genome duplication. <i>Proceedings of the Royal Society of London &#x02013; B. Biological Sciences</i>  <b>287</b>, 20202154. <br/> | <a href="https://doi.org/10.1098/rspb.2020.2154" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5758">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=When everything changes at once: finding a new normal after genome duplication." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5761">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33203329" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5774">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R18"><!-- named anchor --></a><span class="citation"><a id="d4e5781"><!-- named anchor --></a>Borowiec ML (2016) <i>AMAS</i>: a fast tool for alignment manipulation and computing of summary statistics. <i>PeerJ</i>  <b>4</b>, e1660. <br/> | <a href="https://doi.org/10.7717/peerj.1660" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5792">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=AMAS: a fast tool for alignment manipulation and computing of summary statistics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5795">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26835189" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5808">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R19"><!-- named anchor --></a><span class="citation"><a id="d4e5815"><!-- named anchor --></a>Bouckaert R, Heled J, K&#x000FC;hnert D, Vaughan T, Wu C-H, Xie D, Suchard MA, Rambaut A, Drummond AJ (2014) <i>BEAST 2</i>: a software platform for Bayesian evolutionary analysis. <i>PLoS Computational Biology</i>  <b>10</b>, e1003537. <br/> | <a href="https://doi.org/10.1371/journal.pcbi.1003537" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5826">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=BEAST 2: a software platform for Bayesian evolutionary analysis." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5829">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/24722319" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5842">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R20"><!-- named anchor --></a><span class="citation"><a id="d4e5849"><!-- named anchor --></a>Boussau B, Scornavacca C (2020) Reconciling gene trees with species trees. In &#x02018;Phylogenetics in the Genomic Era&#x02019;. (Eds C Scornavacca, F Delsuc, N Galtier) pp. 3.2:1&#x02013;3.2:23. (Published by the authors, CC BY-NC-ND) Available at <a href="https://hal.science/hal-02535529" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5851">https://hal.science/hal-02535529</a></span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R21"><!-- named anchor --></a><span class="citation"><a id="d4e5858"><!-- named anchor --></a>Bradican JP, Tomasello S, Boscutti F, Karbstein K, H&#x000F6;randl E (2023) Phylogenomics of southern European taxa in the <i>Ranunculus auricomus</i> species complex: the apple doesn&#x02019;t fall far from the tree. <i>Plants</i>  <b>12</b>, 3664. <br/> | <a href="https://doi.org/10.3390/plants12213664" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5869">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics of southern European taxa in the Ranunculus auricomus species complex: the apple doesn&#x02019;t fall far from the tree." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5872">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37960021" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5885">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R22"><!-- named anchor --></a><span class="citation"><a id="d4e5892"><!-- named anchor --></a>Breinholt JW, Carey SB, Tiley GP, Davis EC, Endara L, McDaniel SF, Neves LG, Sessa EB, Von Konrat M, Chantanaorrapint S, Fawcett S, Ickert-Bond SM, Labiak PH, Larra&#x000ED;n J, Lehnert M, Lewis LR, Nagalingum NS, Patel N, Rensing SA, Testo W, Vasco A, Villarreal JC, Williams EW, Burleigh JG (2021) A target enrichment probe set for resolving the flagellate land plant tree of life. <i>Applications in Plant Sciences</i>  <b>9</b>, e11406. <br/> | <a href="https://doi.org/10.1002/aps3.11406" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5900">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A target enrichment probe set for resolving the flagellate land plant tree of life." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5903">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33552748" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5916">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R23"><!-- named anchor --></a><span class="citation"><a id="d4e5924"><!-- named anchor --></a>Bryant D, Moulton V (2004) Neighbor-Net: an agglomerative method for the construction of phylogenetic networks. <i>Molecular Biology and Evolution</i>  <b>21</b>, 255-265. <br/> | <a href="https://doi.org/10.1093/molbev/msh018" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5932">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Neighbor-Net: an agglomerative method for the construction of phylogenetic networks." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5935">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/14660700" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5948">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R24"><!-- named anchor --></a><span class="citation"><a id="d4e5955"><!-- named anchor --></a>Cai L, Xi Z, Lemmon EM, Lemmon AR, Mast A, Buddenhagen CE, Liu L, Davis CC (2021) The perfect storm: gene tree estimation error, incomplete lineage sorting, and ancient gene flow explain the most recalcitrant ancient angiosperm clade, Malpighiales. <i>Systematic Biology</i>  <b>70</b>, 491-507. <br/> | <a href="https://doi.org/10.1093/sysbio/syaa083" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5963">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The perfect storm: gene tree estimation error, incomplete lineage sorting, and ancient gene flow explain the most recalcitrant ancient angiosperm clade, Malpighiales." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5966">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33169797" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5979">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R25"><!-- named anchor --></a><span class="citation"><a id="d4e5986"><!-- named anchor --></a>Capella-Guti&#x000E9;rrez S, Silla-Mart&#x000ED;nez JM, Gabald&#x000F3;n T (2009) <i>trimAl</i>: a tool for automated alignment trimming in large-scale phylogenetic analyses. <i>Bioinformatics</i>  <b>25</b>, 1972-1973. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btp348" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e5997">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6000">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/19505945" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6013">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R26"><!-- named anchor --></a><span class="citation"><a id="d4e6020"><!-- named anchor --></a>Cardillo M, Weston PH, Reynolds ZKM, Olde PM, Mast AR, Lemmon EM, Lemmon AR, Bromham L (2017) The phylogeny and biogeography of <i>Hakea</i> (Proteaceae) reveals the role of biome shifts in a continental plant radiation. <i>Evolution</i>  <b>71</b>, 1928-1943. <br/> | <a href="https://doi.org/10.1111/evo.13276" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6031">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The phylogeny and biogeography of Hakea (Proteaceae) reveals the role of biome shifts in a continental plant radiation." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6034">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/28548206" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6047">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R27"><!-- named anchor --></a><span class="citation"><a id="d4e6054"><!-- named anchor --></a>Chaudhary R, Bansal MS, Wehe A, Fern&#x000E1;ndez-Baca D, Eulenstein O (2010) <i>iGTP</i>: a software package for large-scale gene tree parsimony analysis. <i>BMC Bioinformatics</i>  <b>11</b>, 574. <br/> | <a href="https://doi.org/10.1186/1471-2105-11-574" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6065">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=iGTP: a software package for large-scale gene tree parsimony analysis." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6068">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/21092314" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6081">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R28"><!-- named anchor --></a><span class="citation"><a id="d4e6088"><!-- named anchor --></a>Clark JW, Donoghue PCJ (2018) Whole-genome duplication and plant macroevolution. <i>Trends in Plant Science</i>  <b>23</b>, 933-945. <br/> | <a href="https://doi.org/10.1016/j.tplants.2018.07.006" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6096">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Whole-genome duplication and plant macroevolution." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6099">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/30122372" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6112">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R29"><!-- named anchor --></a><span class="citation"><a id="d4e6120"><!-- named anchor --></a>Comte A, Tricou T, Tannier E, Joseph J, Siberchicot A, Penel S, Allio R, Delsuc F, Dray S, De Vienne DM (2023) <i>PhylteR</i>: efficient identification of outlier sequences in phylogenomic datasets. <i>Molecular Biology and Evolution</i>  <b>40</b>, msad234. <br/> | <a href="https://doi.org/10.1093/molbev/msad234" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6131">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=PhylteR: efficient identification of outlier sequences in phylogenomic datasets." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6134">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37879113" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6147">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R30"><!-- named anchor --></a><span class="citation"><a id="d4e6154"><!-- named anchor --></a>Cooper WE, Crayn DM, Joyce EM (2023) <i>Aglaia fellii</i> W.E.Cooper &#x00026; Joyce (Meliaceae), a new species for Cape York Peninsula. <i>Australian Journal of Taxonomy</i>  <b>16</b>, 1-9. <br/> | <a href="https://doi.org/10.54102/ajt.p8to6" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6165">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Aglaia fellii W.E.Cooper &#x00026; Joyce (Meliaceae), a new species for Cape York Peninsula." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6168">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R31"><!-- named anchor --></a><span class="citation"><a id="d4e6185"><!-- named anchor --></a>Crisp MD, Minh BQ, Choi B, Edwards RD, Hereward J, Kulheim C, Lin YP, Meusemann K, Thornhill AH, Toon A, Cook LG (2024) Perianth evolution and implications for generic delimitation in the eucalypts (Myrtaceae), including the description of the new genus, <i>Blakella</i>. <i>Journal of Systematics and Evolution jse</i>  <b>62</b>, 942-962. <br/> | <a href="https://doi.org/10.1111/jse.13047" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6196">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Perianth evolution and implications for generic delimitation in the eucalypts (Myrtaceae), including the description of the new genus, Blakella." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6199">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R32"><!-- named anchor --></a><span class="citation"><a id="d4e6216"><!-- named anchor --></a>De Bodt S, Maere S, Van de Peer Y (2005) Genome duplication and the origin of angiosperms. <i>Trends in Ecology &#x00026; Evolution</i>  <b>20</b>, 591-597. <br/> | <a href="https://doi.org/10.1016/j.tree.2005.07.008" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6224">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Genome duplication and the origin of angiosperms." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6227">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/16701441" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6240">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R33"><!-- named anchor --></a><span class="citation"><a id="d4e6247"><!-- named anchor --></a>del Pozo JC, Ramirez-Parra E (2015) Whole genome duplications in plants: an overview from <i>Arabidopsis</i>. <i>Journal of Experimental Botany</i>  <b>66</b>, 6991-7003. <br/> | <a href="https://doi.org/10.1093/jxb/erv432" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6258">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Whole genome duplications in plants: an overview from Arabidopsis." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6261">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26417017" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6274">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R34"><!-- named anchor --></a><span class="citation"><a id="d4e6281"><!-- named anchor --></a>DeSalle R, Absher R, Amato G (1994) Speciation and phylogenetic resolution. <i>Trends in Ecology &#x00026; Evolution</i>  <b>9</b>, 297-298. <br/> | <a href="https://doi.org/10.1016/0169-5347(94)90034-5" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6289">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Speciation and phylogenetic resolution." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6292">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/21236859" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6305">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R35"><!-- named anchor --></a><span class="citation"><a id="d4e6313"><!-- named anchor --></a>Dillenberger MS, Kadereit JW (2017) Simultaneous speciation in the European high mountain flowering plant genus <i>Facchinia</i> (<i>Minuartia</i> sl, Caryophyllaceae) revealed by genotyping-by-sequencing. <i>Molecular Phylogenetics and Evolution</i>  <b>112</b>, 23-35. <br/> | <a href="https://doi.org/10.1016/j.ympev.2017.04.016" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6327">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Simultaneous speciation in the European high mountain flowering plant genus Facchinia (Minuartia sl, Caryophyllaceae) revealed by genotyping-by-sequencing." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6331">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/28433621" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6343">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R36"><!-- named anchor --></a><span class="citation"><a id="d4e6350"><!-- named anchor --></a>Douglas J, Jim&#x000E9;nez-Silva CL, Bouckaert R (2022) <i>StarBeast3</i>: adaptive parallelized Bayesian inference under the multispecies coalescent. <i>Systematic Biology</i>  <b>71</b>, 901-916. <br/> | <a href="https://doi.org/10.1093/sysbio/syac010" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6361">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=StarBeast3: adaptive parallelized Bayesian inference under the multispecies coalescent." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6364">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35176772" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6377">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R37"><!-- named anchor --></a><span class="citation"><a id="d4e6384"><!-- named anchor --></a>Edelman NB, Mallet J (2021) Prevalence and adaptive impact of introgression. <i>Annual Review of Genetics</i>  <b>55</b>, 265-283. <br/> | <a href="https://doi.org/10.1146/annurev-genet-021821-020805" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6392">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Prevalence and adaptive impact of introgression." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6395">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34579539" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6408">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R38"><!-- named anchor --></a><span class="citation"><a id="d4e6415"><!-- named anchor --></a>Edelman NB, Frandsen PB, Miyagi M, Clavijo B, Davey J, Dikow RB, Garc&#x000ED;a-Accinelli G, Van Belleghem SM, Patterson N, Neafsey DE, Challis R, Kumar S, Moreira GRP, Salazar C, Chouteau M, Counterman BA, Papa R, Blaxter M, Reed RD, Dasmahapatra KK, Kronforst M, Joron M, Jiggins CD, McMillan WO, Di Palma F, Blumberg AJ, Wakeley J, Jaffe D, Mallet J (2019) Genomic architecture and introgression shape a butterfly radiation. <i>Science</i>  <b>366</b>, 594-599. <br/> | <a href="https://doi.org/10.1126/science.aaw2090" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6423">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Genomic architecture and introgression shape a butterfly radiation." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6426">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/31672890" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6439">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R39"><!-- named anchor --></a><span class="citation"><a id="d4e6446"><!-- named anchor --></a>Faircloth BC (2016) <i>PHYLUCE</i> is a software package for the analysis of conserved genomic loci. <i>Bioinformatics</i>  <b>32</b>, 786-788. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btv646" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6457">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=PHYLUCE is a software package for the analysis of conserved genomic loci." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6460">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26530724" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6473">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R40"><!-- named anchor --></a><span class="citation"><a id="d4e6480"><!-- named anchor --></a>F&#x000E9;r T, Schmickl RE (2018) <i>HybPhyloMaker</i>: target enrichment data analysis from raw reads to species trees. <i>Evolutionary Bioinformatics Online</i>  <b>14</b>, 1176934317742613. <br/> | <a href="https://doi.org/10.1177/1176934317742613" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6491">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=HybPhyloMaker: target enrichment data analysis from raw reads to species trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6494">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29348708" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6507">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R41"><!-- named anchor --></a><span class="citation"><a id="d4e6515"><!-- named anchor --></a>Flagel LE, Wendel JF (2009) Gene duplication and evolutionary novelty in plants. <i>The New Phytologist</i>  <b>183</b>, 557-564. <br/> | <a href="https://doi.org/10.1111/j.1469-8137.2009.02923.x" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6523">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Gene duplication and evolutionary novelty in plants." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6526">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/19555435" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6539">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R42"><!-- named anchor --></a><span class="citation"><a id="d4e6546"><!-- named anchor --></a>Fowler RM, McLay TGB, Schuster TM, Buirchell BJ, Murphy DJ, Bayly MJ (2020) Plastid phylogenomic analysis of tribe Myoporeae (Scrophulariaceae). <i>Plant Systematics and Evolution</i>  <b>306</b>, 52. <br/> | <a href="https://doi.org/10.1007/s00606-020-01678-4" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6554">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Plastid phylogenomic analysis of tribe Myoporeae (Scrophulariaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6557">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R43"><!-- named anchor --></a><span class="citation"><a id="d4e6573"><!-- named anchor --></a>Frankel LE, An&#x000E9; C (2023<i><i>a</i></i>) Summary tests of introgression are highly sensitive to rate variation across lineages. <i>bioRxiv</i>  <b>2023</b>, 2023.01.26.525396 [Preprint, published 26 January 2023]. <br/> | <a href="https://doi.org/10.1101/2023.01.26.525396" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6587">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Summary tests of introgression are highly sensitive to rate variation across lineages." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6590">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R44"><!-- named anchor --></a><span class="citation"><a id="d4e6607"><!-- named anchor --></a>Frankel LE, An&#x000E9; C (2023<i><i>b</i></i>) Summary tests of introgression are highly sensitive to rate variation across lineages. <i>Systematic Biology</i>  <b>72</b>(6), 1357-1369. <br/> | <a href="https://doi.org/10.1093/sysbio/syad056" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6621">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Summary tests of introgression are highly sensitive to rate variation across lineages." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6624">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37698548" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6637">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R45"><!-- named anchor --></a><span class="citation"><a id="d4e6644"><!-- named anchor --></a>Freyman WA, Johnson MG, Rothfels CJ (2023) <i>homologizer</i>: phylogenetic phasing of gene copies into polyploid subgenomes. <i>Methods in Ecology and Evolution</i>  <b>14</b>, 1230-1244. <br/> | <a href="https://doi.org/10.1111/2041-210X.14072" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6655">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=homologizer: phylogenetic phasing of gene copies into polyploid subgenomes." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6658">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R46"><!-- named anchor --></a><span class="citation"><a id="d4e6675"><!-- named anchor --></a>Frost LA, Bedoya AM, Lagomarsino LP (2024) Artifactual orthologs and the need for diligent data exploration in complex phylogenomic datasets: a museomic case study from the Andean Flora. <i>Systematic Biology</i>  <b>73</b>, 308-322. <br/> | <a href="https://doi.org/10.1093/sysbio/syad076" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6683">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Artifactual orthologs and the need for diligent data exploration in complex phylogenomic datasets: a museomic case study from the Andean Flora." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6686">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/38170162" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6699">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R47"><!-- named anchor --></a><span class="citation"><a id="d4e6707"><!-- named anchor --></a>Galtier N (2024) An approximate likelihood method reveals ancient gene flow between human, chimpanzee and gorilla. <i>Peer Community Journal</i>  <b>4</b>, e3. <br/> | <a href="https://doi.org/10.24072/pcjournal.359" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6715">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=An approximate likelihood method reveals ancient gene flow between human, chimpanzee and gorilla." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6718">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R48"><!-- named anchor --></a><span class="citation"><a id="d4e6734"><!-- named anchor --></a>Goodman M, Czelusniak J, Moore GW, Romero-Herrera AE, Matsuda G (1979) Fitting the gene lineage into its species lineage, a parsimony strategy illustrated by cladograms constructed from globin sequences. <i>Systematic Biology</i>  <b>28</b>, 132-163. <br/> | <a href="https://doi.org/10.1093/sysbio/28.2.132" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6742">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Fitting the gene lineage into its species lineage, a parsimony strategy illustrated by cladograms constructed from globin sequences." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6745">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R49"><!-- named anchor --></a><span class="citation"><a id="d4e6761"><!-- named anchor --></a>Gunn BF, Murphy DJ, Walsh NG, Conran JG, Pires JC, Macfarlane TD, Birch JL (2020) Evolution of Lomandroideae: multiple origins of polyploidy and biome occupancy in Australia. <i>Molecular Phylogenetics and Evolution</i>  <b>149</b>, 106836. <br/> | <a href="https://doi.org/10.1016/j.ympev.2020.106836" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6769">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Evolution of Lomandroideae: multiple origins of polyploidy and biome occupancy in Australia." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6772">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32304826" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6785">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R50"><!-- named anchor --></a><span class="citation"><a id="d4e6792"><!-- named anchor --></a>Gunn BF, Murphy DJ, Walsh NG, Conran JG, Pires JC, Macfarlane TD, Crisp MD, Cook LG, Birch JL (2024) Genomic data resolve phylogenetic relationships of Australian mat-rushes, <i>Lomandra</i> (Asparagaceae: Lomandroideae). <i>Botanical Journal of the Linnean Society</i>  <b>204</b>, 1-22. <br/> | <a href="https://doi.org/10.1093/botlinnean/boad034" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6803">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Genomic data resolve phylogenetic relationships of Australian mat-rushes, Lomandra (Asparagaceae: Lomandroideae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6806">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R51"><!-- named anchor --></a><span class="citation"><a id="d4e6823"><!-- named anchor --></a>Hammer TA, Biffin E, van Dijk K, Thiele KR, Waycott M (2025) A framework phylogeny of the diverse guinea-flowers (<i>Hibbertia</i>, Dilleniaceae) using high-throughput sequence data. <i>Australian Systematic Botany</i>  <b>38</b>(2), SB24009. <br/> | <a href="https://doi.org/10.1071/SB24009" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6834">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A framework phylogeny of the diverse guinea-flowers (Hibbertia, Dilleniaceae) using high-throughput sequence data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6837">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R52"><!-- named anchor --></a><span class="citation"><a id="d4e6854"><!-- named anchor --></a>Hancock LP, Obbens F, Moore AJ, Thiele K, De Vos JM, West J, Holtum JAM, Edwards EJ (2018) Phylogeny, evolution, and biogeographic history of <i>Calandrinia</i> (Montiaceae). <i>American Journal of Botany</i>  <b>105</b>, 1021-1034. <br/> | <a href="https://doi.org/10.1002/ajb2.1110" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6865">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogeny, evolution, and biogeographic history of Calandrinia (Montiaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6868">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29995314" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6881">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R53"><!-- named anchor --></a><span class="citation"><a id="d4e6889"><!-- named anchor --></a>Handika H, Esselstyn JA (2024) <i>SEGUL</i>: ultrafast, memory&#x02010;efficient and mobile&#x02010;friendly software for manipulating and summarizing phylogenomic datasets. <i>Molecular Ecology Resources</i>  <b>24</b>(7), e13964. <br/> | <a href="https://doi.org/10.1111/1755-0998.13964" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6900">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=SEGUL: ultrafast, memory&#x02010;efficient and mobile&#x02010;friendly software for manipulating and summarizing phylogenomic datasets." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6903">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R54"><!-- named anchor --></a><span class="citation"><a id="d4e6920"><!-- named anchor --></a>Hart ML, Forrest LL, Nicholls JA, Kidner CA (2016) Retrieval of hundreds of nuclear loci from herbarium specimens. <i>TAXON</i>  <b>65</b>, 1081-1092. <br/> | <a href="https://doi.org/10.12705/655.9" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6928">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Retrieval of hundreds of nuclear loci from herbarium specimens." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6931">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R55"><!-- named anchor --></a><span class="citation"><a id="d4e6947"><!-- named anchor --></a>Hendriks KP, Kiefer C, Al-Shehbaz IA, Bailey CD, Hooft van Huysduynen A, Nikolov LA, Nauheimer L, Zuntini AR, German DA, Franzke A, Koch MA, Lysak MA, Toro-N&#x000FA;&#x000F1;ez &#x000D3;, &#x000D6;z&#x000FC;do&#x0011F;ru B, Invern&#x000F3;n VR, Walden N, Maurin O, Hay NM, Shushkov P, Mand&#x000E1;kov&#x000E1; T, Schranz ME, Thulin M, Windham MD, Re&#x00161;etnik I, &#x00160;paniel S, Ly E, Pires JC, Harkess A, Neuffer B, Vogt R, Br&#x000E4;uchler C, Rainer H, Janssens SB, Schmull M, Forrest A, Guggisberg A, Zmarzty S, Lepschi BJ, Scarlett N, Stauffer FW, Sch&#x000F6;nberger I, Heenan P, Baker WJ, Forest F, Mummenhoff K, Lens F (2023) Global Brassicaceae phylogeny based on filtering of 1,000-gene dataset. <i>Current Biology</i>  <b>33</b>, 4052-4068. <br/> | <a href="https://doi.org/10.1016/j.cub.2023.08.026" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6955">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Global Brassicaceae phylogeny based on filtering of 1,000-gene dataset." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6958">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37659415" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6971">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R56"><!-- named anchor --></a><span class="citation"><a id="d4e6978"><!-- named anchor --></a>Hibbins MS, Hahn MW (2022) Phylogenomic approaches to detecting and characterizing introgression. <i>Genetics</i>  <b>220</b>, iyab173. <br/> | <a href="https://doi.org/10.1093/genetics/iyab173" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6986">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic approaches to detecting and characterizing introgression." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e6989">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34788444" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7002">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R57"><!-- named anchor --></a><span class="citation"><a id="d4e7009"><!-- named anchor --></a>Hoelzer GA, Meinick DJ (1994) Patterns of speciation and limits to phylogenetic resolution. <i>Trends in Ecology &#x00026; Evolution</i>  <b>9</b>, 104-107. <br/> | <a href="https://doi.org/10.1016/0169-5347(94)90207-0" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7017">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Patterns of speciation and limits to phylogenetic resolution." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7020">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/21236789" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7033">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R58"><!-- named anchor --></a><span class="citation"><a id="d4e7040"><!-- named anchor --></a>Huson DH (1998) <i>SplitsTree</i>: analyzing and visualizing evolutionary data. <i>Bioinformatics</i>  <b>14</b>, 68-73. <br/> | <a href="https://doi.org/10.1093/bioinformatics/14.1.68" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7051">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=SplitsTree: analyzing and visualizing evolutionary data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7054">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/9520503" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7067">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R59"><!-- named anchor --></a><span class="citation"><a id="d4e7075"><!-- named anchor --></a>Huson DH, Bryant D (2006) Application of phylogenetic networks in evolutionary studies. <i>Molecular Biology and Evolution</i>  <b>23</b>(2), 254-267. <br/> | <a href="https://doi.org/10.1093/molbev/msj030" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7083">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Application of phylogenetic networks in evolutionary studies." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7086">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R60"><!-- named anchor --></a><span class="citation"><a id="d4e7102"><!-- named anchor --></a>Jackson C, McLay T, Schmidt-Lebuhn AN (2023) <i>hybpiper-nf</i> and <i>paragone-nf</i>: containerization and additional options for target capture assembly and paralog resolution. <i>Applications in Plant Sciences</i>  <b>11</b>, e11532. <br/> | <a href="https://doi.org/10.1002/aps3.11532" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7116">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=hybpiper-nf and paragone-nf: containerization and additional options for target capture assembly and paralog resolution." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7120">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37601313" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7132">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R61"><!-- named anchor --></a><span class="citation"><a id="d4e7139"><!-- named anchor --></a>Johnson MG, Gardner EM, Liu Y, Medina R, Goffinet B, Shaw AJ, Zerega NJ, Wickett NJ (2016) <i>HybPiper</i>: extracting coding sequence and introns for phylogenetics from high-throughput sequencing reads using target enrichment. <i>Applications in Plant Sciences</i>  <b>4</b>, 1600016. <br/> | <a href="https://doi.org/10.3732/apps.1600016" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7150">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=HybPiper: extracting coding sequence and introns for phylogenetics from high-throughput sequencing reads using target enrichment." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7153">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/27437175" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7166">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R62"><!-- named anchor --></a><span class="citation"><a id="d4e7173"><!-- named anchor --></a>Johnson MG, Pokorny L, Dodsworth S, Botigu&#x000E9; LR, Cowan RS, Devault A, Eiserhardt WL, Epitawalage N, Forest F, Kim JT, Leebens-Mack JH, Leitch IJ, Maurin O, Soltis DE, Soltis PS, Wong GK, Baker WJ, Wickett NJ (2019) A universal probe set for targeted sequencing of 353 nuclear genes from any flowering plant designed using <i>k</i>-medoids clustering. <i>Systematic Biology</i>  <b>68</b>, 594-606. <br/> | <a href="https://doi.org/10.1093/sysbio/syy086" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7184">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A universal probe set for targeted sequencing of 353 nuclear genes from any flowering plant designed using k-medoids clustering." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7187">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/30535394" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7200">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R63"><!-- named anchor --></a><span class="citation"><a id="d4e7207"><!-- named anchor --></a>Joly S (2012) <i>JML</i>: testing hybridization from species trees. <i>Molecular Ecology Resources</i>  <b>12</b>, 179-184. <br/> | <a href="https://doi.org/10.1111/j.1755-0998.2011.03065.x" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7218">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=JML: testing hybridization from species trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7221">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/21899723" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7234">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R64"><!-- named anchor --></a><span class="citation"><a id="d4e7241"><!-- named anchor --></a>Joly S, McLenachan PA, Lockhart PJ (2009) A statistical approach for distinguishing hybridization and incomplete lineage sorting. <i>The American Naturalist</i>  <b>174</b>, 54-70. <br/> | <a href="https://doi.org/10.1086/600082" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7249">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A statistical approach for distinguishing hybridization and incomplete lineage sorting." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7252">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/19519219" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7265">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R65"><!-- named anchor --></a><span class="citation"><a id="d4e7273"><!-- named anchor --></a>Joyce EM, Appelhans MS, Buerki S, Cheek M, de Vos JM, Pirani JR, Zuntini AR, Bachelier JB, Bayly MJ, Callmander MW, Devecchi MF, Pell SK, Groppo M, Lowry PP, Mitchell J, Siniscalchi CM, Munzinger J, Orel HK, Pannell CM, Nauheimer L, Sauquet H, Weeks A, Muellner-Riehl AN, Leitch IJ, Maurin O, Forest F, Nargar K, Thiele KR, Baker WJ, Crayn DM (2023) Phylogenomic analyses of Sapindales support new family relationships, rapid Mid-Cretaceous Hothouse diversification, and heterogeneous histories of gene duplication. <i>Frontiers in Plant Science</i>  <b>14</b>, 1063174. <br/> | <a href="https://doi.org/10.3389/fpls.2023.1063174" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7281">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic analyses of Sapindales support new family relationships, rapid Mid-Cretaceous Hothouse diversification, and heterogeneous histories of gene duplication." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7284">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/36959945" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7297">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R66"><!-- named anchor --></a><span class="citation"><a id="d4e7304"><!-- named anchor --></a>Joyce EM, Schmidt-Lebuhn AN, Orel HK, Nge FJ, Anderson BM, Hammer TA, McLay TGB (2024) Navigating phylogenetic conflict and evolutionary inference in plants with target capture data. <i>EcoEvoRxiv</i>  <b>2024</b>, Version 2 [Preprint, published 27 May 2024, updated 7 March 2025]. <br/> | <a href="https://doi.org/10.32942/X2WP6V" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7312">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Navigating phylogenetic conflict and evolutionary inference in plants with target capture data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7315">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R67"><!-- named anchor --></a><span class="citation"><a id="d4e7331"><!-- named anchor --></a>Karimi N, Grover CE, Gallagher JP, Wendel JF, An&#x000E9; C, Baum DA (2020) Reticulate evolution helps explain apparent homoplasy in floral biology and pollination in baobabs (<i>Adansonia</i>; Bombacoideae; Malvaceae). <i>Systematic Biology</i>  <b>69</b>, 462-478. <br/> | <a href="https://doi.org/10.1093/sysbio/syz073" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7342">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Reticulate evolution helps explain apparent homoplasy in floral biology and pollination in baobabs (Adansonia; Bombacoideae; Malvaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7345">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/31693158" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7358">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R68"><!-- named anchor --></a><span class="citation"><a id="d4e7365"><!-- named anchor --></a>Kates HR, Johnson MG, Gardner EM, Zerega NJC, Wickett NJ (2018) Allele phasing has minimal impact on phylogenetic reconstruction from targeted nuclear gene sequences in a case study of <i>Artocarpus</i>. <i>American Journal of Botany</i>  <b>105</b>, 404-416. <br/> | <a href="https://doi.org/10.1002/ajb2.1068" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7376">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Allele phasing has minimal impact on phylogenetic reconstruction from targeted nuclear gene sequences in a case study of Artocarpus." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7379">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29729187" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7392">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R69"><!-- named anchor --></a><span class="citation"><a id="d4e7399"><!-- named anchor --></a>Kozlov AM, Darriba D, Flouri T, Morel B, Stamatakis A (2019) <i>RAxML-NG</i>: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference. <i>Bioinformatics</i>  <b>35</b>, 4453-4455. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btz305" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7410">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7413">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/31070718" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7426">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R70"><!-- named anchor --></a><span class="citation"><a id="d4e7433"><!-- named anchor --></a>K&#x000FC;ck P, Romahn J, Meusemann K (2022) Pitfalls of the site-concordance factor (sCF) as measure of phylogenetic branch support. <i>NAR Genomics and Bioinformatics</i>  <b>4</b>, lqac064. <br/> | <a href="https://doi.org/10.1093/nargab/lqac064" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7441">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Pitfalls of the site-concordance factor (sCF) as measure of phylogenetic branch support." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7444">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/36128424" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7457">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R71"><!-- named anchor --></a><span class="citation"><a id="d4e7465"><!-- named anchor --></a>Kumar S, Filipski AJ, Battistuzzi FU, Kosakovsky Pond SL, Tamura K (2012) Statistics and truth in phylogenomics. <i>Molecular Biology and Evolution</i>  <b>29</b>, 457-472. <br/> | <a href="https://doi.org/10.1093/molbev/msr202" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7473">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Statistics and truth in phylogenomics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7476">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/21873298" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7489">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R72"><!-- named anchor --></a><span class="citation"><a id="d4e7496"><!-- named anchor --></a>Landis JB, Soltis DE, Li Z, Marx HE, Barker MS, Tank DC, Soltis PS (2018) Impact of whole&#x02010;genome duplication events on diversification rates in angiosperms. <i>American Journal of Botany</i>  <b>105</b>, 348-363. <br/> | <a href="https://doi.org/10.1002/ajb2.1060" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7504">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Impact of whole&#x02010;genome duplication events on diversification rates in angiosperms." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7507">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29719043" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7520">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R73"><!-- named anchor --></a><span class="citation"><a id="d4e7527"><!-- named anchor --></a>Lanfear R, Hahn MW (2024) The meaning and measure of concordance factors in phylogenomics. <i>Molecular Biology and Evolution</i>  <b>41</b>, msae214. <br/> | <a href="https://doi.org/10.1093/molbev/msae214" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7535">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The meaning and measure of concordance factors in phylogenomics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7538">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/39418118" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7551">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R74"><!-- named anchor --></a><span class="citation"><a id="d4e7558"><!-- named anchor --></a>Larget BR, Kotha SK, Dewey CN, An&#x000E9; C (2010) <i>BUCKy</i>: gene tree/species tree reconciliation with Bayesian concordance analysis. <i>Bioinformatics</i>  <b>26</b>, 2910-2911. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btq539" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7569">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=BUCKy: gene tree/species tree reconciliation with Bayesian concordance analysis." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7572">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/20861028" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7585">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R75"><!-- named anchor --></a><span class="citation"><a id="d4e7592"><!-- named anchor --></a>Larridon I, Zuntini AR, Barrett RL, Wilson KL, Bruhl JJ, Goetghebeur P, Baker WJ, Brewer GE, Epitawalage N, Fairlie I, Forest F, Sabino Kikuchi IAB, Pokorny L, Semmouri I, Spalink D, Simpson DA, Muasya AM, Roalson EH (2021) Resolving generic limits in Cyperaceae tribe Abildgaardieae using targeted sequencing. <i>Botanical Journal of the Linnean Society</i>  <b>196</b>, 163-187. <br/> | <a href="https://doi.org/10.1093/botlinnean/boaa099" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7600">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Resolving generic limits in Cyperaceae tribe Abildgaardieae using targeted sequencing." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7603">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R76"><!-- named anchor --></a><span class="citation"><a id="d4e7619"><!-- named anchor --></a>Leach&#x000E9; AD, Rannala B (2011) The accuracy of species tree estimation under simulation: a comparison of methods. <i>Systematic Biology</i>  <b>60</b>, 126-137. <br/> | <a href="https://doi.org/10.1093/sysbio/syq073" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7627">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The accuracy of species tree estimation under simulation: a comparison of methods." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7630">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/21088009" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7643">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R77"><!-- named anchor --></a><span class="citation"><a id="d4e7651"><!-- named anchor --></a>Lemmon AR, Emme SA, Lemmon EM (2012) Anchored hybrid enrichment for massively high-throughput phylogenomics. <i>Systematic Biology</i>  <b>61</b>(5), 727-744. <br/> | <a href="https://doi.org/10.1093/sysbio/sys049" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7659">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Anchored hybrid enrichment for massively high-throughput phylogenomics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7662">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/22605266" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7675">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R78"><!-- named anchor --></a><span class="citation"><a id="d4e7682"><!-- named anchor --></a>Lewis PO, Holder MT, Holsinger KE (2005) Polytomies and Bayesian phylogenetic inference. <i>Systematic Biology</i>  <b>54</b>, 241-253. <br/> | <a href="https://doi.org/10.1080/10635150590924208" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7690">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Polytomies and Bayesian phylogenetic inference." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7693">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/16012095" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7706">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R79"><!-- named anchor --></a><span class="citation"><a id="d4e7713"><!-- named anchor --></a>Li D, Liu C-M, Luo R, Sadakane K, Lam T-W (2015) <i>MEGAHIT</i>: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct <i>de Bruijn</i> graph. <i>Bioinformatics</i>  <b>31</b>, 1674-1676. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btv033" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7727">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7731">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/25609793" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7743">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R80"><!-- named anchor --></a><span class="citation"><a id="d4e7750"><!-- named anchor --></a>Liao X, Li M, Zou Y, Wu F, Yi&#x02010;Pan , Wang J (2019) Current challenges and solutions of <i>de novo</i> assembly. <i>Quantitative Biology</i>  <b>7</b>, 90-109. <br/> | <a href="https://doi.org/10.1007/s40484-019-0166-9" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7761">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Current challenges and solutions of de novo assembly." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7764">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R81"><!-- named anchor --></a><span class="citation"><a id="d4e7781"><!-- named anchor --></a>McKinnon GE, Steane DA, Potts BM, Vaillancourt RE (1999) Incongruence between chloroplast and species phylogenies in <i>Eucalyptus</i> subgenus <i>Monocalyptus</i> (Myrtaceae). <i>American Journal of Botany</i>  <b>86</b>, 1038-1046. <br/> | <a href="https://doi.org/10.2307/2656621" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7795">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Incongruence between chloroplast and species phylogenies in Eucalyptus subgenus Monocalyptus (Myrtaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7799">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/10406727" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7811">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R82"><!-- named anchor --></a><span class="citation"><a id="d4e7818"><!-- named anchor --></a>McLay TGB, Birch JL, Gunn BF, Ning W, Tate JA, Nauheimer L, Joyce EM, Simpson L, Schmidt-Lebuhn AN, Baker WJ, Forest F, Jackson CJ (2021) New targets acquired: improving locus recovery from the Angiosperms353 probe set. <i>Applications in Plant Sciences</i>  <b>9</b>, e11420. <br/> | <a href="https://doi.org/10.1002/aps3.11420" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7826">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=New targets acquired: improving locus recovery from the Angiosperms353 probe set." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7829">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34336399" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7842">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R83"><!-- named anchor --></a><span class="citation"><a id="d4e7850"><!-- named anchor --></a>McLay TGB, Fowler RM, Fahey PS, Murphy DJ, Udovicic F, Cantrill DJ, Bayly MJ (2023) Phylogenomics reveals extreme gene tree discordance in a lineage of dominant trees: hybridization, introgression, and incomplete lineage sorting blur deep evolutionary relationships despite clear species groupings in <i>Eucalyptus</i> subgenus <i>Eudesmia</i>. <i>Molecular Phylogenetics and Evolution</i>  <b>187</b>, 107869. <br/> | <a href="https://doi.org/10.1016/j.ympev.2023.107869" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7864">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics reveals extreme gene tree discordance in a lineage of dominant trees: hybridization, introgression, and incomplete lineage sorting blur deep evolutionary relationships despite clear species groupings in Eucalyptus subgenus Eudesmia." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7868">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37423562" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7880">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R84"><!-- named anchor --></a><span class="citation"><a id="d4e7887"><!-- named anchor --></a>Maddison W (1989) Reconstructing character evolution on polytomous cladograms. <i>Cladistics</i>  <b>5</b>, 365-377. <br/> | <a href="https://doi.org/10.1111/j.1096-0031.1989.tb00569.x" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7895">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Reconstructing character evolution on polytomous cladograms." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7898">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34933477" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7911">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R85"><!-- named anchor --></a><span class="citation"><a id="d4e7918"><!-- named anchor --></a>Maddison WP (1997) Gene trees in species trees. <i>Systematic Biology</i>  <b>46</b>, 523-536. <br/> | <a href="https://doi.org/10.1093/sysbio/46.3.523" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7926">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Gene trees in species trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7929">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R86"><!-- named anchor --></a><span class="citation"><a id="d4e7945"><!-- named anchor --></a>Mai U, Mirarab S (2018) <i>TreeShrink</i>: fast and accurate detection of outlier long branches in collections of phylogenetic trees. <i>BMC Genomics</i>  <b>19</b>, 272. <br/> | <a href="https://doi.org/10.1186/s12864-018-4620-2" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7956">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=TreeShrink: fast and accurate detection of outlier long branches in collections of phylogenetic trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7959">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29745847" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7972">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R87"><!-- named anchor --></a><span class="citation"><a id="d4e7979"><!-- named anchor --></a>Marques DA, Meier JI, Seehausen O (2019) A combinatorial view on speciation and adaptive radiation. <i>Trends in Ecology &#x00026; Evolution</i>  <b>34</b>, 531-544. <br/> | <a href="https://doi.org/10.1016/j.tree.2019.02.008" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7987">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A combinatorial view on speciation and adaptive radiation." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e7990">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/30885412" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8003">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R88"><!-- named anchor --></a><span class="citation"><a id="d4e8010"><!-- named anchor --></a>Mason AS, Wendel JF (2020) Homoeologous exchanges, segmental allopolyploidy, and polyploid genome evolution. <i>Frontiers in Genetics</i>  <b>11</b>, 1014. <br/> | <a href="https://doi.org/10.3389/fgene.2020.01014" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8018">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Homoeologous exchanges, segmental allopolyploidy, and polyploid genome evolution." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8021">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33005183" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8034">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R89"><!-- named anchor --></a><span class="citation"><a id="d4e8042"><!-- named anchor --></a>Matsubayashi KW, Yamaguchi R (2022) The speciation view: disentangling multiple causes of adaptive and non-adaptive radiation in terms of speciation. <i>Population Ecology</i>  <b>64</b>, 95-107. <br/> | <a href="https://doi.org/10.1002/1438-390X.12103" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8050">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The speciation view: disentangling multiple causes of adaptive and non-adaptive radiation in terms of speciation." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8053">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R90"><!-- named anchor --></a><span class="citation"><a id="d4e8069"><!-- named anchor --></a>Minh BQ, Hahn MW, Lanfear R (2020<i><i>a</i></i>) New methods to calculate concordance factors for phylogenomic datasets. <i>Molecular Biology and Evolution</i>  <b>37</b>, 2727-2733. <br/> | <a href="https://doi.org/10.1093/molbev/msaa106" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8083">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=New methods to calculate concordance factors for phylogenomic datasets." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8086">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32365179" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8099">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R91"><!-- named anchor --></a><span class="citation"><a id="d4e8106"><!-- named anchor --></a>Minh BQ, Schmidt HA, Chernomor O, Schrempf D, Woodhams MD, von Haeseler A, Lanfear R (2020<i><i>b</i></i>) <i>IQ-TREE 2</i>: new models and efficient methods for phylogenetic inference in the genomic era. <i>Molecular Biology and Evolution</i>  <b>37</b>, 1530-1534. <br/> | <a href="https://doi.org/10.1093/molbev/msaa015" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8123">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=IQ-TREE 2: new models and efficient methods for phylogenetic inference in the genomic era." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8127">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32011700" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8139">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R92"><!-- named anchor --></a><span class="citation"><a id="d4e8146"><!-- named anchor --></a>Mirarab S, Reaz R, Bayzid MdS, Zimmermann T, Swenson MS, Warnow T (2014) <i>ASTRAL</i>: genome-scale coalescent-based species tree estimation. <i>Bioinformatics</i>  <b>30</b>, 541-8. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btu462" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8157">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=ASTRAL: genome-scale coalescent-based species tree estimation." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8160">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/25161245" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8173">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R93"><!-- named anchor --></a><span class="citation"><a id="d4e8180"><!-- named anchor --></a>Mirarab S, Bayzid MS, Warnow T (2016) Evaluating summary methods for multilocus species tree estimation in the presence of incomplete lineage sorting. <i>Systematic Biology</i>  <b>65</b>, 366-380. <br/> | <a href="https://doi.org/10.1093/sysbio/syu063" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8188">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Evaluating summary methods for multilocus species tree estimation in the presence of incomplete lineage sorting." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8191">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/25164915" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8204">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R94"><!-- named anchor --></a><span class="citation"><a id="d4e8211"><!-- named anchor --></a>Mishra S, Smith ML, Hahn MW (2023) <i>reconcILS</i>: a gene tree-species tree reconciliation algorithm that allows for incomplete lineage sorting. <i>bioRxiv</i>  <b>2023</b>, 2023.11.03.565544 [Preprint, published 13 October 2024]. <br/> | <a href="https://doi.org/10.1101/2023.11.03.565544" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8222">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=reconcILS: a gene tree-species tree reconciliation algorithm that allows for incomplete lineage sorting." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8225">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R95"><!-- named anchor --></a><span class="citation"><a id="d4e8243"><!-- named anchor --></a>Mo YK, Lanfear R, Hahn MW, Minh BQ (2023) Updated site concordance factors minimize effects of homoplasy and taxon sampling. <i>Bioinformatics</i>  <b>39</b>, btac741. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btac741" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8251">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Updated site concordance factors minimize effects of homoplasy and taxon sampling." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8254">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/36383168" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8267">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R96"><!-- named anchor --></a><span class="citation"><a id="d4e8274"><!-- named anchor --></a>Mo YK, Hahn MW, Smith ML (2024) Applications of machine learning in phylogenetics. <i>Molecular Phylogenetics and Evolution</i>  <b>196</b>, 108066. <br/> | <a href="https://doi.org/10.1016/j.ympev.2024.108066" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8282">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Applications of machine learning in phylogenetics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8285">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/38565358" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8298">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R97"><!-- named anchor --></a><span class="citation"><a id="d4e8305"><!-- named anchor --></a>Molloy EK, Warnow T (2018) To include or not to include: the impact of gene filtering on species tree estimation methods. <i>Systematic Biology</i>  <b>67</b>, 285-303. <br/> | <a href="https://doi.org/10.1093/sysbio/syx077" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8313">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=To include or not to include: the impact of gene filtering on species tree estimation methods." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8316">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29029338" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8329">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R98"><!-- named anchor --></a><span class="citation"><a id="d4e8336"><!-- named anchor --></a>Molloy EK, Warnow T (2020) <i>FastMulRFS</i>: fast and accurate species tree estimation under generic gene duplication and loss models. <i>Bioinformatics</i>  <b>36</b>, i57-i65. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btaa444" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8347">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=FastMulRFS: fast and accurate species tree estimation under generic gene duplication and loss models." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8350">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32657396" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8363">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R99"><!-- named anchor --></a><span class="citation"><a id="d4e8370"><!-- named anchor --></a>Mongiardino Koch N (2021) Phylogenomic subsampling and the search for phylogenetically reliable loci. <i>Molecular Biology and Evolution</i>  <b>38</b>, 4025-4038. <br/> | <a href="https://doi.org/10.1093/molbev/msab151" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8378">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic subsampling and the search for phylogenetically reliable loci." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8381">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33983409" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8394">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R100"><!-- named anchor --></a><span class="citation"><a id="d4e8401"><!-- named anchor --></a>Morales-Briones DF, Gehrke B, Huang C-H, Liston A, Ma H, Marx HE, Tank DC, Yang Y (2021) Analysis of paralogs in target enrichment data pinpoints multiple ancient polyploidy events in <i>Alchemilla</i> s.l. (Rosaceae). <i>Systematic Biology</i>  <b>71</b>, 190-207. <br/> | <a href="https://doi.org/10.1093/sysbio/syab032" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8412">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Analysis of paralogs in target enrichment data pinpoints multiple ancient polyploidy events in Alchemilla s.l. (Rosaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8415">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33978764" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8428">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R101"><!-- named anchor --></a><span class="citation"><a id="d4e8436"><!-- named anchor --></a>Morel B, Schade P, Lutteropp S, Williams TA, Sz&#x000F6;ll&#x00151;si GJ, Stamatakis A (2022) <i>SpeciesRax</i>: a tool for maximum likelihood species tree inference from gene family trees under duplication, transfer, and loss. <i>Molecular Biology and Evolution</i>  <b>39</b>, msab365. <br/> | <a href="https://doi.org/10.1093/molbev/msab365" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8447">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=SpeciesRax: a tool for maximum likelihood species tree inference from gene family trees under duplication, transfer, and loss." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8450">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35021210" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8463">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R102"><!-- named anchor --></a><span class="citation"><a id="d4e8470"><!-- named anchor --></a>Morel B, Williams TA, Stamatakis A, Sz&#x000F6;ll&#x00151;si GJ (2024) <i>AleRax</i>: a tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer, and loss. <i>Bioinformatics</i>  <b>40</b>, btae162. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btae162" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8481">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=AleRax: a tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer, and loss." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8484">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/38514421" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8497">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R103"><!-- named anchor --></a><span class="citation"><a id="d4e8504"><!-- named anchor --></a>Nauheimer L, Cui L, Clarke C, Crayn DM, Bourke G, Nargar K (2019) Genome skimming provides well resolved plastid and nuclear phylogenies, showing patterns of deep reticulate evolution in the tropical carnivorous plant genus <i>Nepenthes</i> (Caryophyllales). <i>Australian Systematic Botany</i>  <b>32</b>, 243-254. <br/> | <a href="https://doi.org/10.1071/SB18057" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8515">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Genome skimming provides well resolved plastid and nuclear phylogenies, showing patterns of deep reticulate evolution in the tropical carnivorous plant genus Nepenthes (Caryophyllales)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8518">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R104"><!-- named anchor --></a><span class="citation"><a id="d4e8535"><!-- named anchor --></a>Nauheimer L, Weigner N, Joyce E, Crayn D, Clarke C, Nargar K (2021) <i>HybPhaser</i>: a workflow for the detection and phasing of hybrids in target capture data sets. <i>Applications in Plant Sciences</i>  <b>9</b>, e11441. <br/> | <a href="https://doi.org/10.1002/aps3.11441" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8546">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=HybPhaser: a workflow for the detection and phasing of hybrids in target capture data sets." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8549">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34336402" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8562">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R105"><!-- named anchor --></a><span class="citation"><a id="d4e8569"><!-- named anchor --></a>Nevill PG, Despr&#x000E9;s T, Bayly MJ, Bossinger G, Ades PK (2014) Shared phylogeographic patterns and widespread chloroplast haplotype sharing in <i>Eucalyptus</i> species with different ecological tolerances. <i>Tree Genetics &#x00026; Genomes</i>  <b>10</b>, 1079-1092. <br/> | <a href="https://doi.org/10.1007/s11295-014-0744-y" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8580">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Shared phylogeographic patterns and widespread chloroplast haplotype sharing in Eucalyptus species with different ecological tolerances." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8583">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R106"><!-- named anchor --></a><span class="citation"><a id="d4e8600"><!-- named anchor --></a>Nge FJ, Biffin E, Thiele KR, Waycott M (2021<i><i>a</i></i>) Reticulate evolution, ancient chloroplast haplotypes, and rapid radiation of the Australian plant genus <i>Adenanthos</i> (Proteaceae). <i>Frontiers in Ecology and Evolution</i>  <b>8</b>, 616741. <br/> | <a href="https://doi.org/10.3389/fevo.2020.616741" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8617">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Reticulate evolution, ancient chloroplast haplotypes, and rapid radiation of the Australian plant genus Adenanthos (Proteaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8621">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R107"><!-- named anchor --></a><span class="citation"><a id="d4e8638"><!-- named anchor --></a>Nge FJ, Kellermann J, Biffin E, Waycott M, Thiele KR (2021<i><i>b</i></i>) Historical biogeography of <i>Pomaderris</i> (Rhamnaceae): continental vicariance in Australia and repeated independent dispersals to New Zealand. <i>Molecular Phylogenetics and Evolution</i>  <b>158</b>, 107085. <br/> | <a href="https://doi.org/10.1016/j.ympev.2021.107085" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8655">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Historical biogeography of Pomaderris (Rhamnaceae): continental vicariance in Australia and repeated independent dispersals to New Zealand." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8659">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33540078" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8671">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R108"><!-- named anchor --></a><span class="citation"><a id="d4e8678"><!-- named anchor --></a>Nge FJ, Biffin E, Waycott M, Thiele KR (2022) Phylogenomics and continental biogeographic disjunctions: insight from the Australian starflowers (<i>Calytrix</i>). <i>American Journal of Botany</i>  <b>109</b>, 291-308. <br/> | <a href="https://doi.org/10.1002/ajb2.1790" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8689">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics and continental biogeographic disjunctions: insight from the Australian starflowers (Calytrix)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8692">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34671970" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8705">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R109"><!-- named anchor --></a><span class="citation"><a id="d4e8712"><!-- named anchor --></a>Nge FJ, Kellermann J, Biffin E, Thiele KR, Waycott M (2024) Rise and fall of a continental mesic radiation in Australia: spine evolution, biogeography, and diversification of <i>Cryptandra</i> (Rhamnaceae: Pomaderreae). <i>Botanical Journal of the Linnean Society</i>  <b>204</b>, 327-342. <br/> | <a href="https://doi.org/10.1093/botlinnean/boad051" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8723">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Rise and fall of a continental mesic radiation in Australia: spine evolution, biogeography, and diversification of Cryptandra (Rhamnaceae: Pomaderreae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8726">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R110"><!-- named anchor --></a><span class="citation"><a id="d4e8743"><!-- named anchor --></a>Nge FJ, Biffin E, Rye BL, Wilson PG, van Dijk K, Thiele KR, Waycott M, Barrett MD (2025) Australian biogeography, climate-dependent diversification and phylogenomics of the spectacular Chamelaucieae tribe (Myrtaceae). <i>Australian Systematic Botany</i>  <b>38</b>(1), SB24014. <br/> | <a href="https://doi.org/10.1071/SB24014" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8751">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Australian biogeography, climate-dependent diversification and phylogenomics of the spectacular Chamelaucieae tribe (Myrtaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8754">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R111"><!-- named anchor --></a><span class="citation"><a id="d4e8770"><!-- named anchor --></a>Nguyen L-T, Schmidt HA, von Haeseler A, Minh BQ (2015) <i>IQ-TREE</i>: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. <i>Molecular Biology and Evolution</i>  <b>32</b>, 268-274. <br/> | <a href="https://doi.org/10.1093/molbev/msu300" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8781">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8784">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/25371430" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8797">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R112"><!-- named anchor --></a><span class="citation"><a id="d4e8804"><!-- named anchor --></a>Nicol DA, Saldivia P, Summerfield TC, Heads M, Lord JM, Khaing EP, Larcombe MJ (2024) Phylogenomics and morphology of Celmisiinae (Asteraceae: Astereae): taxonomic and evolutionary implications. <i>Molecular Phylogenetics and Evolution</i>  <b>195</b>, 108064. <br/> | <a href="https://doi.org/10.1016/j.ympev.2024.108064" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8812">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics and morphology of Celmisiinae (Asteraceae: Astereae): taxonomic and evolutionary implications." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8815">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/38508479" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8828">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R113"><!-- named anchor --></a><span class="citation"><a id="d4e8836"><!-- named anchor --></a>Nute M, Chou J, Molloy EK, Warnow T (2018) The performance of coalescent-based species tree estimation methods under models of missing data. <i>BMC Genomics</i>  <b>19</b>, 286. <br/> | <a href="https://doi.org/10.1186/s12864-018-4619-8" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8844">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The performance of coalescent-based species tree estimation methods under models of missing data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8847">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29745854" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8860">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R114"><!-- named anchor --></a><span class="citation"><a id="d4e8867"><!-- named anchor --></a>Orel HK, McLay TGB, Neal WC, Forster PI, Bayly MJ (2023) Plastid phylogenomics of the <i>Eriostemon</i> group (Rutaceae; Zanthoxyloideae): support for major clades and investigation of a backbone polytomy. <i>Australian Systematic Botany</i>  <b>36</b>, 355-385. <br/> | <a href="https://doi.org/10.1071/SB23011" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8878">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Plastid phylogenomics of the Eriostemon group (Rutaceae; Zanthoxyloideae): support for major clades and investigation of a backbone polytomy." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8881">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R115"><!-- named anchor --></a><span class="citation"><a id="d4e8898"><!-- named anchor --></a>Orel HK, McLay TG, Guja LK, Duretto MF, Bayly MJ (2024) Genomic data inform taxonomy and conservation of critically endangered shrubs: a case study of <i>Zieria</i> (Rutaceae) species from eastern Australia. <i>Botanical Journal of the Linnean Society</i>  <b>205</b>, 292-312. <br/> | <a href="https://doi.org/10.1093/botlinnean/boad069" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8909">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Genomic data inform taxonomy and conservation of critically endangered shrubs: a case study of Zieria (Rutaceae) species from eastern Australia." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8912">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R116"><!-- named anchor --></a><span class="citation"><a id="d4e8929"><!-- named anchor --></a>Orel HK, McLay TGB, Forster PI, Bayly MJ (2025) Target capture sequencing clarifies key relationships in the <i>Eriostemon</i> group (Rutaceae: Zanthoxyloideae) and supports a reclassification of <i>Philotheca</i>, including the recognition of two new genera. <i>TAXON.</i> [Published online early 6 February 2025]. <br/> | <a href="https://doi.org/10.1002/tax.13308" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8940">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Target capture sequencing clarifies key relationships in the Eriostemon group (Rutaceae: Zanthoxyloideae) and supports a reclassification of Philotheca, including the recognition of two new genera." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8943">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R117"><!-- named anchor --></a><span class="citation"><a id="d4e8960"><!-- named anchor --></a>Ortiz EM, H&#x000F6;wener A, Shigita G, Raza M, Maurin O, Zuntini A, Forest F, Baker WJ, Schaefer H (2023) A novel phylogenomics pipeline reveals complex pattern of reticulate evolution in Cucurbitales. <i>bioRxiv</i>  <b>2023</b>, 2023.10.27.564367 [Preprint, published 26 December 2024]. <br/> | <a href="https://doi.org/10.1101/2023.10.27.564367" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8968">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A novel phylogenomics pipeline reveals complex pattern of reticulate evolution in Cucurbitales." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8971">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R118"><!-- named anchor --></a><span class="citation"><a id="d4e8987"><!-- named anchor --></a>Ostevik KL, Andrew RL, Otto SP, Rieseberg LH (2016) Multiple reproductive barriers separate recently diverged sunflower ecotypes. <i>Evolution</i>  <b>70</b>, 2322-2335. <br/> | <a href="https://doi.org/10.1111/evo.13027" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8995">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Multiple reproductive barriers separate recently diverged sunflower ecotypes." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e8998">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/27479368" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9011">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R119"><!-- named anchor --></a><span class="citation"><a id="d4e9019"><!-- named anchor --></a>Page RD (1994) Maps between trees and cladistic analysis of historical associations among genes, organisms, and areas. <i>Systematic Biology</i>  <b>43</b>, 58-77. <br/> | <a href="https://doi.org/10.1093/sysbio/43.1.58" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9027">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Maps between trees and cladistic analysis of historical associations among genes, organisms, and areas." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9030">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R120"><!-- named anchor --></a><span class="citation"><a id="d4e9046"><!-- named anchor --></a>Pamilo P, Nei M (1988) Relationships between gene trees and species trees. <i>Molecular Biology and Evolution</i>  <b>5</b>, 568-583. <br/> | <a href="https://doi.org/10.1093/oxfordjournals.molbev.a040517" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9054">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Relationships between gene trees and species trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9057">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/3193878" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9070">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R121"><!-- named anchor --></a><span class="citation"><a id="d4e9077"><!-- named anchor --></a>Panchy N, Lehti-Shiu M, Shiu S-H (2016) Evolution of gene duplication in plants. <i>Plant Physiology</i>  <b>171</b>, 2294-2316. <br/> | <a href="https://doi.org/10.1104/pp.16.00523" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9085">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Evolution of gene duplication in plants." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9088">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/27288366" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9101">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R122"><!-- named anchor --></a><span class="citation"><a id="d4e9108"><!-- named anchor --></a>Paradis E (2013) Molecular dating of phylogenies by likelihood methods: a comparison of models and a new information criterion. <i>Molecular Phylogenetics and Evolution</i>  <b>67</b>, 436-444. <br/> | <a href="https://doi.org/10.1016/j.ympev.2013.02.008" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9116">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Molecular dating of phylogenies by likelihood methods: a comparison of models and a new information criterion." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9119">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/23454091" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9132">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R123"><!-- named anchor --></a><span class="citation"><a id="d4e9139"><!-- named anchor --></a>Paradis E, Schliep K (2019) <i>ape</i> 5.0: an environment for modern phylogenetics and evolutionary analyses in <i>R</i>. <i>Bioinformatics</i>  <b>35</b>, 526-528. <br/> | <a href="https://doi.org/10.1093/bioinformatics/bty633" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9153">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9157">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/30016406" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9169">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R124"><!-- named anchor --></a><span class="citation"><a id="d4e9176"><!-- named anchor --></a>Peakall R, Wong DCJ, Phillips RD, Ruibal M, Eyles R, Rodriguez-Delgado C, Linde CC (2021) A multitiered sequence capture strategy spanning broad evolutionary scales: application for phylogenetic and phylogeographic studies of orchids. <i>Molecular Ecology Resources</i>  <b>21</b>, 1118-1140. <br/> | <a href="https://doi.org/10.1111/1755-0998.13327" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9184">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A multitiered sequence capture strategy spanning broad evolutionary scales: application for phylogenetic and phylogeographic studies of orchids." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9187">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33453072" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9200">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R125"><!-- named anchor --></a><span class="citation"><a id="d4e9208"><!-- named anchor --></a>Pease JB, Brown JW, Walker JF, Hinchliff CE, Smith SA (2018) <i>Quartet Sampling</i> distinguishes lack of support from conflicting support in the green plant tree of life. <i>American Journal of Botany</i>  <b>105</b>, 385-403. <br/> | <a href="https://doi.org/10.1002/ajb2.1016" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9219">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Quartet Sampling distinguishes lack of support from conflicting support in the green plant tree of life." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9222">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29746719" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9235">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R126"><!-- named anchor --></a><span class="citation"><a id="d4e9242"><!-- named anchor --></a>Pillon Y, Hopkins HCF, Maurin O, Epitawalage N, Bradford J, Rogers ZS, Baker WJ, Forest F (2021) Phylogenomics and biogeography of Cunoniaceae (Oxalidales) with complete generic sampling and taxonomic realignments. <i>American Journal of Botany</i>  <b>108</b>, 1181-1200. <br/> | <a href="https://doi.org/10.1002/ajb2.1688" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9250">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics and biogeography of Cunoniaceae (Oxalidales) with complete generic sampling and taxonomic realignments." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9253">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34278558" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9266">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R127"><!-- named anchor --></a><span class="citation"><a id="d4e9273"><!-- named anchor --></a>Potts AJ, Hedderson TA, Grimm GW (2014) Constructing phylogenies in the presence of intra-individual site polymorphisms (2ISPs) with a focus on the nuclear ribosomal cistron. <i>Systematic Biology</i>  <b>63</b>, 1-16. <br/> | <a href="https://doi.org/10.1093/sysbio/syt052" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9281">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Constructing phylogenies in the presence of intra-individual site polymorphisms (2ISPs) with a focus on the nuclear ribosomal cistron." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9284">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/23939193" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9297">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R128"><!-- named anchor --></a><span class="citation"><a id="d4e9304"><!-- named anchor --></a>Rannala B, Edwards VS, Leach&#x000E9; A, Yang Z (2020) The multi-species coalescent model and species tree inference. In &#x02018;Phylogenetics in the Genomic Era)&#x02019;. pp. 3.3:1&#x02013;3.3:21. (Published by the authors, CC BY-NC-ND) Available at <a href="https://hal.science/hal-02535622v1" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9306">https://hal.science/hal-02535622v1</a></span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R129"><!-- named anchor --></a><span class="citation"><a id="d4e9313"><!-- named anchor --></a>Raza M, Ortiz EM, Schwung L, Shigita G, Schaefer H (2023) Resolving the phylogeny of <i>Thladiantha</i> (Cucurbitaceae) with three different target capture pipelines. <i>BMC Ecology and Evolution</i>  <b>23</b>, 75. <br/> | <a href="https://doi.org/10.1186/s12862-023-02185-z" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9324">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Resolving the phylogeny of Thladiantha (Cucurbitaceae) with three different target capture pipelines." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9327">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/38087247" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9340">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R130"><!-- named anchor --></a><span class="citation"><a id="d4e9347"><!-- named anchor --></a>Ren R, Wang H, Guo C, Zhang N, Zeng L, Chen Y, Ma H, Qi J (2018) Widespread whole genome duplications contribute to genome complexity and species diversity in angiosperms. <i>Molecular Plant</i>  <b>11</b>, 414-428. <br/> | <a href="https://doi.org/10.1016/j.molp.2018.01.002" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9355">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Widespread whole genome duplications contribute to genome complexity and species diversity in angiosperms." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9358">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29317285" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9371">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R131"><!-- named anchor --></a><span class="citation"><a id="d4e9379"><!-- named anchor --></a>Salichos L, Rokas A (2013) Inferring ancient divergences requires genes with strong phylogenetic signals. <i>Nature</i>  <b>497</b>, 327-331. <br/> | <a href="https://doi.org/10.1038/nature12130" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9387">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Inferring ancient divergences requires genes with strong phylogenetic signals." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9390">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/23657258" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9403">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R132"><!-- named anchor --></a><span class="citation"><a id="d4e9410"><!-- named anchor --></a>Sanderson MJ (2002) Estimating absolute rates of molecular evolution and divergence times: a penalized likelihood approach. <i>Molecular Biology and Evolution</i>  <b>19</b>, 101-109. <br/> | <a href="https://doi.org/10.1093/oxfordjournals.molbev.a003974" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9418">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Estimating absolute rates of molecular evolution and divergence times: a penalized likelihood approach." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9421">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/11752195" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9434">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R133"><!-- named anchor --></a><span class="citation"><a id="d4e9441"><!-- named anchor --></a>Sanderson MJ (2003) <i>r8s</i>: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock. <i>Bioinformatics</i>  <b>19</b>, 301-302. <br/> | <a href="https://doi.org/10.1093/bioinformatics/19.2.301" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9452">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9455">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/12538260" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9468">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R134"><!-- named anchor --></a><span class="citation"><a id="d4e9475"><!-- named anchor --></a>Sayyari E, Mirarab S (2016) Fast coalescent-based computation of local branch support from quartet frequencies. <i>Molecular Biology and Evolution</i>  <b>33</b>, 1654-1668. <br/> | <a href="https://doi.org/10.1093/molbev/msw079" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9483">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Fast coalescent-based computation of local branch support from quartet frequencies." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9486">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/27189547" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9499">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R135"><!-- named anchor --></a><span class="citation"><a id="d4e9506"><!-- named anchor --></a>Sayyari E, Mirarab S (2018) Testing for polytomies in phylogenetic species trees using quartet frequencies. <i>Genes</i>  <b>9</b>, 132. <br/> | <a href="https://doi.org/10.3390/genes9030132" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9514">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Testing for polytomies in phylogenetic species trees using quartet frequencies." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9517">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29495636" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9530">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R136"><!-- named anchor --></a><span class="citation"><a id="d4e9537"><!-- named anchor --></a>Sayyari E, Whitfield JB, Mirarab S (2018) <i>DiscoVista</i>: interpretable visualizations of gene tree discordance. <i>Molecular Phylogenetics and Evolution</i>  <b>122</b>, 110-115. <br/> | <a href="https://doi.org/10.1016/j.ympev.2018.01.019" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9548">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=DiscoVista: interpretable visualizations of gene tree discordance." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9551">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29421312" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9564">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R137"><!-- named anchor --></a><span class="citation"><a id="d4e9572"><!-- named anchor --></a>Scannell DR, Byrne KP, Gordon JL, Wong S, Wolfe KH (2006) Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts. <i>Nature</i>  <b>440</b>, 341-345. <br/> | <a href="https://doi.org/10.1038/nature04562" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9580">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9583">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/16541074" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9596">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R138"><!-- named anchor --></a><span class="citation"><a id="d4e9603"><!-- named anchor --></a>Schmidt-Lebuhn AN (2022) Sequence capture data support the taxonomy of <i>Pogonolepis</i> (Asteraceae: Gnaphalieae) and show unexpected genetic structure. <i>Australian Systematic Botany</i>  <b>35</b>(4), 317-325. <br/> | <a href="https://doi.org/10.1071/SB22010" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9614">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Sequence capture data support the taxonomy of Pogonolepis (Asteraceae: Gnaphalieae) and show unexpected genetic structure." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9617">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R139"><!-- named anchor --></a><span class="citation"><a id="d4e9634"><!-- named anchor --></a>Schmidt-Lebuhn AN, Bovill J (2021) Phylogenomic data reveal four major clades of Australian <i>Gnaphalieae</i> (Asteraceae). <i>TAXON</i>  <b>70</b>, 1020-1034. <br/> | <a href="https://doi.org/10.1002/tax.12510" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9645">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic data reveal four major clades of Australian Gnaphalieae (Asteraceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9648">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R140"><!-- named anchor --></a><span class="citation"><a id="d4e9665"><!-- named anchor --></a>Schmidt-Lebuhn AN, Grealy A (2024) Transfer of <i>Cotula alpina</i> to the genus <i>Leptinella</i> (Asteraceae: Anthemideae). <i>Australian Systematic Botany</i>  <b>37</b>, SB23012. <br/> | <a href="https://doi.org/10.1071/SB23012" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9679">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Transfer of Cotula alpina to the genus Leptinella (Asteraceae: Anthemideae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9683">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R141"><!-- named anchor --></a><span class="citation"><a id="d4e9699"><!-- named anchor --></a>Schmidt-Lebuhn AN, Egli D, Grealy A, Nicholls JA, Zwick A, Dymock JJ, Gooden B (2024<i><i>a</i></i>) Genetic data confirm the presence of <i>Senecio madagascariensis</i> in New Zealand. <i>New Zealand Journal of Botany</i>  <b>62</b>, 1-13. <br/> | <a href="https://doi.org/10.1080/0028825X.2022.2148544" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9716">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Genetic data confirm the presence of Senecio madagascariensis in New Zealand." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9720">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R142"><!-- named anchor --></a><span class="citation"><a id="d4e9736"><!-- named anchor --></a>Schmidt-Lebuhn AN, Chen SH, Grealy A (2024<i><i>b</i></i>) <i>Elachanthus</i>, <i>Isoetopsis</i> and <i>Kippistia</i> are nested in the genus <i>Minuria</i> (Asteraceae: Astereae). <i>Australian Systematic Botany</i>  <b>37</b>(4), SB23028. <br/> | <a href="https://doi.org/10.1071/SB23028" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9763">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Elachanthus, Isoetopsis and Kippistia are nested in the genus Minuria (Asteraceae: Astereae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9766">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R143"><!-- named anchor --></a><span class="citation"><a id="d4e9783"><!-- named anchor --></a>Shee ZQ, Frodin DG, C&#x000E1;mara-Leret R, Pokorny L (2020) Reconstructing the complex evolutionary history of the Papuasian <i>Schefflera</i> radiation through herbariomics. <i>Frontiers in Plant Science</i>  <b>11</b>, 258. <br/> | <a href="https://doi.org/10.3389/fpls.2020.00258" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9794">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Reconstructing the complex evolutionary history of the Papuasian Schefflera radiation through herbariomics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9797">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32265950" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9810">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R144"><!-- named anchor --></a><span class="citation"><a id="d4e9817"><!-- named anchor --></a>Simmons MP, Gatesy J (2015) Coalescence vs. concatenation: sophisticated analyses vs. first principles applied to rooting the angiosperms. <i>Molecular Phylogenetics and Evolution</i>  <b>91</b>, 98-122. <br/> | <a href="https://doi.org/10.1016/j.ympev.2015.05.011" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9825">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Coalescence vs. concatenation: sophisticated analyses vs. first principles applied to rooting the angiosperms." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9828">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26002829" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9841">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R145"><!-- named anchor --></a><span class="citation"><a id="d4e9848"><!-- named anchor --></a>Simmons MP, Gatesy J (2021) Collapsing dubiously resolved gene-tree branches in phylogenomic coalescent analyses. <i>Molecular Phylogenetics and Evolution</i>  <b>158</b>, 107092. <br/> | <a href="https://doi.org/10.1016/j.ympev.2021.107092" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9856">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Collapsing dubiously resolved gene-tree branches in phylogenomic coalescent analyses." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9859">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33545272" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9872">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R146"><!-- named anchor --></a><span class="citation"><a id="d4e9879"><!-- named anchor --></a>Simpson J, Conran JG, Biffin E, Van Dijk K, Waycott M (2022) The <i>Crinum flaccidum</i> (Amaryllidaceae) species complex in Australia. <i>Australian Systematic Botany</i>  <b>35</b>, 395-402. <br/> | <a href="https://doi.org/10.1071/SB21038" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9890">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The Crinum flaccidum (Amaryllidaceae) species complex in Australia." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9893">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R147"><!-- named anchor --></a><span class="citation"><a id="d4e9910"><!-- named anchor --></a>Siniscalchi CM, Hidalgo O, Palazzesi L, Pellicer J, Pokorny L, Maurin O, Leitch IJ, Forest F, Baker WJ, Mandel JR (2021) Lineage&#x02010;specific vs. universal: a comparison of the Compositae1061 and Angiosperms353 enrichment panels in the sunflower family. <i>Applications in Plant Sciences</i>  <b>9</b>, e11422. <br/> | <a href="https://doi.org/10.1002/aps3.11422" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9918">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Lineage&#x02010;specific vs. universal: a comparison of the Compositae1061 and Angiosperms353 enrichment panels in the sunflower family." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9921">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R148"><!-- named anchor --></a><span class="citation"><a id="d4e9937"><!-- named anchor --></a>Smirnov V, Warnow T (2021) Phylogeny estimation given sequence length heterogeneity. <i>Systematic Biology</i>  <b>70</b>(2), 268-282. <br/> | <a href="https://doi.org/10.1093/sysbio/syaa058" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9945">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogeny estimation given sequence length heterogeneity." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9948">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R149"><!-- named anchor --></a><span class="citation"><a id="d4e9965"><!-- named anchor --></a>Smith ML, Hahn MW (2021) New approaches for inferring phylogenies in the presence of paralogs. <i>Trends in Genetics</i>  <b>37</b>, 174-187. <br/> | <a href="https://doi.org/10.1016/j.tig.2020.08.012" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9973">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=New approaches for inferring phylogenies in the presence of paralogs." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9976">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32921510" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e9989">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R150"><!-- named anchor --></a><span class="citation"><a id="d4e9996"><!-- named anchor --></a>Smith ML, Hahn MW (2022) The frequency and topology of pseudoorthologs. <i>Systematic Biology</i>  <b>71</b>, 649-659. <br/> | <a href="https://doi.org/10.1093/sysbio/syab097" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10004">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The frequency and topology of pseudoorthologs." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10007">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34951639" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10020">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R151"><!-- named anchor --></a><span class="citation"><a id="d4e10027"><!-- named anchor --></a>Smith SA, O&#x02019;Meara BC (2012) <i>treePL</i>: divergence time estimation using penalized likelihood for large phylogenies. <i>Bioinformatics</i>  <b>28</b>, 2689-2690. <br/> | <a href="https://doi.org/10.1093/bioinformatics/bts492" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10038">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=treePL: divergence time estimation using penalized likelihood for large phylogenies." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10041">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/22908216" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10054">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R152"><!-- named anchor --></a><span class="citation"><a id="d4e10061"><!-- named anchor --></a>Smith SA, Moore MJ, Brown JW, Yang Y (2015) Analysis of phylogenomic datasets reveals conflict, concordance, and gene duplications with examples from animals and plants. <i>BMC Evolutionary Biology</i>  <b>15</b>, 150. <br/> | <a href="https://doi.org/10.1186/s12862-015-0423-0" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10069">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Analysis of phylogenomic datasets reveals conflict, concordance, and gene duplications with examples from animals and plants." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10072">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26239519" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10085">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R153"><!-- named anchor --></a><span class="citation"><a id="d4e10092"><!-- named anchor --></a>Smith SA, Brown JW, Walker JF (2018) So many genes, so little time: a practical approach to divergence-time estimation in the genomic era. <i>PLoS ONE</i>  <b>13</b>, e0197433. <br/> | <a href="https://doi.org/10.1371/journal.pone.0197433" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10100">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=So many genes, so little time: a practical approach to divergence-time estimation in the genomic era." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10103">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29772020" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10116">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R154"><!-- named anchor --></a><span class="citation"><a id="d4e10123"><!-- named anchor --></a>Smith BT, Mauck WM, Benz BW, Andersen MJ (2020) Uneven missing data skew phylogenomic relationships within the lories and lorikeets. <i>Genome Biology and Evolution</i>  <b>12</b>, 1131-1147. <br/> | <a href="https://doi.org/10.1093/gbe/evaa113" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10131">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Uneven missing data skew phylogenomic relationships within the lories and lorikeets." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10134">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32470111" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10147">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R155"><!-- named anchor --></a><span class="citation"><a id="d4e10155"><!-- named anchor --></a>Sol&#x000ED;s-Lemus C, An&#x000E9; C (2016) Inferring phylogenetic networks with maximum pseudolikelihood under incomplete lineage sorting. <i>PLoS Genetics</i>  <b>12</b>, e1005896. <br/> | <a href="https://doi.org/10.1371/journal.pgen.1005896" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10163">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Inferring phylogenetic networks with maximum pseudolikelihood under incomplete lineage sorting." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10166">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26950302" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10179">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R156"><!-- named anchor --></a><span class="citation"><a id="d4e10186"><!-- named anchor --></a>Sol&#x000ED;s-Lemus C, Bastide P, An&#x000E9; C (2017) <i>PhyloNetworks</i>: a package for phylogenetic networks. <i>Molecular Biology and Evolution</i>  <b>34</b>(12), 3292-3298. <br/> | <a href="https://doi.org/10.1093/molbev/msx235" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10197">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=PhyloNetworks: a package for phylogenetic networks." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10200">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R157"><!-- named anchor --></a><span class="citation"><a id="d4e10217"><!-- named anchor --></a>Soltis DE, Albert VA, Leebens-Mack J, Bell CD, Paterson AH, Zheng C, Sankoff D, de Pamphilis CW, Wall PK, Soltis PS (2009) Polyploidy and angiosperm diversification. <i>American Journal of Botany</i>  <b>96</b>, 336-348. <br/> | <a href="https://doi.org/10.3732/ajb.0800079" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10225">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Polyploidy and angiosperm diversification." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10228">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/21628192" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10241">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R158"><!-- named anchor --></a><span class="citation"><a id="d4e10248"><!-- named anchor --></a>Stamatakis A (2014) <i>RAxML</i> version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. <i>Bioinformatics</i>  <b>30</b>, 1312-1313. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btu033" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10259">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10262">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/24451623" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10275">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R159"><!-- named anchor --></a><span class="citation"><a id="d4e10282"><!-- named anchor --></a>Steenwyk JL, Buida TJ, Li Y, Shen X-X, Rokas A (2020) <i>ClipKIT</i>: a multiple sequence alignment trimming software for accurate phylogenomic inference. <i>PLoS Biology</i>  <b>18</b>, e3001007. <br/> | <a href="https://doi.org/10.1371/journal.pbio.3001007" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10293">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=ClipKIT: a multiple sequence alignment trimming software for accurate phylogenomic inference." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10296">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33264284" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10309">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R160"><!-- named anchor --></a><span class="citation"><a id="d4e10316"><!-- named anchor --></a>Steenwyk JL, Li Y, Zhou X, Shen X-X, Rokas A (2023) Incongruence in the phylogenomics era. <i>Nature Reviews Genetics</i>  <b>24</b>, 834-850. <br/> | <a href="https://doi.org/10.1038/s41576-023-00620-x" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10324">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Incongruence in the phylogenomics era." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10327">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37369847" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10340">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R161"><!-- named anchor --></a><span class="citation"><a id="d4e10348"><!-- named anchor --></a>Stegemann S, Keuthe M, Greiner S, Bock R (2012) Horizontal transfer of chloroplast genomes between plant species. <i>Proceedings of the National Academy of Sciences of the United States of America</i>  <b>109</b>, 2434-2438. <br/> | <a href="https://doi.org/10.1073/pnas.1114076109" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10356">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Horizontal transfer of chloroplast genomes between plant species." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10359">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/22308367" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10372">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R162"><!-- named anchor --></a><span class="citation"><a id="d4e10379"><!-- named anchor --></a>Strobel V (2018) Pold87/academic-keyword-occurrence: first release. <i>Zenodo</i>  <b>2018</b>, v1.0.0 [Dataset, published 14 April 2014]. <br/> | <a href="https://doi.org/10.5281/ZENODO.1218409" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10387">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Pold87/academic-keyword-occurrence: first release." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10390">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R163"><!-- named anchor --></a><span class="citation"><a id="d4e10406"><!-- named anchor --></a>Struck TH (2013) The impact of paralogy on phylogenomic studies &#x02013; a case study on annelid relationships. <i>PLoS ONE</i>  <b>8</b>, e62892. <br/> | <a href="https://doi.org/10.1371/journal.pone.0062892" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10414">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The impact of paralogy on phylogenomic studies &#x02013; a case study on annelid relationships." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10417">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/23667537" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10430">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R164"><!-- named anchor --></a><span class="citation"><a id="d4e10437"><!-- named anchor --></a>Stull GW, Pham KK, Soltis PS, Soltis DE (2023) Deep reticulation: the long legacy of hybridization in vascular plant evolution. <i>The Plant Journal</i>  <b>114</b>, 743-766. <br/> | <a href="https://doi.org/10.1111/tpj.16142" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10445">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Deep reticulation: the long legacy of hybridization in vascular plant evolution." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10448">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/36775995" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10461">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R165"><!-- named anchor --></a><span class="citation"><a id="d4e10468"><!-- named anchor --></a>Suarez-Gonzalez A, Lexer C, Cronk QCB (2018) Adaptive introgression: a plant perspective. <i>Biology Letters</i>  <b>14</b>, 20170688. <br/> | <a href="https://doi.org/10.1098/rsbl.2017.0688" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10476">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Adaptive introgression: a plant perspective." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10479">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29540564" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10492">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R166"><!-- named anchor --></a><span class="citation"><a id="d4e10499"><!-- named anchor --></a>Swofford DL, Olsen GJ, Waddell PJ, Hillis DM (1996) Phylogenetic inference. In &#x02018;Molecular Systematics&#x02019;. (Eds DM Hillis, C Moritz, BK Mable) pp. 407&#x02013;514. (Sinauer Associates: Sunderland, MA, USA)</span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R167"><!-- named anchor --></a><span class="citation"><a id="d4e10506"><!-- named anchor --></a>Tamura K, Battistuzzi FU, Billing-Ross P, Murillo O, Filipski A, Kumar S (2012) Estimating divergence times in large molecular phylogenies. <i>Proceedings of the National Academy of Sciences of the United States of America</i>  <b>109</b>, 19333-19338. <br/> | <a href="https://doi.org/10.1073/pnas.1213199109" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10514">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Estimating divergence times in large molecular phylogenies." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10517">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/23129628" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10530">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R168"><!-- named anchor --></a><span class="citation"><a id="d4e10537"><!-- named anchor --></a>Tamura K, Tao Q, Kumar S (2018) Theoretical foundation of the <i>RelTime</i> method for estimating divergence times from variable evolutionary rates. <i>Molecular Biology and Evolution</i>  <b>35</b>, 1770-1782. <br/> | <a href="https://doi.org/10.1093/molbev/msy044" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10548">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Theoretical foundation of the RelTime method for estimating divergence times from variable evolutionary rates." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10551">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29893954" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10564">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R169"><!-- named anchor --></a><span class="citation"><a id="d4e10571"><!-- named anchor --></a>Tea Y-K, Xu X, DiBattista JD, Lo N, Cowman PF, Ho SYW (2022) Phylogenomic analysis of concatenated ultraconserved elements reveals the recent evolutionary radiation of the fairy wrasses (Teleostei: Labridae: Cirrhilabrus). <i>Systematic Biology</i>  <b>71</b>, 1-12. <br/> | <a href="https://doi.org/10.1093/sysbio/syab012" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10579">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic analysis of concatenated ultraconserved elements reveals the recent evolutionary radiation of the fairy wrasses (Teleostei: Labridae: Cirrhilabrus)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10582">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/33620490" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10595">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R170"><!-- named anchor --></a><span class="citation"><a id="d4e10602"><!-- named anchor --></a>Than C, Ruths D, Nakhleh L (2008) <i>PhyloNet</i>: a software package for analyzing and reconstructing reticulate evolutionary relationships. <i>BMC Bioinformatics</i>  <b>9</b>, 322. <br/> | <a href="https://doi.org/10.1186/1471-2105-9-322" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10613">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=PhyloNet: a software package for analyzing and reconstructing reticulate evolutionary relationships." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10616">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/18662388" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10629">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R171"><!-- named anchor --></a><span class="citation"><a id="d4e10636"><!-- named anchor --></a>Thomson RC, Brown JM (2022) On the need for new measures of phylogenomic support. <i>Systematic Biology</i>  <b>71</b>, 917-920. <br/> | <a href="https://doi.org/10.1093/sysbio/syac002" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10644">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=On the need for new measures of phylogenomic support." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10647">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35088868" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10660">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R172"><!-- named anchor --></a><span class="citation"><a id="d4e10667"><!-- named anchor --></a>Thomas GWC, Ather SH, Hahn MW (2017) Gene-tree reconciliation with MUL-trees to resolve polyploidy events. <i>Systematic Biology</i>  <b>66</b>, 1007-1018. <br/> | <a href="https://doi.org/10.1093/sysbio/syx044" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10675">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Gene-tree reconciliation with MUL-trees to resolve polyploidy events." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10678">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/28419377" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10691">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R173"><!-- named anchor --></a><span class="citation"><a id="d4e10699"><!-- named anchor --></a>Tumescheit C, Firth AE, Brown K (2022) <i>CIAlign</i>: a highly customisable command line tool to clean, interpret and visualise multiple sequence alignments. <i>PeerJ</i>  <b>10</b>, e12983. <br/> | <a href="https://doi.org/10.7717/peerj.12983" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10710">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=CIAlign: a highly customisable command line tool to clean, interpret and visualise multiple sequence alignments." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10713">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35310163" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10726">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R174"><!-- named anchor --></a><span class="citation"><a id="d4e10733"><!-- named anchor --></a>Ufimov R, Gorospe JM, F&#x000E9;r T, Kandziora M, Salomon L, van Loo M, Schmickl R (2022) Utilizing paralogues for phylogenetic reconstruction has the potential to increase species tree support and reduce gene tree discordance in target enrichment data. <i>Molecular Ecology Resources</i>  <b>22</b>, 3018-3034. <br/> | <a href="https://doi.org/10.1111/1755-0998.13684" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10741">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Utilizing paralogues for phylogenetic reconstruction has the potential to increase species tree support and reduce gene tree discordance in target enrichment data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10744">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35796729" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10757">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R175"><!-- named anchor --></a><span class="citation"><a id="d4e10764"><!-- named anchor --></a>Van Dijk K, Waycott M, Biffin E, Creed JC, Albertazzi FJ, Samper-Villarreal J (2023) Phylogenomic insights into the phylogeography of <i>Halophila baillonii</i> Asch. <i>Diversity</i>  <b>15</b>, 111. <br/> | <a href="https://doi.org/10.3390/d15010111" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10775">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic insights into the phylogeography of Halophila baillonii Asch." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10778">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R176"><!-- named anchor --></a><span class="citation"><a id="d4e10795"><!-- named anchor --></a>Vatanparast M, Powell A, Doyle JJ, Egan AN (2018) Targeting legume loci: a comparison of three methods for target enrichment bait design in Leguminosae phylogenomics. <i>Applications in Plant Sciences</i>  <b>6</b>, e1036. <br/> | <a href="https://doi.org/10.1002/aps3.1036" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10803">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Targeting legume loci: a comparison of three methods for target enrichment bait design in Leguminosae phylogenomics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10806">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/29732266" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10819">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R177"><!-- named anchor --></a><span class="citation"><a id="d4e10826"><!-- named anchor --></a>Walsh HE, Kidd MG, Moum T, Friesen VL (1999) Polytomies and the power of phylogenetic inference. <i>Evolution</i>  <b>53</b>, 932-937. <br/> | <a href="https://doi.org/10.1111/j.1558-5646.1999.tb05386.x" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10834">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Polytomies and the power of phylogenetic inference." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10837">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/28565639" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10850">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R178"><!-- named anchor --></a><span class="citation"><a id="d4e10857"><!-- named anchor --></a>Waycott M, Van Dijk K, Biffin E (2021) A hybrid capture RNA bait set for resolving genetic and evolutionary relationships in angiosperms from deep phylogeny to intraspecific lineage hybridization. <i>bioRxiv</i>  <b>2021</b>, 2021.09.06.456727 [Preprint, published 7 September 2021]. <br/> | <a href="https://doi.org/10.1101/2021.09.06.456727" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10865">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A hybrid capture RNA bait set for resolving genetic and evolutionary relationships in angiosperms from deep phylogeny to intraspecific lineage hybridization." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10868">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R179"><!-- named anchor --></a><span class="citation"><a id="d4e10885"><!-- named anchor --></a>Wen D, Yu Y, Nakhleh L (2016) Bayesian inference of reticulate phylogenies under the multispecies network coalescent. <i>PLoS Genetics</i>  <b>12</b>, e1006006. <br/> | <a href="https://doi.org/10.1371/journal.pgen.1006006" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10893">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Bayesian inference of reticulate phylogenies under the multispecies network coalescent." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10896">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/27144273" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10909">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R180"><!-- named anchor --></a><span class="citation"><a id="d4e10916"><!-- named anchor --></a>Wen D, Yu Y, Zhu J, Nakhleh L (2018) Inferring phylogenetic networks using <i>PhyloNet</i>. <i>Systematic Biology</i>  <b>67</b>(4), 735-740. <br/> | <a href="https://doi.org/10.1093/sysbio/syy015" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10927">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Inferring phylogenetic networks using PhyloNet." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10930">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R181"><!-- named anchor --></a><span class="citation"><a id="d4e10947"><!-- named anchor --></a>Whitfield JB, Lockhart PJ (2007) Deciphering ancient rapid radiations. <i>Trends in Ecology &#x00026; Evolution</i>  <b>22</b>, 258-265. <br/> | <a href="https://doi.org/10.1016/j.tree.2007.01.012" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10955">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Deciphering ancient rapid radiations." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10958">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/17300853" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10971">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R182"><!-- named anchor --></a><span class="citation"><a id="d4e10978"><!-- named anchor --></a>Williamson L, Biffin E, Hammer T, van Dijk K, Conran J, Waycott M (2025) Phylogenomics of Australian sundews (<i>Drosera</i>: Droseraceae). <i>Australian Systematic Botany</i>  <b>38</b>, SB24016. <br/> | <a href="https://doi.org/10.1071/SB24016" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10989">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics of Australian sundews (Drosera: Droseraceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e10992">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R183"><!-- named anchor --></a><span class="citation"><a id="d4e11009"><!-- named anchor --></a>Willson J, Roddur MS, Liu B, Zaharias P, Warnow T (2022) <i>DISCO</i>: species tree inference using multicopy gene family tree decomposition. <i>Systematic Biology</i>  <b>71</b>, 610-629. <br/> | <a href="https://doi.org/10.1093/sysbio/syab070" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11020">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=DISCO: species tree inference using multicopy gene family tree decomposition." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11023">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34450658" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11036">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R184"><!-- named anchor --></a><span class="citation"><a id="d4e11043"><!-- named anchor --></a>Xi Z, Liu L, Davis CC (2016) The impact of missing data on species tree estimation. <i>Molecular Biology and Evolution</i>  <b>33</b>, 838-860. <br/> | <a href="https://doi.org/10.1093/molbev/msv266" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11051">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=The impact of missing data on species tree estimation." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11054">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26589995" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11067">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R185"><!-- named anchor --></a><span class="citation"><a id="d4e11075"><!-- named anchor --></a>Xiong H, Wang D, Shao C, Yang X, Yang J, Ma T, Davis CC, Liu L, Xi Z (2022) Species tree estimation and the impact of gene loss following whole-genome duplication. <i>Systematic Biology</i>  <b>71</b>, 1348-1361. <br/> | <a href="https://doi.org/10.1093/sysbio/syac040" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11083">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Species tree estimation and the impact of gene loss following whole-genome duplication." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11086">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/35689633" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11099">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R186"><!-- named anchor --></a><span class="citation"><a id="d4e11106"><!-- named anchor --></a>Yan Z, Smith ML, Du P, Hahn MW, Nakhleh L (2022) Species tree inference methods intended to deal with incomplete lineage sorting are robust to the presence of paralogs. <i>Systematic Biology</i>  <b>71</b>, 367-381. <br/> | <a href="https://doi.org/10.1093/sysbio/syab056" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11114">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Species tree inference methods intended to deal with incomplete lineage sorting are robust to the presence of paralogs." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11117">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34245291" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11130">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R187"><!-- named anchor --></a><span class="citation"><a id="d4e11137"><!-- named anchor --></a>Yang Z (2007) <i>PAML 4</i>: phylogenetic analysis by maximum likelihood. <i>Molecular Biology and Evolution</i>  <b>24</b>, 1586-1591. <br/> | <a href="https://doi.org/10.1093/molbev/msm088" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11148">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=PAML 4: phylogenetic analysis by maximum likelihood." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11151">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/17483113" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11164">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R188"><!-- named anchor --></a><span class="citation"><a id="d4e11171"><!-- named anchor --></a>Yang Y, Smith SA (2014) Orthology inference in nonmodel organisms using transcriptomes and low-coverage genomes: improving accuracy and matrix occupancy for phylogenomics. <i>Molecular Biology and Evolution</i>  <b>31</b>, 3081-3092. <br/> | <a href="https://doi.org/10.1093/molbev/msu245" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11179">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Orthology inference in nonmodel organisms using transcriptomes and low-coverage genomes: improving accuracy and matrix occupancy for phylogenomics." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11182">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/25158799" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11195">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R189"><!-- named anchor --></a><span class="citation"><a id="d4e11202"><!-- named anchor --></a>Yang Y, Moore MJ, Brockington SF, Mikenas J, Olivieri J, Walker JF, Smith SA (2018) Improved transcriptome sampling pinpoints 26 ancient and more recent polyploidy events in Caryophyllales, including two allopolyploidy events. <i>New Phytologist</i>  <b>217</b>, 855-870. <br/> | <a href="https://doi.org/10.1111/nph.14812" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11210">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Improved transcriptome sampling pinpoints 26 ancient and more recent polyploidy events in Caryophyllales, including two allopolyploidy events." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11213">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/28944472" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11226">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R190"><!-- named anchor --></a><span class="citation"><a id="d4e11233"><!-- named anchor --></a>Yu Y, Nakhleh L (2015) A maximum pseudo-likelihood approach for phylogenetic networks. <i>BMC Genomics</i>  <b>16</b>, S10. <br/> | <a href="https://doi.org/10.1186/1471-2164-16-S10-S10" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11241">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A maximum pseudo-likelihood approach for phylogenetic networks." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11244">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/26450642" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11257">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R191"><!-- named anchor --></a><span class="citation"><a id="d4e11265"><!-- named anchor --></a>Zhang C, Mirarab S (2022) <i>ASTRAL-Pro 2</i>: ultrafast species tree reconstruction from multi-copy gene family trees. <i>Bioinformatics</i>  <b>38</b>, 4949-4950. <br/> | <a href="https://doi.org/10.1093/bioinformatics/btac620" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11276">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=ASTRAL-Pro 2: ultrafast species tree reconstruction from multi-copy gene family trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11279">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/36094339" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11292">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R192"><!-- named anchor --></a><span class="citation"><a id="d4e11299"><!-- named anchor --></a>Zhang C, Ogilvie HA, Drummond AJ, Stadler T (2018<i><i>a</i></i>) Bayesian inference of species networks from multilocus sequence data. <i>Molecular Biology and Evolution</i>  <b>35</b>, 504-517. <br/> | <a href="https://doi.org/10.1093/molbev/msx307" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11313">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Bayesian inference of species networks from multilocus sequence data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11316">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R193"><!-- named anchor --></a><span class="citation"><a id="d4e11333"><!-- named anchor --></a>Zhang C, Rabiee M, Sayyari E, Mirarab S (2018<i><i>b</i></i>) <i>ASTRAL-III</i>: polynomial time species tree reconstruction from partially resolved gene trees. <i>BMC Bioinformatics</i>  <b>19</b>, 153. <br/> | <a href="https://doi.org/10.1186/s12859-018-2129-y" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11350">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11354">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R194"><!-- named anchor --></a><span class="citation"><a id="d4e11370"><!-- named anchor --></a>Zhang C, Scornavacca C, Molloy EK, Mirarab S (2020) <i>ASTRAL-Pro</i>: quartet-based species-tree inference despite paralogy. <i>Molecular Biology and Evolution</i>  <b>37</b>, 3292-3307. <br/> | <a href="https://doi.org/10.1093/molbev/msaa139" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11381">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=ASTRAL-Pro: quartet-based species-tree inference despite paralogy." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11384">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/32886770" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11397">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R195"><!-- named anchor --></a><span class="citation"><a id="d4e11404"><!-- named anchor --></a>Zhang C, Zhao Y, Braun EL, Mirarab S (2021) <i>TAPER</i>: pinpointing errors in multiple sequence alignments despite varying rates of evolution. <i>Methods in Ecology and Evolution</i>  <b>12</b>(11), 2145-2158. <br/> | <a href="https://doi.org/10.1111/2041-210x.13696" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11415">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=TAPER: pinpointing errors in multiple sequence alignments despite varying rates of evolution." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11418">Google Scholar</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R196"><!-- named anchor --></a><span class="citation"><a id="d4e11435"><!-- named anchor --></a>Zhang Q, Folk RA, Mo Z-Q, Ye H, Zhang Z-Y, Peng H, Zhao J-L, Yang S-X, Yu X-Q (2023) Phylotranscriptomic analyses reveal deep gene tree discordance in <i>Camellia</i> (Theaceae). <i>Molecular Phylogenetics and Evolution</i>  <b>188</b>, 107912. <br/> | <a href="https://doi.org/10.1016/j.ympev.2023.107912" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11446">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylotranscriptomic analyses reveal deep gene tree discordance in Camellia (Theaceae)." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11449">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/37648181" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11462">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R197"><!-- named anchor --></a><span class="citation"><a id="d4e11470"><!-- named anchor --></a>Zhang Z, Xie P, Guo Y, Zhou W, Liu E, Yu Y (2022) <i>Easy353</i>: a tool to get Angiosperms353 genes for phylogenomic research. <i>Molecular Biology and Evolution</i>  <b>39</b>, msac261. <br/> | <a href="https://doi.org/10.1093/molbev/msac261" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11481">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Easy353: a tool to get Angiosperms353 genes for phylogenomic research." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11484">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/36458838" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11497">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R198"><!-- named anchor --></a><span class="citation"><a id="d4e11504"><!-- named anchor --></a>Zhou W, Soghigian J, Xiang QJ (2022) A new pipeline for removing paralogs in target enrichment data. <i>Systematic Biology</i>  <b>71</b>, 410-425. <br/> | <a href="https://doi.org/10.1093/sysbio/syab044" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11512">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=A new pipeline for removing paralogs in target enrichment data." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11515">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/34146111" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11528">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R199"><!-- named anchor --></a><span class="citation"><a id="d4e11535"><!-- named anchor --></a>Zhou X, Lutteropp S, Czech L, Stamatakis A, Looz MV, Rokas A (2020) Quartet-based computations of internode certainty provide robust measures of phylogenetic incongruence. <i>Systematic Biology</i>  <b>69</b>, 308-324. <br/> | <a href="https://doi.org/10.1093/sysbio/syz058" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11543">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Quartet-based computations of internode certainty provide robust measures of phylogenetic incongruence." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11546">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/31504977" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11559">PubMed</a> | </span></p></div></div><div class="row"><div class="ref-content cell"><p><a id="R200"><!-- named anchor --></a><span class="citation"><a id="d4e11566"><!-- named anchor --></a>Zuntini A, Carruthers T, Maurin O, Bailey PC, Leempoel K, Brewer GE, Epitawalage N, Fran&#x000E7;oso E, Gallego-Paramo B, McGinnie C, Negr&#x000E3;o R, Roy SR, Simpson L, Toledo Romero E, Barber VMA, Botigu&#x000E9; L, Clarkson JJ, Cowan RS, Dodsworth S, Johnson MG, Kim JT, Pokorny L, Wickett NJ, Antar GM, DeBolt L, Gutierrez K, Hendriks KP, Hoewener A, Hu A-Q, Joyce EM, Kikuchi IABS, Larridon I, Larson DA, de L&#x000ED;rio EJ, Liu J-X, Malakasi P, Przelomska NAS, Shah T, Viruel J, Allnutt TR, Ameka GK, Andrew RL, Appelhans MS, Arista M, Ariza MJ, Arroyo J, Arthan W, Bachelier JB, Bailey CD, Barnes HF, Barrett MD, Barrett RL, Bayer RJ, Bayly MJ, Biffin E, Biggs N, Birch JL, Bogar&#x000ED;n D, Borosova R, Bowles AMC, Boyce PC, Bramley GLC, Briggs M, Broadhurst L, Brown GK, Bruhl JJ, Bruneau A, Buerki S, Burns E, Byrne M, Cable S, Calladine A, Callmander MW, Cano &#x000C1;, Cantrill DJ, Cardinal-McTeague WM, Carlsen MM, Carruthers AJA, de Castro Mateo A, Chase MW, Chatrou LW, Cheek M, Chen S, Christenhusz MJM, Christin P-A, Clements MA, Coffey SC, Conran JG, Cornejo X, Couvreur TLP, Cowie ID, Csiba L, Darbyshire I, Davidse G, Davies NMJ, Davis AP, van Dijk K, Downie SR, Duretto MF, Duvall MR, Edwards SL, Eggli U, Erkens RHJ, Escudero M, de la Estrella M, Fabriani F, Fay MF, Ferreira Pde L, Ficinski SZ, Fowler RM, Frisby S, Fu L, Fulcher T, Galbany-Casals M, Gardner EM, German DA, Giaretta A, Gibernau M, Gillespie LJ, Gonz&#x000E1;lez CC, Goyder DJ, Graham SW, Grall A, Green L, Gunn BF, Guti&#x000E9;rrez DG, Hackel J, Haevermans T, Haigh A, Hall JC, Hall T, Harrison MJ, Hatt SA, Hidalgo O, Hodkinson TR, Holmes GD, Hopkins HCF, Jackson CJ, James SA, Jobson RW, Kadereit G, Kahandawala IM, Kainulainen K, Kato M, Kellogg EA, King GJ, Klejevskaja B, Klitgaard BB, Klopper RR, Knapp S, Koch MA, Leebens-Mack JH, Lens F, Leon CJ, L&#x000E9;veill&#x000E9;-Bourret &#x000C9;, Lewis GP, Li D-Z, Li L, Liede-Schumann S, Livshultz T, Lorence D, Lu M, Lu-Irving P, Luber J, Lucas EJ, Luj&#x000E1;n M, Lum M, Macfarlane TD, Magdalena C, Mansano VF, Masters LE, Mayo SJ, McColl K, McDonnell AJ, McDougall AE, McLay TGB, McPherson H, Meneses RI, Merckx VSFT, Michelangeli FA, Mitchell JD, Monro AK, Moore MJ, Mueller TL, Mummenhoff K, Munzinger J, Muriel P, Murphy DJ, Nargar K, Nauheimer L, Nge FJ, Nyffeler R, Orejuela A, Ortiz EM, Palazzesi L, Peixoto AL, Pell SK, Pellicer J, Penneys DS, Perez-Escobar OA, Persson C, Pignal M, Pillon Y, Pirani JR, Plunkett GM, Powell RF, Prance GT, Puglisi C, Qin M, Rabeler RK, Rees PEJ, Renner M, Roalson EH, Rodda M, Rogers ZS, Rokni S, Rutishauser R, de Salas MF, Schaefer H, Schley RJ, Schmidt-Lebuhn A, Shapcott A, Al-Shehbaz I, Shepherd KA, Simmons MP, Sim&#x000F5;es AO, Sim&#x000F5;es A, Rita G, Siros M, Smidt EC, Smith JF, Snow N, Soltis DE, Soltis PS, Soreng RJ, Sothers CA, Starr JR, Stevens PF, Straub SCK, Struwe L, Taylor JM, Telford IRH, Thornhill AH, Tooth I, Trias-Blasi A, Udovicic F, Utteridge TMA, Del Valle JC, Verboom GA, Vonow HP, Vorontsova MS, de Vos JM, Al-Wattar N, Waycott M, Welker CAD, White AJ, Wieringa JJ, Williamson LT, Wilson TC, Wong SY, Woods LA, Woods R, Worboys S, Xanthos M, Yang Y, Zhang Y-X, Zhou M-Y, Zmarzty S, Zuloaga FO, Antonelli A, Bellot S, Crayn DM, Grace OM, Kersey PJ, Leitch IJ, Sauquet H, Smith SA, Eiserhardt WL, Forest F, Baker WJ (2024) Phylogenomics and the rise of the angiosperms. <i>Nature</i>  <b>629</b>, 843-850. <br/> | <a href="https://doi.org/10.1038/s41586-024-07324-0" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11574">Crossref</a> | <a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics and the rise of the angiosperms." target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11577">Google Scholar</a> | <a href="https://pubmed.ncbi.nlm.nih.gov/38658746" target="_blank" rel="nofollow noopener noreferrer" style="overflow-wrap: anywhere;" id="d4e11590">PubMed</a> | </span></p></div></div></div></div></div></div>
			</div>
		
			</section>
		

				<section id="second-sidebar" class="region-second-sidebar small-12 large-3 medium-6 no-pad columns">
					<div id="ftlinks">
						
	<div class="highlight-box article-action">
		<p><a href="/sb/PublishingPolicies"><img src="/media/client/cope_logo_sb.png" alt="Committee on Publication Ethics" data-rjs="2"></a></p>
	</div>
	
		<div class="gray-highlight-box article-action" tabindex=0>
			
							<a class="article-html" href="/sb/Fulltext/SB24011">
								Full Text
							</a>
						
							<a class="article-pdf" href="/sb/pdf/SB24011">PDF (2.7 MB)</a>
						
			<a class="article-export" href="/sb/ExportCitation/SB24011">Export Citation</a>
			
				<a class="article-export" href="https://www.copyright.com/openurl.do?contentIdType=doi&issn=1030-1887&contentID=10.1071/SB24011">Get Permission</a>
		
		</div>
		
	<div class="gray-highlight-box" tabindex=1>
		<p>Share</p>
		<a href="/linkout/facebook/u=www.publish.csiro.au/SB/SB24011" target="_blank" rel="nofollow noopener noreferrer"><img src="/images/facebook.png" alt="Share on Facebook" data-rjs="2" width="28" height="28"></a>
		<a href="/linkout/twitter/url=www.publish.csiro.au/SB/SB24011&text=Navigating phylogenetic conflict and evolutionary inference in plants with target-capture data" target="_blank" rel="nofollow noopener noreferrer"><img src="/images/twitter.png" alt="Share on Twitter" data-rjs="2" width="28" height="28"></a>
		<a href="/linkout/linkedin/mini=true&amp;url=www.publish.csiro.au/SB/SB24011" target="_blank" rel="nofollow noopener noreferrer"><img src="/images/linkedin.png" alt="Share on LinkedIn" data-rjs="2" width="28" height="28"></a>
 		<a href="mailto:?to=&subject=Currently reading in Australian Systematic Botany&body=Navigating phylogenetic conflict and evolutionary inference in plants with target-capture data https://www.publish.csiro.au/SB/SB24011"><img src="/images/email.png" alt="Share via Email" data-rjs="2" width="28" height="28"></a>
	</div>

		<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
		<div class="gray-highlight-box article-action" tabindex=2>
			<p>View Dimensions</p>
			<div class="__dimensions_badge_embed__"  data-doi="10.1071/SB24011" data-style="small_circle" data-legend="never"></div>
		</div>
	
			<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
			<div class="gray-highlight-box article-action" tabindex=1>
				<p>View Altmetrics</p>
				<div data-badge-type="donut" data-doi="10.1071/SB24011" data-condensed="true" data-hide-no-mentions="true" class="altmetric-embed"></div>
			</div>
		
					</div>
				</section>
	
		   </main>
		</div>
	<aside id="first-sidebar" class="region-first-sidebar small-12 large-3 large-pull-9 medium-3 medium-pull-9 columns"> 
		    <section class="region region-first-sidebar sidebar">
	            <h2 class="element-invisible" id="sidenavigation">Journal Navigation</h2>
    	        <nav class="side-nav gray-highlight-box" aria-labelledby="sidenavigation">
					<ul>
		


					<li>

						
							<span class="hiddentext">Left Panel Navigation</span><a href="/sb">Journal Home</a>
						

						<ul>
						
								<li><a href="/sb/AbouttheJournal" class="linknav">About the Journal</a></li>
								
								
								<li><a href="/sb/EditorialStructure" class="linknav">Editorial Structure</a></li>
								
								
								<li><a href="/sb/PublishingPolicies" class="linknav">Publishing Policies</a></li>
								
								
								<li><a href="/sb/LowerIncomeCountries" class="linknav">Lower Income Countries</a></li>
								
								
								<li><a href="/sb/Contacts" class="linknav">Contacts</a></li>
								
								 
						</ul>
					
					</li>
				


					<li>

						
							<span class="hiddentext">Left Panel Navigation</span><a href="/sb/content">Content</a>
						

						<ul>
						
											<li>
												<a href="/sb#Latest" class="open-tab" onclick="window.location.href='/sb/#Latest';location.load();"><span class="hiddentext">Left Panel Navigation</span>Latest</a>
											</li>
											
													<li>
														<a href="/sb#JustAccepted" class="open-tab" onclick="window.location.href='/sb/#JustAccepted';location.load();"><span class="hiddentext">Left Panel Navigation</span>Just Accepted</a>
													</li>
													
													  <li>
														  <a href="/sb#MostRead" class="open-tab" onclick="window.location.href='/sb/#MostRead';location.load();"><span class="hiddentext">Left Panel Navigation</span>Most Read</a>
													</li>
												
												<li>
													<a href="/sb#Collections" class="open-tab" onclick="window.location.href='/sb/#Collections';location.load();"><span class="hiddentext">Left Panel Navigation</span>Collections</a>
											  	</li>
										  
								<li><a href="/sb/content/allissues" class="linknav">All Content</a></li>
								
								
								<li><a href="/sb/content/specialissues" class="linknav">Special Issues</a></li>
								
								
								<li><a href="/sb/content/L.A.S.JohnsonReviewSeries" class="linknav">L.A.S. Johnson Review Series</a></li>
								
								
								<li><a href="/sb/content/CallforPapers" class="linknav">Call for Papers</a></li>
								
								 
						</ul>
					
					</li>
				


					<li>

						
							<span class="hiddentext">Left Panel Navigation</span><a href="/sb/forauthors">For Authors</a>
						

						<ul>
						
								<li><a href="/sb/forauthors/Publishwithus" class="linknav">Publish with us</a></li>
								
								
								<li><a href="/sb/forauthors/Scope" class="linknav">Scope</a></li>
								
								
										<li>
										
											<a href="#" onclick="javascript:window.open('https://mc.manuscriptcentral.com/csiro-sb','',    'toolbar=1,location=1,directories=1,status=1,menubar=1,scrollbars=1,resizable=1,width=860,height=800')" class="linknavTitleJournals">
										Submit Article
										</a>
										</li>
									
								<li><a href="/sb/forauthors/AuthorInstructions" class="linknav">Author Instructions</a></li>
								
								
								<li><a href="/sb/forauthors/LicencetoPublish" class="linknav">Licence to Publish</a></li>
								
								
								<li><a href="/sb/forauthors/OpenAccess" class="linknav">Open Access</a></li>
								
								
								<li><a href="/sb/forauthors/ReadandPublish" class="linknav">Read and Publish</a></li>
								
								
								<li><a href="/sb/forauthors/PublishingCharges" class="linknav">Publishing Charges</a></li>
								
								
								<li><a href="/sb/forauthors/AwardsandPrizes" class="linknav">Awards and Prizes</a></li>
								
								 
						</ul>
					
					</li>
				


					<li>

						
							<span class="hiddentext">Left Panel Navigation</span><a href="/sb/forreviewers">For Reviewers</a>
						

						<ul>
						
								<li><a href="/sb/forreviewers/ReviewerGuidelines" class="linknav">Reviewer Guidelines</a></li>
								
								
										<li>
										
											<a href="#" onclick="javascript:window.open('https://mc.manuscriptcentral.com/csiro-sb','',    'toolbar=1,location=1,directories=1,status=1,menubar=1,scrollbars=1,resizable=1,width=860,height=800')" class="linknavTitleJournals">
										Review Article </a>
										</li>
									
								<li><a href="/sb/forreviewers/ReviewerRecognition" class="linknav">Reviewer Recognition</a></li>
								
								
								<li><a href="/sb/forreviewers/AnnualReviewerIndex" class="linknav">Annual Reviewer Index</a></li>
								
								 
						</ul>
					
					</li>
				


					<li>

						
							<span class="hiddentext">Left Panel Navigation</span><a href="/sb/forsubscribers">For Subscribers</a>
						

						<ul>
						
								<li><a href="/sb/forsubscribers/SubscriptionPrices" class="linknav">Subscription Prices</a></li>
								
								
								<li><a href="/sb/forsubscribers/customerservice" class="linknav">Customer Service</a></li>
								
								
								<li><a href="/sb/forsubscribers/libraryrecommendation" class="linknav">Library Recommendation</a></li>
								
								 
						</ul>
					
					</li>
				


					<li>

						
							<span class="hiddentext">Left Panel Navigation</span><a href="/sb/foradvertisers">For Advertisers</a>
						

						<ul>
						 
						</ul>
					
					</li>
				

				</ul>
				</nav>
				</section>

			
	 <div class="blue-highlight-box"><img src="/images/e-alerts-big.png" alt="email icon" data-rjs="2"  width="43" height="21" aria-hidden="true">
	 	<h2>e-Alerts</h2>
	 	
		<p>Subscribe to our <A href="?nid=25&aid=685">Email Alert</A> or <a href="http://www.publish.csiro.au/RSS_Feed/CSIRO_Publishing_Recent_SB.xml"><IMG alt="RSS" src="/media/client/rss_smaller.gif" align=none style="margin-bottom:-2px;" border=0></a> feeds for the latest journal papers.</p>
	 </div>
	 <br />


			<div class="gray-highlight-box">
						<h2>Brunonia</h2>


					
					<p><A href="http://www.publish.csiro.au/nid/150.htm?nid=151&aid=118">Brunonia</A>, the predecessor journal to <b>Australian Systematic Botany</b>, is available online.</p>

			</div>
		

			    </aside>
		


	</div>
    <footer id="footer" class="cf region region-footer">
		<div class="row">
         <div class="small-12 medium-3 large-3 no-pad-left columns">
          <h2>Links</h2>
            <nav class="footer-nav" aria-labelledby="firstlinksnav">
              <p class="element-invisible" id="firstlinksnav">Links first column</p>
              <ul>
                <li><a href="/aboutus">About Us</a></li>
                <li><a href="/aboutus/contactus">Contact Us</a></li>
                <li><a href="/aboutus/help">Help</a></li>
                <li><a href="/workshops">Workshops</a></li>
                <li><a href="/learning/cosmos">Cosmos </a></li>
              </ul>
            </nav>
        </div>
        <div class="small-12 medium-6 no-pad-left large-6 columns">
          <h2>Browse by subject</h2>
          <div class="small-12 medium-6 large-6 no-pad-left columns">
            <nav class="footer-nav" aria-labelledby="firstbrowsebysubjectnav">
              <p class="element-invisible" id="firstbrowsebysubjectnav">Browse by subject first column</p>
              <ul>
                <li><a href="/Animals">Animals</a></li>
                <li><a href="/BuiltEnvironment">Built Environment</a></li>
                <li><a href="/FoodAndAgriculture">Food &amp; Agriculture</a></li>
                <li><a href="/GardeningAndHorticulture">Gardening &amp; Horticulture</a></li>
                <li><a href="/Children">Children</a></li>
              </ul>
            </nav>
          </div>
          <div class="small-12 medium-6 large-6 no-pad-left columns">
            <nav class="footer-nav" aria-labelledby="secondbrowsebysubjectnav">
              <p class="element-invisible" id="secondbrowsebysubjectnav">Browse by subject second column</p>
              <ul>
                <li><a href="/MarineAndFreshwater">Marine &amp; Freshwater</a></li>
                <li><a href="/NaturalEnvironment">Natural Environment</a></li>
                <li><a href="/PhysicalSciences">Physical Sciences</a></li>
                <li><a href="/PlantsAndFungi">Plants &amp; Fungi</a></li>
                <li><a href="/ScienceinSociety">Science in Society</a></li>
              </ul>
            </nav>
          </div>
        </div>
        <div class="small-12 medium-3 large-3 footer-connect no-pad-left columns">
          <h2>Connect with us</h2>
            <span><a class="icon-facebook" href="https://www.facebook.com/pages/CSIRO-PUBLISHING/70534682887" target="_blank" rel="noopener noreferrer"><img src="/images/facebook.png" alt="Like us on Facebook" width="37" height="37"></a></span>
            <span><a class="icon-bluesky" href="https://bsky.app/profile/csiropublishing.bsky.social" target="_blank" rel="noopener noreferrer"><img src="/images/bluesky.png" alt="Follow us on Bluesky" width="37" height="37"></a></span>
            <span><a class="icon-instagram" href="https://www.instagram.com/csiropublishing/" target="_blank" rel="noopener noreferrer"><img src="/images/instagram.png" alt="Connect with us on Instagram" width="37" height="37"></a></span>
            <span><a class="icon-linkedin" href="https://www.linkedin.com/company/csiro-publishing" target="_blank" rel="noopener noreferrer"><img src="/images/linkedin.png" alt="Connect with us on LinkedIn" width="37" height="37"></a></span>
        </div> 
<div class="small-12 large-12 columns">
	<div class="small-12 large-12 footer-legal text-center columns">
		<p><span>We acknowledge the Traditional Owners of the land, sea and waters, of the areas that we live and work on across Australia. We acknowledge their continuing connection to their culture, their contribution to our shared knowledge, and pay our respects to their Elders past and present.</span></p>
		<div class="footer-legal-content">
			<p><a href="/journals/copyright">Copyright</a> <a href="/aboutus/legal&privacy">Legal Notice and Disclaimer</a> <a href="/aboutus/legal&privacy#7">Privacy</a></p>
		</div>
		<p><a href="https://www.csiro.au"><img src="/images/layout/csiro-logo.png" alt="CSIRO" data-rjs="2" width="58" height="58"></a>
		
			<a href="http://www.science.org.au"><img src="/media/client/aos_logo.png" alt="Australian Academy Of Science" data-rjs="2" width="133" height="48"></a>
		
	
	</div>
</div>

     <script src="https://static.getclicky.com/59359.js" type="text/javascript"></script>
     <noscript><p><img alt="Clicky" width="1" height="1" src="https://static.getclicky.com/59359-db4.gif" /></p></noscript>

		</div>
    </footer>
	<script src="/themes/new/js/vendor/jquery.min.js"></script>
<script src="/themes/new/js/vendor/what-input.js"></script>
<script src="/themes/new/js/vendor/foundation.min.js"></script>
<script src="/themes/new/js/app.js"></script>
<script src="/themes/new/js/jquery.okayNav.js"></script>
<script src="/themes/new/js/jquery.validate.min.js"></script>
<script src="/themes/new/js/navigation.js"></script>


<script src="/themes/new/js/journals_article.js"></script>


<script src="/themes/new/js/common.js"></script>
<script src="/themes/new/js/cart.js"></script>
<script src="/themes/new/js/journals.js"></script>
</body>
</html>
